diff --git a/src/dev_ng/game/task/General/TaskBasic.h b/src/dev_ng/game/task/General/TaskBasic.h
--- a/src/dev_ng/game/task/General/TaskBasic.h
+++ b/src/dev_ng/game/task/General/TaskBasic.h
@@ -5,20 +5,20 @@
 #include "ai/task/taskchannel.h"
 
 // Game headers
-#include "Event/Events.h"
-#include "IK/IkManager.h" // Required for default IK priority argument 
-#include "Network/Objects/Synchronisation/SyncNodes/PedSyncNodes.h"
+#include "event/Events.h"
+#include "ik/IkManager.h" // Required for default IK priority argument 
+#include "network/Objects/Synchronisation/SyncNodes/PedSyncNodes.h"
 #include "Peds/DefensiveArea.h"
 #include "Peds/QueriableInterface.h"
-#include "Physics/Constrainthandle.h"
+#include "physics/constrainthandle.h"
 #include "script/Handlers/GameScriptHandler.h"
 #include "script/Handlers/GameScriptResources.h"
-#include "Task/Default/TaskAmbient.h"
-#include "Task/System/TaskFSMClone.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/TaskMove.h"
-#include "Task/System/TaskTypes.h"
-#include "Task/System/TaskComplex.h"
+#include "task/Default/TaskAmbient.h"
+#include "task/System/TaskFSMClone.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/TaskMove.h"
+#include "task/System/TaskTypes.h"
+#include "task/System/TaskComplex.h"
 
 class CEntity;
 class CPed;
