diff --git a/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.cpp b/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.cpp
--- a/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.cpp
+++ b/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.cpp
@@ -5,19 +5,19 @@
 //
 
 // File header
-#include "Weapons/Components/WeaponComponentLaserSight.h"
+#include "weapons/components/WeaponComponentLaserSight.h"
 
 // Game headers
-#include "Peds/Ped.h"
+#include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
-#include "Physics/Physics.h"
-#include "physics/WorldProbe/worldprobe.h"
-#include "Task/Weapons/Gun/TaskAimGun.h"
-#include "Vfx/Decals/DecalManager.h"
-#include "Vfx/Misc/Coronas.h"
-#include "Vfx/Systems/VfxWeapon.h"
-#include "Weapons/WeaponManager.h"
-#include "Weapons/Helpers/WeaponHelpers.h"
+#include "physics/physics.h"
+#include "physics/WorldProbe/WorldProbe.h"
+#include "task/Weapons/Gun/TaskAimGun.h"
+#include "vfx/decals/DecalManager.h"
+#include "vfx/misc/Coronas.h"
+#include "vfx/systems/VfxWeapon.h"
+#include "weapons/WeaponManager.h"
+#include "weapons/helpers/WeaponHelpers.h"
 
 WEAPON_OPTIMISATIONS()
 
