diff --git a/src/dev_ng/game/weapons/components/WeaponComponentPool.cpp b/src/dev_ng/game/weapons/components/WeaponComponentPool.cpp
--- a/src/dev_ng/game/weapons/components/WeaponComponentPool.cpp
+++ b/src/dev_ng/game/weapons/components/WeaponComponentPool.cpp
@@ -5,18 +5,18 @@
 //
 
 // File header
-#include "Weapons/Components/WeaponComponent.h"
+#include "weapons/components/WeaponComponent.h"
 
 // Game headers
-#include "Weapons/Components/WeaponComponentClip.h"
-#include "Weapons/Components/WeaponComponentFlashLight.h"
-#include "Weapons/Components/WeaponComponentGroup.h"
-#include "Weapons/Components/WeaponComponentLaserSight.h"
-#include "Weapons/Components/WeaponComponentProgrammableTargeting.h"
-#include "Weapons/Components/WeaponComponentScope.h"
-#include "Weapons/Components/WeaponComponentSuppressor.h"
-#include "Weapons/Components/WeaponComponentVariantModel.h"
-#include "Weapons/Weapon.h"
+#include "weapons/components/WeaponComponentClip.h"
+#include "weapons/components/WeaponComponentFlashLight.h"
+#include "weapons/components/WeaponComponentGroup.h"
+#include "weapons/components/WeaponComponentLaserSight.h"
+#include "weapons/components/WeaponComponentProgrammableTargeting.h"
+#include "weapons/components/WeaponComponentScope.h"
+#include "weapons/components/WeaponComponentSuppressor.h"
+#include "weapons/components/WeaponComponentVariantModel.h"
+#include "weapons/Weapon.h"
 
 WEAPON_OPTIMISATIONS()
 
