diff --git a/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.h b/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.h
--- a/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.h
+++ b/src/dev_ng/game/weapons/components/WeaponComponentLaserSight.h
@@ -8,8 +8,8 @@
 #define WEAPON_COMPONENT_LASER_SIGHT_H
 
 // Game headers
-#include "Weapons/Components/WeaponComponent.h"
-#include "physics/WorldProbe/worldprobe.h"
+#include "weapons/components/WeaponComponent.h"
+#include "physics/WorldProbe/WorldProbe.h"
 #include "vector/vector3.h"
 
 
