diff --git a/src/dev_ng/game/weapons/components/WeaponComponentVariantModel.h b/src/dev_ng/game/weapons/components/WeaponComponentVariantModel.h
--- a/src/dev_ng/game/weapons/components/WeaponComponentVariantModel.h
+++ b/src/dev_ng/game/weapons/components/WeaponComponentVariantModel.h
@@ -8,7 +8,7 @@
 #define WEAPON_COMPONENT_VARIANT_MODEL_H
 
 // Game headers
-#include "Weapons/Components/WeaponComponent.h"
+#include "weapons/components/WeaponComponent.h"
 
 
 ////////////////////////////////////////////////////////////////////////////////
