diff --git a/src/dev_ng/game/weapons/components/WeaponComponent.cpp b/src/dev_ng/game/weapons/components/WeaponComponent.cpp
--- a/src/dev_ng/game/weapons/components/WeaponComponent.cpp
+++ b/src/dev_ng/game/weapons/components/WeaponComponent.cpp
@@ -5,15 +5,15 @@
 //
 
 // File header
-#include "Weapons/Components/WeaponComponent.h"
+#include "weapons/components/WeaponComponent.h"
 
 // Game headers
-#include "Peds/Ped.h"
+#include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
-#include "Weapons/Helpers/WeaponHelpers.h"
+#include "weapons/helpers/WeaponHelpers.h"
 #include "fwscene/stores/txdstore.h"
-#include "modelinfo\BaseModelInfo.h"
-#include "Network/NetworkInterface.h"
+#include "modelinfo/BaseModelInfo.h"
+#include "network/NetworkInterface.h"
 #include "Objects/object.h"
 #include "entity/drawdata.h"
 #include "shaders/CustomShaderEffectWeapon.h"
