diff --git a/src/dev_ng/rage/suite/src/softrasterizer/softrasterizer.h b/src/dev_ng/rage/suite/src/softrasterizer/softrasterizer.h
--- a/src/dev_ng/rage/suite/src/softrasterizer/softrasterizer.h
+++ b/src/dev_ng/rage/suite/src/softrasterizer/softrasterizer.h
@@ -16,11 +16,11 @@
 #include "data/base.h"
 
 #include "softrasterizer/boxoccluder.h"
-#include "softrasterizer/occludemodel.h"
+#include "softrasterizer/occludeModel.h"
 #include "softrasterizer/tiler.h"
 
 #if __BANK
-#include "vector\color32.h"
+#include "vector/color32.h"
 #endif
 namespace rage {
 
