diff --git a/src/dev_ng/rage/suite/src/rmptfx/ptxbiaslink.cpp b/src/dev_ng/rage/suite/src/rmptfx/ptxbiaslink.cpp
--- a/src/dev_ng/rage/suite/src/rmptfx/ptxbiaslink.cpp
+++ b/src/dev_ng/rage/suite/src/rmptfx/ptxbiaslink.cpp
@@ -10,7 +10,7 @@
 #include "rmptfx/ptxbiaslink_parser.h"
 
 // includes (rmptfx)
-#include "rmptfx/ptxbytebuff.h"
+#include "rmptfx/ptxByteBuff.h"
 #include "rmptfx/ptxdebug.h"
 #include "rmptfx/ptxkeyframeprop.h"
 
