diff --git a/src/dev_ng/rage/suite/src/rmptfx/ptxshader.cpp b/src/dev_ng/rage/suite/src/rmptfx/ptxshader.cpp
--- a/src/dev_ng/rage/suite/src/rmptfx/ptxshader.cpp
+++ b/src/dev_ng/rage/suite/src/rmptfx/ptxshader.cpp
@@ -9,7 +9,7 @@
 #include "rmptfx/ptxshader_parser.h"
 
 // includes (rmptfx)
-#include "rmptfx/ptxbytebuff.h"
+#include "rmptfx/ptxByteBuff.h"
 #include "rmptfx/ptxchannel.h"
 #include "rmptfx/ptxdebug.h"
 #include "rmptfx/ptxmanager.h"
