diff --git a/src/dev_ng/game/weapons/ThrownWeaponInfo.h b/src/dev_ng/game/weapons/ThrownWeaponInfo.h
--- a/src/dev_ng/game/weapons/ThrownWeaponInfo.h
+++ b/src/dev_ng/game/weapons/ThrownWeaponInfo.h
@@ -2,10 +2,10 @@
 #define INC_THROWNWEAPONINFO_H_
 
 // Rage headers
-#include "Vector/Matrix34.h"
+#include "vector/matrix34.h"
 
 // Game headers
-#include "ModelInfo/BaseModelInfo.h"
+#include "modelinfo/BaseModelInfo.h"
 #include "scene/RegdRefTypes.h"
 
 // Forward declarations
