diff --git a/src/dev_ng/game/weapons/ThrownWeaponInfo.h b/src/dev_ng/game/weapons/ThrownWeaponInfo.h
--- a/src/dev_ng/game/weapons/ThrownWeaponInfo.h
+++ b/src/dev_ng/game/weapons/ThrownWeaponInfo.h
@@ -5,7 +5,7 @@
 #include "Vector/Matrix34.h"
 
 // Game headers
-#include "ModelInfo/BaseModelInfo.h"
+#include "modelinfo/BaseModelInfo.h"
 #include "scene/RegdRefTypes.h"
 
 // Forward declarations
