diff --git a/src/dev_ng/game/scene/loader/mapTypes.cpp b/src/dev_ng/game/scene/loader/mapTypes.cpp
--- a/src/dev_ng/game/scene/loader/mapTypes.cpp
+++ b/src/dev_ng/game/scene/loader/mapTypes.cpp
@@ -15,13 +15,13 @@
 
 //game include 
 #include "modelinfo/BaseModelInfo.h"
-#include "modelInfo/CompEntityModelInfo.h"
+#include "modelinfo/CompEntityModelInfo.h"
 #include "scene/scene_channel.h"
 #include "system/memmanager.h"
 
 //pargen includes
 #include "MapData_Extensions.h"
-#include "mapTypes_parser.h"
+#include "MapTypes_parser.h"
 
 SCENE_OPTIMISATIONS();
 
