diff --git a/src/dev_ng/rage/base/src/file/remote.cpp b/src/dev_ng/rage/base/src/file/remote.cpp
--- a/src/dev_ng/rage/base/src/file/remote.cpp
+++ b/src/dev_ng/rage/base/src/file/remote.cpp
@@ -55,7 +55,7 @@
 #include <net.h>
 #endif
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 namespace rage {
 	unsigned CustomMessageBox(HWND hWnd, LPCSTR lpText, LPCSTR lpCaption, UINT uType, UINT uOptions);
 }
@@ -2477,6 +2477,10 @@ unsigned fiRemoteShowAssertMessageBox(const char *message,const char *title,unsi
 	sysInterlockedDecrement(&fiIsShowingMessageBox);
 	return mbr.result;
 }
+#elif __LINUX
+unsigned fiRemoteShowMessageBox(const char *message,const char *title,unsigned flags,unsigned defaultAnswer,unsigned options)
+{
+}
 #elif __WIN32
 #include "system/xtl.h"
 
