diff --git a/src/dev_ng/game/control/gamelogic.cpp b/src/dev_ng/game/control/gamelogic.cpp
--- a/src/dev_ng/game/control/gamelogic.cpp
+++ b/src/dev_ng/game/control/gamelogic.cpp
@@ -77,7 +77,7 @@
 #include "task/Movement/TaskParachute.h"
 #include "Task/System/Task.h"
 #include "Task/Vehicle/TaskCar.h"
-#include "Task\System\TaskTypes.h"
+#include "Task/System/TaskTypes.h"
 #include "Task/Vehicle/TaskCarAccessories.h"
 #include "text/messages.h"
 #include "vehicles/automobile.h"
@@ -87,7 +87,7 @@
 #include "vehicles/vehicle.h"
 #include "vehicles/vehicleFactory.h"
 #include "vehicles/vehiclepopulation.h"
-#include "Vfx\Misc\Fire.h"
+#include "Vfx/Misc/Fire.h"
 #include "vfx/particles/PtFxManager.h"
 #include "weapons/inventory/PedInventoryLoadOut.h"
 #include "weapons/projectiles/projectilemanager.h"
