diff --git a/src/dev_ng/game/control/GarageEditor.cpp b/src/dev_ng/game/control/GarageEditor.cpp
--- a/src/dev_ng/game/control/GarageEditor.cpp
+++ b/src/dev_ng/game/control/GarageEditor.cpp
@@ -8,11 +8,11 @@
 /////////////////////////////////////////////////////////////////////////////////
 
 #include "GarageEditor.h"
-#include "Garages.h"
+#include "garages.h"
 #include "garages_parser.h"
-#include "objects/Door.h"
-#include "grcore\debugdraw.h"
-#include "grcore\setup.h"
+#include "Objects/Door.h"
+#include "grcore/debugdraw.h"
+#include "grcore/setup.h"
 
 AI_OPTIMISATIONS()
 
@@ -21,7 +21,7 @@ AI_OPTIMISATIONS()
 #include "camera/CamInterface.h"
 #include "grcore/debugdraw.h"
 #include "fwscene/world/WorldLimits.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "script/script.h"
 #include "game/ModelIndices.h"
 
