diff --git a/src/dev_ng/game/control/Route.h b/src/dev_ng/game/control/Route.h
--- a/src/dev_ng/game/control/Route.h
+++ b/src/dev_ng/game/control/Route.h
@@ -19,7 +19,7 @@
 // Game headers
 #include "vehicleAi/pathFind.h"
 #include "debug/debug.h"
-#include "Task\Animation/TaskAnims.h"
+#include "Task/Animation/TaskAnims.h"
 
 // Used to store dictionary name and anim name from script commands.
 class CAnimNameDescriptor
