diff --git a/src/dev_ng/game/control/trafficlights.cpp b/src/dev_ng/game/control/trafficlights.cpp
--- a/src/dev_ng/game/control/trafficlights.cpp
+++ b/src/dev_ng/game/control/trafficlights.cpp
@@ -37,9 +37,9 @@
 #include "scene/RefMacros.h"
 #include "scene/world/gameWorld.h"
 #include "system/timer.h"
-#include "TimeCycle\TimeCycleConfig.h"
-#include "Vfx\Misc\BrightLights.h"
-#include "Vfx\VfxHelper.h"
+#include "TimeCycle/TimeCycleConfig.h"
+#include "Vfx/Misc/BrightLights.h"
+#include "Vfx/VfxHelper.h"
 #include "physics/WorldProbe/worldprobe.h"
 #include "renderer/lights/lights.h"
 #include "vehicleAi/vehicleintelligence.h"
