diff --git a/src/dev_ng/rage/base/src/string/string.h b/src/dev_ng/rage/base/src/string/string.h
--- a/src/dev_ng/rage/base/src/string/string.h
+++ b/src/dev_ng/rage/base/src/string/string.h
@@ -40,14 +40,14 @@ namespace rage {
 #else
 #define stricmp     strcasecmp
 #define strnicmp  strncasecmp
-# if __PS3 || __PSP2 || RSG_ORBIS
+# if __PS3 || __PSP2 || RSG_ORBIS || __LINUX
 extern "C" char* strupr(char*);
 extern "C" char* strlwr(char*);
 #define strcmpi stricmp
 # endif
 #endif
 
-#ifdef _MSC_VER 
+#if __WIN32 && _MSC_VER < 1900
 #define snprintf _snprintf
 #endif
 
