diff --git a/src/dev_ng/game/frontend/Credits.h b/src/dev_ng/game/frontend/Credits.h
--- a/src/dev_ng/game/frontend/Credits.h
+++ b/src/dev_ng/game/frontend/Credits.h
@@ -15,9 +15,9 @@
 #include "parser/macros.h"
 
 // Game headers
-#include "renderer/Sprite2d.h"
-#include "text\text.h"
-#include "Network\Cloud\CloudManager.h"
+#include "renderer/sprite2d.h"
+#include "text/text.h"
+#include "network/Cloud/CloudManager.h"
 
 enum eCreditType
 {
