diff --git a/src/dev_ng/game/frontend/DisplayCalibration.cpp b/src/dev_ng/game/frontend/DisplayCalibration.cpp
--- a/src/dev_ng/game/frontend/DisplayCalibration.cpp
+++ b/src/dev_ng/game/frontend/DisplayCalibration.cpp
@@ -7,26 +7,26 @@
 //
 /////////////////////////////////////////////////////////////////////////////////
 
-#include "Frontend/DisplayCalibration.h"
+#include "frontend/DisplayCalibration.h"
 
 #include "grprofile/timebars.h"
 
-#include "core/app.h"
+#include "Core/app.h"
 #include "audio/frontendaudioentity.h"
-#include "Frontend/MousePointer.h"
-#include "Frontend/LoadingScreens.h"
-#include "Frontend/PauseMenu.h"
+#include "frontend/MousePointer.h"
+#include "frontend/loadingscreens.h"
+#include "frontend/PauseMenu.h"
 #include "frontend/ProfileSettings.h"
-#include "Frontend/ui_channel.h"
-#include "frontend/scaleform/scaleformmgr.h"
-#include "Frontend/WarningScreen.h"
-#include "Frontend/NewHud.h"
-#include "Renderer/PostProcessFX.h"
-#include "Renderer/Util/ShaderUtil.h"
-#include "Renderer/rendertargets.h"
-#include "System/controlMgr.h"
+#include "frontend/ui_channel.h"
+#include "frontend/Scaleform/ScaleFormMgr.h"
+#include "frontend/WarningScreen.h"
+#include "frontend/NewHud.h"
+#include "renderer/PostProcessFX.h"
+#include "renderer/Util/ShaderUtil.h"
+#include "renderer/rendertargets.h"
+#include "system/controlMgr.h"
 #include "text/TextFile.h"
-#include "Renderer/sprite2d.h"
+#include "renderer/sprite2d.h"
 #include "rline/rlpresence.h"
 #include "system/service.h"
 
@@ -237,7 +237,7 @@ void CDisplayCalibration::Update()
 	{
 		PF_FRAMEINIT_TIMEBARS(0);
 
-#if RSG_PC
+#if !RSG_LINUX
 		CApp::CheckExit();
 #endif
 
