diff --git a/src/dev_ng/game/frontend/GameStream.cpp b/src/dev_ng/game/frontend/GameStream.cpp
--- a/src/dev_ng/game/frontend/GameStream.cpp
+++ b/src/dev_ng/game/frontend/GameStream.cpp
@@ -21,7 +21,7 @@
 #include "frontend/PauseMenu.h"
 #include "frontend/Scaleform/ScaleFormMgr.h"
 #include "frontend/ui_channel.h"
-#include "frontend/Loadingscreens.h"
+#include "frontend/loadingscreens.h"
 #include "frontend/NewHud.h"
 #include "frontend/MiniMap.h"
 #include "frontend/Credits.h"
@@ -30,10 +30,10 @@
 #include "frontend/SocialClubMenu.h"
 #include "frontend/MovieClipText.h"
 #include "renderer/rendertargets.h"
-#include "Cutscene/CutsceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "text/TextFile.h"
 #include "GameStream.h"
-#include "Network/NetworkInterface.h"
+#include "network/NetworkInterface.h"
 #include "system/controlMgr.h"
 
 FRONTEND_OPTIMISATIONS()
