diff --git a/src/dev_ng/game/frontend/HudMarkerRenderer.h b/src/dev_ng/game/frontend/HudMarkerRenderer.h
--- a/src/dev_ng/game/frontend/HudMarkerRenderer.h
+++ b/src/dev_ng/game/frontend/HudMarkerRenderer.h
@@ -13,7 +13,7 @@
 #include "system/criticalsection.h"
 
 #include "frontend/Scaleform/ScaleFormMgr.h"
-#include "Scaleform/Include/GFxPlayer.h"
+#include "scaleform/Include/GFxPlayer.h"
 
 class CHudMarkerRenderer
 {
