diff --git a/src/dev_ng/game/frontend/HudMarkerManager.cpp b/src/dev_ng/game/frontend/HudMarkerManager.cpp
--- a/src/dev_ng/game/frontend/HudMarkerManager.cpp
+++ b/src/dev_ng/game/frontend/HudMarkerManager.cpp
@@ -18,9 +18,9 @@
 #include "frontend/HudTools.h"
 #include "frontend/ui_channel.h"
 
-#include "Camera/Base/BaseCamera.h"
-#include "Camera/CamInterface.h"
-#include "Camera/Viewports/ViewportManager.h"
+#include "camera/Base/BaseCamera.h"
+#include "camera/CamInterface.h"
+#include "camera/Viewports/ViewportManager.h"
 
 #include "scene/world/GameWorld.h"
 #include "script/script_hud.h"
