diff --git a/src/dev_ng/game/frontend/ProfileSettings.cpp b/src/dev_ng/game/frontend/ProfileSettings.cpp
--- a/src/dev_ng/game/frontend/ProfileSettings.cpp
+++ b/src/dev_ng/game/frontend/ProfileSettings.cpp
@@ -5,7 +5,7 @@
 
 // --- Include Files ------------------------------------------------------------
 
-#include "profilesettings.h"
+#include "ProfileSettings.h"
 
 #if __PS3
 #include <np.h>
@@ -40,15 +40,15 @@
 // Game headers
 #include "debug/Debug.h"
 #include "frontend/DisplayCalibration.h"
-#include "Frontend/ui_channel.h"
-#include "network/live/livemanager.h"
-#include "Network/Network.h"
+#include "frontend/ui_channel.h"
+#include "network/Live/livemanager.h"
+#include "network/Network.h"
 #include "SaveLoad/GenericGameStorage.h"
 #include "SaveLoad/savegame_queue.h"
 #include "fwsys/timer.h"
-#include "peds/PedWeapons/PedTargetEvaluator.h"
+#include "Peds/PedWeapons/PedTargetEvaluator.h"
 #include "SaveLoad/savegame_new_game_checks.h"
-#include "stats/StatsInterface.h"
+#include "Stats/StatsInterface.h"
 #include "script/script.h"
 
 // --- Defines ------------------------------------------------------------------
@@ -288,6 +288,8 @@ bool CProfileSettings::StartRead()
 		Warningf("CProfileSettings::StartRead - CGenericGameStorage::PushOnToSavegameQueue failed\n");
 		success = false;
 	}
+#elif RSG_LINUX
+	// TODO: add Linux support
 #elif RSG_PC
 	m_valid = true;
 	m_settings.clear();
@@ -532,6 +534,7 @@ static volatile bool s_bSavePcProfileThreadFailed = false;
 //	DECLARE_THREAD_FUNC
 void s_SavePcProfileThreadFunc(void*)
 {
+#if !RSG_LINUX
 	PF_PUSH_MARKER("SavePcProfile");
 	s_bSavePcProfileThreadFailed = false;
 
@@ -560,6 +563,7 @@ void s_SavePcProfileThreadFunc(void*)
 	{
 		s_bSavePcProfileThreadFailed = true;
 	}
+#endif // !RSG_LINUX
 
 #if !__FINAL
 	unsigned current, maxi;
@@ -809,7 +813,7 @@ void CProfileSettings::UpdateProfileSettings(bool bWait, bool ORBIS_ONLY(ignoreS
 {
 	PF_AUTO_PUSH_TIMEBAR("CProfileSettings UpdateProfileSettings");
 
-#if __WIN32PC && (!__FINAL || defined(MASTER_NO_SCUI))
+#if __WIN32PC && (!__FINAL || defined(MASTER_NO_SCUI)) && !__LINUX
 	if((g_rlPc.IsInitialized() == false))
 	{
 		// for cases where -nosocialclub is used, we won't have a profile signed in
