diff --git a/src/dev_ng/game/renderer/Entities/CutSceneDrawHandler.cpp b/src/dev_ng/game/renderer/Entities/CutSceneDrawHandler.cpp
--- a/src/dev_ng/game/renderer/Entities/CutSceneDrawHandler.cpp
+++ b/src/dev_ng/game/renderer/Entities/CutSceneDrawHandler.cpp
@@ -12,14 +12,14 @@
 // Framework Headers
 
 // Game Headers
-#include "cutscene/cutscene_channel.h"
+#include "Cutscene/cutscene_channel.h"
 #include "modelinfo/PedModelInfo.h"
-#include "Peds/Ped.h"
+#include "Peds/ped.h"
 #include "Peds/rendering/PedVariationStream.h"
 #include "scene/world/GameWorld.h"
 #include "renderer/DrawLists/DrawListProfileStats.h"
 #include "renderer/Entities/PedDrawHandler.h"
-#include "shaders\CustomShaderEffectPed.h"
+#include "shaders/CustomShaderEffectPed.h"
 
 RENDER_OPTIMISATIONS()
 
