diff --git a/src/dev_ng/game/renderer/Entities/ObjectDrawHandler.cpp b/src/dev_ng/game/renderer/Entities/ObjectDrawHandler.cpp
--- a/src/dev_ng/game/renderer/Entities/ObjectDrawHandler.cpp
+++ b/src/dev_ng/game/renderer/Entities/ObjectDrawHandler.cpp
@@ -16,18 +16,18 @@
 // Game Headers
 #include "control/replay/replay.h"
 #include "debug/DebugScene.h"
-#include "objects/object.h"
-#include "objects/ObjectPopulation.h"
+#include "Objects/object.h"
+#include "Objects/ObjectPopulation.h"
 #include "scene/Entity.h"
-#include "peds/Ped.h"
+#include "Peds/ped.h"
 #include "pickups/Pickup.h"
 #include "pickups/Data/PickupData.h"
 #include "renderer/DrawLists/DrawListProfileStats.h"
 #include "renderer/OcclusionQueries.h"
 #include "vfx/vehicleglass/VehicleGlassManager.h"
 #include "fwscene/stores/drawablestore.h"
-#include "Weapons/Weapon.h"
-#include "Weapons/components/WeaponComponent.h"
+#include "weapons/Weapon.h"
+#include "weapons/components/WeaponComponent.h"
 
 RENDER_OPTIMISATIONS()
 
