diff --git a/src/dev_ng/game/network/Objects/Entities/NetObjSubmarine.cpp b/src/dev_ng/game/network/Objects/Entities/NetObjSubmarine.cpp
--- a/src/dev_ng/game/network/Objects/Entities/NetObjSubmarine.cpp
+++ b/src/dev_ng/game/network/Objects/Entities/NetObjSubmarine.cpp
@@ -7,12 +7,12 @@
 
 
 //Network Headers
-#include "network/objects/entities/NetObjSubmarine.h"
-#include "network/objects/entities/NetObjAutomobile.h"
-#include "network/network.h"
+#include "network/Objects/Entities/NetObjSubmarine.h"
+#include "network/Objects/Entities/NetObjAutomobile.h"
+#include "network/Network.h"
 
 //Game Headers
-#include "vehicles/Submarine.h"
+#include "Vehicles/Submarine.h"
 #include "Peds/ped.h"
 #include "scene/world/GameWorld.h"
 
