diff --git a/src/dev_ng/game/network/Objects/Entities/NetObjBike.h b/src/dev_ng/game/network/Objects/Entities/NetObjBike.h
--- a/src/dev_ng/game/network/Objects/Entities/NetObjBike.h
+++ b/src/dev_ng/game/network/Objects/Entities/NetObjBike.h
@@ -8,8 +8,8 @@
 #ifndef NETOBJBIKE_H
 #define NETOBJBIKE_H
 
-#include "network/objects/entities/NetObjVehicle.h"
-#include "network/objects/synchronisation/SyncTrees/ProjectSyncTrees.h"
+#include "network/Objects/Entities/NetObjVehicle.h"
+#include "network/Objects/Synchronisation/SyncTrees/ProjectSyncTrees.h"
 
 class CNetObjBike : public CNetObjVehicle, public IBikeNodeDataAccessor
 {
