diff --git a/src/dev_ng/game/network/Objects/Entities/NetObjAutomobile.h b/src/dev_ng/game/network/Objects/Entities/NetObjAutomobile.h
--- a/src/dev_ng/game/network/Objects/Entities/NetObjAutomobile.h
+++ b/src/dev_ng/game/network/Objects/Entities/NetObjAutomobile.h
@@ -8,8 +8,8 @@
 #ifndef NETOBJAUTOMOBILE_H
 #define NETOBJAUTOMOBILE_H
 
-#include "network/objects/entities/NetObjVehicle.h"
-#include "network/objects/synchronisation/SyncTrees/ProjectSyncTrees.h"
+#include "network/Objects/Entities/NetObjVehicle.h"
+#include "network/Objects/Synchronisation/SyncTrees/ProjectSyncTrees.h"
 
 class CNetObjAutomobile : public CNetObjVehicle, public IAutomobileNodeDataAccessor
 {
