diff --git a/src/dev_ng/game/control/replay/Ped/PedPacket.cpp b/src/dev_ng/game/control/replay/Ped/PedPacket.cpp
--- a/src/dev_ng/game/control/replay/Ped/PedPacket.cpp
+++ b/src/dev_ng/game/control/replay/Ped/PedPacket.cpp
@@ -11,7 +11,7 @@
 #include "control/replay/ReplayInterfacePed.h"
 #include "control/replay/ReplayInterfaceVeh.h"
 #include "control/replay/ReplayInternal.h"
-#include "control\replay\Ped\BonedataMap.h"
+#include "control/replay/Ped/BonedataMap.h"
 
 #include "debug/Debug.h"
 #include "fwscene/stores/drawablestore.h"
