diff --git a/src/dev_ng/game/audio/cutsceneaudioentity.cpp b/src/dev_ng/game/audio/cutsceneaudioentity.cpp
--- a/src/dev_ng/game/audio/cutsceneaudioentity.cpp
+++ b/src/dev_ng/game/audio/cutsceneaudioentity.cpp
@@ -16,7 +16,7 @@
 #include "audiosoundtypes/streamingsound.h"
 #include "audioengine/categorymanager.h"
 #include "camera/CamInterface.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "scriptaudioentity.h"
 
 #include "audioengine/controller.h"
@@ -24,7 +24,7 @@
 #include "audiohardware/driver.h"
 #include "audiohardware/syncsource.h"
 #include "audiohardware/waveplayer.h"
-#include "control/replay/audio/CutSceneAudioPacket.h"
+#include "control/replay/Audio/CutSceneAudioPacket.h"
 #include "system/bootmgr.h"
 
 #if RSG_DURANGO
