diff --git a/src/dev_ng/game/audio/northaudioengine.cpp b/src/dev_ng/game/audio/northaudioengine.cpp
--- a/src/dev_ng/game/audio/northaudioengine.cpp
+++ b/src/dev_ng/game/audio/northaudioengine.cpp
@@ -57,13 +57,13 @@
 #include "camera/viewports/ViewportManager.h"
 #include "control/gamelogic.h"
 #include "control/stuntjump.h"
-#include "core/game.h"
-#include "cutscene/CutSceneManagerNew.h"
-#include "debug/debug.h"
-#include "debug/debugscene.h"
+#include "Core/game.h"
+#include "Cutscene/CutSceneManagerNew.h"
+#include "debug/Debug.h"
+#include "debug/DebugScene.h"
 #include "grcore/debugdraw.h"
-#include "game/clock.h"
-#include "vehicles/Metadata/VehicleSeatInfo.h"
+#include "game/Clock.h"
+#include "Vehicles/Metadata/VehicleSeatInfo.h"
 #include "input/mouse.h"
 #include "scene/DataFileMgr.h"
 #include "scene/FileLoader.h"
@@ -99,15 +99,15 @@
 #include "parser/manager.h"
 #include "profile/profiler.h"
 #include "renderer/PostProcessFX.h"
-#include "renderer/water.h"
+#include "renderer/Water.h"
 #include "scene/playerswitch/PlayerSwitchInterface.h"
 #include "scene/scene.h"
-#include "system/ControlMgr.h"
+#include "system/controlMgr.h"
 #include "system/memory.h"
 #include "system/param.h"
 #include "system/performancetimer.h"
 #include "system/simpleallocator.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "vfx/VisualEffects.h"
 #include "net/nethardware.h"
 #include "frontend/MobilePhone.h"
@@ -120,10 +120,10 @@
 
 #include "grprofile/pix.h"
 
-#include "Network/Commerce/CommerceManager.h"
-#include "Network/Live/livemanager.h"
-#include "Network/Network.h"
-#include "Network/Voice/NetworkVoice.h"
+#include "network/Commerce/CommerceManager.h"
+#include "network/Live/livemanager.h"
+#include "network/Network.h"
+#include "network/Voice/NetworkVoice.h"
 
 #if RSG_ORBIS
 #include "audiohardware/device_orbis.h"
@@ -139,7 +139,7 @@
 #if RSG_PC
 #include "audiohardware/device_xaudio_pc.h"
 
-#if __BANK && RSG_PC
+#if __BANK && RSG_PC && !RSG_LINUX
 bool g_ShowAudioFrameDebugInfo;
 u32 g_NumberOfMixBuffers = rage::kXAudioNumberOfBuffers;
 bool g_OverrideCPULimitedAudioSetting = false;
@@ -753,7 +753,7 @@ void audNorthAudioEngine::GenerateMemoryReport(const char *destFileName)
 
 RAGE_DEFINE_SUBCHANNEL(Audio,NorthAudio)
 
-#if __ASSERT
+#if __USEDEBUGAUDIO
 class audNorthAudioEngineThreadVerifier : public audControllerThreadVerifier
 {
 public:
@@ -902,7 +902,7 @@ bool audNorthAudioEngine::InitClass()
 	}
 #endif
 
-	ASSERT_ONLY(sm_AudioController.SetThreadVerifier(&g_NorthAudioThreadVerifier));
+	DEBUGAUDIO_ONLY(sm_AudioController.SetThreadVerifier(&g_NorthAudioThreadVerifier));
 
 #if __DEV
 	g_AssertOnSoundPoolFull = false;
@@ -1858,7 +1858,7 @@ void audNorthAudioEngine::StartUpdate()
 	
 		bool isMinSpec = (sysIpcGetProcessorCount() <= uMinCores) || (sysTimerConsts::CpuSpeed < cfMinClockSpeed) || forceMinSpec;
 
-#if __BANK
+#if __BANK && !RSG_LINUX
 		if(g_OverrideCPULimitedAudioSetting)
 		{
 			isMinSpec = g_ForceMinSpecMode;
@@ -2926,7 +2926,7 @@ void audNorthAudioEngine::Update()
 {
 	USE_MEMBUCKET(MEMBUCKET_AUDIO);
 
-#if RSG_PC && __BANK
+#if RSG_PC && __BANK && !RSG_LINUX
 	audMixerDeviceXAudio2::SetNumberOfAudioMixBuffers(g_NumberOfMixBuffers); 
 #endif
 
@@ -5334,7 +5334,7 @@ void audNorthAudioEngine::CreateWidgets()
 
 	bank.AddToggle("g_UseInteriorCarFilter", &g_UseInteriorCarFilter);
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 	bank.AddToggle("Show Audio Frame Debug Info", &g_ShowAudioFrameDebugInfo);
 	bank.AddSlider("Number of mix buffers", &g_NumberOfMixBuffers, 8, 16, 1);
 
@@ -5552,6 +5552,7 @@ void audNorthAudioEngine::DebugDrawSlowMo()
 
 	drawMgr.PopSection();
 
+#if __USEDEBUGAUDIO
 	formatf(sectionHeader, "Current SlowMo Mode");
 	drawMgr.PushSection(sectionHeader);
 	formatf(sectionHeader, "%s: %s : %s", SlowMoType_ToString(sm_SlowMoMode), sm_SlowMoScene ? DYNAMICMIXMGR.GetMetadataManager().GetObjectNameFromNameTableOffset(sm_SlowMoScene->GetSceneSettings()->NameTableOffset): NULL, sm_SlowMoSound ? g_AudioEngine.GetSoundManager().GetFactory().GetMetadataManager().GetObjectNameFromNameTableOffset(sm_SlowMoSound->GetNameTableOffset()): NULL);
@@ -5559,6 +5560,7 @@ void audNorthAudioEngine::DebugDrawSlowMo()
 	drawMgr.DrawLine(sectionHeader);
 
 	drawMgr.PopSection();
+#endif
 	
 	grcFont::GetCurrent().SetInternalColor( prevColor );
 	grcLighting(bOldLighting);
