diff --git a/src/dev_ng/game/audio/streamslot.h b/src/dev_ng/game/audio/streamslot.h
--- a/src/dev_ng/game/audio/streamslot.h
+++ b/src/dev_ng/game/audio/streamslot.h
@@ -7,7 +7,7 @@
 #ifndef AUD_STREAMSLOT_H
 #define AUD_STREAMSLOT_H
 
-#include "Control/Replay/ReplaySettings.h"
+#include "control/replay/ReplaySettings.h"
 
 const u32 g_NumStreamSlots = 5;
 const u32 g_NumVirtualStreamSlots = 2;
