diff --git a/src/dev_ng/game/audio/vehiclefireaudio.cpp b/src/dev_ng/game/audio/vehiclefireaudio.cpp
--- a/src/dev_ng/game/audio/vehiclefireaudio.cpp
+++ b/src/dev_ng/game/audio/vehiclefireaudio.cpp
@@ -15,8 +15,8 @@
 #include "audiohardware/driverutil.h"
 #include "audioengine/curve.h"
 #include "audioengine/engineutil.h"
-#include "vehicles/vehicle.h"
-#include "vfx\misc/fire.h"
+#include "Vehicles/vehicle.h"
+#include "vfx/misc/Fire.h"
 #include "frontendaudioentity.h"
 
 #include "debugaudio.h"
