diff --git a/src/dev_ng/game/renderer/DrawLists/drawList.cpp b/src/dev_ng/game/renderer/DrawLists/drawList.cpp
--- a/src/dev_ng/game/renderer/DrawLists/drawList.cpp
+++ b/src/dev_ng/game/renderer/DrawLists/drawList.cpp
@@ -50,7 +50,7 @@
 #include "physics/breakable.h"
 #include "pickups/PickupManager.h"
 #include "vehicles/vehicle.h"
-#include "renderer\Util\ShaderUtil.h"
+#include "renderer/Util/ShaderUtil.h"
 #include "renderer/Deferred/DeferredLighting.h"
 #include "renderer/DrawLists/drawListMgr.h"
 #include "renderer/DrawLists/DrawListProfileStats.h"
