diff --git a/src/dev_ng/game/renderer/SpuPM/SpuPmMgr.h b/src/dev_ng/game/renderer/SpuPM/SpuPmMgr.h
--- a/src/dev_ng/game/renderer/SpuPM/SpuPmMgr.h
+++ b/src/dev_ng/game/renderer/SpuPM/SpuPmMgr.h
@@ -30,7 +30,7 @@
 	#include "renderer/DrawLists/drawListMgr.h"
 	#define CSPULIGHTCLASSIFICATIONINFO_DEF_STRUCT
 		#include "renderer/Lights/TiledLightingSettings.h"
-	#include "renderer/lights/LightSource.h"
+	#include "renderer/Lights/LightSource.h"
 	#include "renderer/Deferred/DeferredConfig.h"
 #endif
 
@@ -316,7 +316,7 @@ struct CWBAllocBlockHeader
 };
 
 #if __PPU
-#include "grcore\texturegcm.h"
+#include "grcore/texturegcm.h"
 
 //
 //
@@ -353,8 +353,8 @@ private:
 
 #if __PPU
 
-#include "templates\LinkList.h"
-#include "system\simpleallocator.h"
+#include "templates/LinkList.h"
+#include "system/simpleallocator.h"
 
 typedef CLink<CSPUPostProcessDriverInfo>	CSPUPostProcessDriverInfoLink;
 typedef CLink<CWBAllocBlockHeader>			CWBAllocBlockHeaderLink;
