diff --git a/src/dev_ng/game/network/Live/FriendClanData.h b/src/dev_ng/game/network/Live/FriendClanData.h
--- a/src/dev_ng/game/network/Live/FriendClanData.h
+++ b/src/dev_ng/game/network/Live/FriendClanData.h
@@ -8,13 +8,13 @@
 #define FRIEND_CLAN_DATA
 
 // rage
-#include "rline\clan\rlclancommon.h"
+#include "rline/clan/rlclancommon.h"
 #include "rline/rlfriendsmanager.h"
-#include "data\base.h"
+#include "data/base.h"
 
 // game
-#include "Frontend/CCrewMenu.h" // for VISIBLE_CREW_ITEMS
-#include "Network/Live/ClanNetStatusArray.h"
+#include "frontend/CCrewMenu.h" // for VISIBLE_CREW_ITEMS
+#include "network/Live/ClanNetStatusArray.h"
 
 class CGroupClanData: public datBase
 {
