diff --git a/src/dev_ng/game/network/Live/livemanager.h b/src/dev_ng/game/network/Live/livemanager.h
--- a/src/dev_ng/game/network/Live/livemanager.h
+++ b/src/dev_ng/game/network/Live/livemanager.h
@@ -21,18 +21,18 @@
 #include "rline/scmembership/rlscmembership.h"
 #include "system/service.h"
 
-#include "fwnet/netscgamerdatamgr.h"
+#include "fwnet/netscgamerDataMgr.h"
 
-#include "Core/Game.h"
-#include "network/commerce/CommerceManager.h"
-#include "network/crews/NetworkCrewDataMgr.h"
+#include "Core/game.h"
+#include "network/Commerce/CommerceManager.h"
+#include "network/Crews/NetworkCrewDataMgr.h"
 #include "network/Facebook/Facebook.h"
 #include "network/Live/AchMgr.h"
 #include "network/Live/InviteMgr.h"
 #include "network/Live/SocialClubMgr.h"
 #include "network/Party/NetworkPartyInvites.h"
 
-#define ENABLE_SINGLE_PLAYER_CLOUD_SAVES (RSG_PC && 1)
+#define ENABLE_SINGLE_PLAYER_CLOUD_SAVES (RSG_PC && !RSG_LINUX)
 #if ENABLE_SINGLE_PLAYER_CLOUD_SAVES
 #include "rline/rlpc.h"
 #endif
