diff --git a/src/dev_ng/game/network/Live/livemanager.cpp b/src/dev_ng/game/network/Live/livemanager.cpp
--- a/src/dev_ng/game/network/Live/livemanager.cpp
+++ b/src/dev_ng/game/network/Live/livemanager.cpp
@@ -5,7 +5,7 @@
 //
 
 #include <algorithm>
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 #include <tchar.h>
 #endif
 
@@ -58,36 +58,35 @@
 #include "livep2pkeys.h"
 #include "frontend/CFriendsMenu.h"
 #include "frontend/Store/StoreScreenMgr.h"
-#include "Network/Network.h"
-#include "Network/xlast/Fuzzy.schema.h"
+#include "network/Network.h"
+#include "network/xlast/Fuzzy.schema.h"
 #include "network/xlast/presenceutil.h"
-#include "Network/NetworkInterface.h"
-#include "Network/Live/NetworkRichPresence.h"
-#include "Network/Sessions/NetworkSession.h"
-#include "Network/Sessions/NetworkVoiceSession.h"
-#include "Network/Live/NetworkClan.h"
-#include "Network/Live/PlayerCardDataManager.h"
-#include "Network/xlast/presenceutil.h"
-#include "Network/live/NetworkProfileStats.h"
-#include "Network/live/NetworkSCPresenceUtil.h"
-#include "Network/Shop/GameTransactionsSessionMgr.h"
-#include "Network/Shop/NetworkShopping.h"
-#include "Network/Shop/Catalog.h"
-#include "Network/SocialClubServices/GamePresenceEventDispatcher.h"
-#include "Network/SocialClubServices/GamePresenceEvents.h"
-#include "Network/SocialClubServices/SocialClubCommunityEventMgr.h"
-#include "Network/SocialClubServices/SocialClubEmailMgr.h"
-#include "Network/SocialClubServices/SocialClubFeedMgr.h"
-#include "Network/SocialClubServices/SocialClubFeedTilesMgr.h"
-#include "Network/SocialClubServices/SocialClubNewsStoryMgr.h"
-#include "Network/SocialClubServices/SocialClubInboxManager.h"
-#include "Network/Cloud/CloudManager.h"
+#include "network/NetworkInterface.h"
+#include "network/Live/NetworkRichPresence.h"
+#include "network/Sessions/NetworkSession.h"
+#include "network/Sessions/NetworkVoiceSession.h"
+#include "network/Live/NetworkClan.h"
+#include "network/Live/PlayerCardDataManager.h"
+#include "network/Live/NetworkProfileStats.h"
+#include "network/Live/NetworkSCPresenceUtil.h"
+#include "network/Shop/GameTransactionsSessionMgr.h"
+#include "network/Shop/NetworkShopping.h"
+#include "network/Shop/Catalog.h"
+#include "network/SocialClubServices/GamePresenceEventDispatcher.h"
+#include "network/SocialClubServices/GamePresenceEvents.h"
+#include "network/SocialClubServices/SocialClubCommunityEventMgr.h"
+#include "network/SocialClubServices/SocialClubEmailMgr.h"
+#include "network/SocialClubServices/SocialClubFeedMgr.h"
+#include "network/SocialClubServices/SocialClubFeedTilesMgr.h"
+#include "network/SocialClubServices/SocialClubNewsStoryMgr.h"
+#include "network/SocialClubServices/SocialClubInboxManager.h"
+#include "network/Cloud/CloudManager.h"
 #include "network/Live/NetworkLegalRestrictionsManager.h"
 #include "network/Cloud/VideoUploadManager.h"
-#include "Network/Voice/NetworkVoice.h"
+#include "network/Voice/NetworkVoice.h"
 
 #if RSG_GEN9
-#include "Network/live/NetworkLandingPageStatsMgr.h"
+#include "network/Live/NetworkLandingPageStatsMgr.h"
 #endif
 
 // Game Headers
@@ -98,17 +97,17 @@
 #include "Peds/PlayerInfo.h"
 #include "SaveLoad/GenericGameStorage.h"
 #include "SaveLoad/savegame_autoload.h"
-#include "Scene/ExtraContent.h"
+#include "scene/ExtraContent.h"
 #include "scene/FocusEntity.h"
-#include "Scene/world/gameWorld.h"
+#include "scene/world/GameWorld.h"
 #include "streaming/streamingengine.h"
-#include "Control/gamelogic.h"
-#include "System/controlMgr.h"
+#include "control/gamelogic.h"
+#include "system/controlMgr.h"
 #include "system/pad.h"
 #include "Cutscene/CutSceneManagerNew.h"
-#include "Frontend/PauseMenu.h"
-#include "Frontend/WarningScreen.h"
-#include "Frontend/SocialClubMenu.h"
+#include "frontend/PauseMenu.h"
+#include "frontend/WarningScreen.h"
+#include "frontend/SocialClubMenu.h"
 #include "system/memvisualize.h"
 #include "system/EntitlementManager.h"
 #include "frontend/ProfileSettings.h"
@@ -139,19 +138,19 @@ namespace rage
 #include "frontend/TextInputBox.h"
 #endif // RSG_PC
 
-#include "Network/Live/NetworkTelemetry.h"
+#include "network/Live/NetworkTelemetry.h"
 #include "Stats/StatsMgr.h"
 #include "Stats/StatsDataMgr.h"
 #include "Stats/StatsInterface.h"
 #include "text/messages.h"
 #include "game/config.h"
 
-#include "Network/Cloud/CloudManager.h"
-#include "Network/Cloud/VideoUploadManager.h"
-#include "Network/Cloud/UserContentManager.h"
+#include "network/Cloud/CloudManager.h"
+#include "network/Cloud/VideoUploadManager.h"
+#include "network/Cloud/UserContentManager.h"
 
-#include "Network/Live/NetworkGamerAttributes.h"
-#include "Network/Shop/NetworkGameTransactions.h"
+#include "network/Live/NetworkGamerAttributes.h"
+#include "network/Shop/NetworkGameTransactions.h"
 
 NETWORK_OPTIMISATIONS()
 
@@ -467,7 +466,7 @@ bool CLiveManager::Init(const unsigned initMode)
 #endif
 		s_RlineHeap       = rage_new u8[nRlineHeapSize];
 
-#if !__FINAL && (RSG_PC || RSG_DURANGO)
+#if !__FINAL && (RSG_PC || RSG_DURANGO) && !RSG_LINUX
 		if(PARAM_rlineSparseAllocator.Get())
 		{
 			gnetDebug1("-rlineSparseAllocator enabled, using a sparse allocator for rline allocations");
@@ -822,7 +821,7 @@ bool CLiveManager::Init(const unsigned initMode)
 		}
 
 		// Language needs to be set before rlInit
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 		// -uilanguage can override the system language
 		int language = LANGUAGE_UNDEFINED;
 
@@ -873,7 +872,7 @@ bool CLiveManager::Init(const unsigned initMode)
 		
 		rlInit(sm_RlineAllocator, sm_SocketMgr->GetMainSocket(), &titleId, MIN_AGE_RATING);
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 		// PC Downloader Pipe
 		g_rlPc.ConnectDownloaderPipe("\\\\.\\pipe\\GTAVLauncher_Pipe");
 #endif
@@ -3040,7 +3039,7 @@ CLiveManager::ShowSigninUi(unsigned nSignInFlags)
 {
 	gnetAssert(sm_IsInitialized);
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 	if (IsSignedIn())
 	{
 		gnetWarning("Attempting to show signin UI while already signed in");
@@ -5887,7 +5886,7 @@ static void NetworkBank_GetFriendPage()
 }
 #endif
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 static void NetworkBank_PipeDownloadMessage()
 {
 	g_rlPc.GetDownloaderPipe()->StartDownloadWithToken("TestPlatform", "TestMessage");
@@ -6537,7 +6536,7 @@ CLiveManager::InitWidgets()
 			sysMemSet(s_AchLabels[i], 0, ACHIEVEMENT_LABEL);
 		}
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 		pBank->PushGroup("PC Pipe");
 		pBank->AddButton("DOWN: Message", datCallback(NetworkBank_PipeDownloadMessage));
 		pBank->AddText("Download URL", s_DownloadUrl, 511, false);
