diff --git a/src/dev_ng/game/network/Live/NetworkGamerAttributes.cpp b/src/dev_ng/game/network/Live/NetworkGamerAttributes.cpp
--- a/src/dev_ng/game/network/Live/NetworkGamerAttributes.cpp
+++ b/src/dev_ng/game/network/Live/NetworkGamerAttributes.cpp
@@ -8,7 +8,7 @@
 #include "fwnet/netchannel.h"
 #include "optimisations.h"
 #include "NetworkGamerAttributes.h"
-#include "Network/NetworkInterface.h"
+#include "network/NetworkInterface.h"
 
 NETWORK_OPTIMISATIONS()
 
