diff --git a/src/dev_ng/game/vfx/gpuparticles/PtFxGPUManager.cpp b/src/dev_ng/game/vfx/gpuparticles/PtFxGPUManager.cpp
--- a/src/dev_ng/game/vfx/gpuparticles/PtFxGPUManager.cpp
+++ b/src/dev_ng/game/vfx/gpuparticles/PtFxGPUManager.cpp
@@ -24,25 +24,25 @@
 #include "vfx/vfxwidget.h"
 
 // game
-#include "Camera/CamInterface.h"
-#include "Camera/gameplay/GameplayDirector.h"
+#include "camera/CamInterface.h"
+#include "camera/gameplay/GameplayDirector.h"
 #include "Cutscene/CutSceneManagerNew.h"
-#include "Game/Weather.h"
+#include "game/Weather.h"
 #include "Peds/ped.h"
 #include "Peds/PopCycle.h"
 #include "Peds/PlayerInfo.h"
 #include "Peds/PlayerSpecialAbility.h"
-#include "Renderer/Lights/Lights.h"
-#include "Renderer/PostProcessFX.h"
-#include "Renderer/RenderPhases/RenderPhaseCascadeShadows.h"
-#include "Scene/Portals/PortalTracker.h"
-#include "Scene/World/GameWorld.h"
-#include "TimeCycle/TimeCycleConfig.h"
+#include "renderer/Lights/Lights.h"
+#include "renderer/PostProcessFX.h"
+#include "renderer/RenderPhases/RenderPhaseCascadeShadows.h"
+#include "scene/portals/PortalTracker.h"
+#include "scene/world/GameWorld.h"
+#include "timecycle/TimeCycleConfig.h"
 #include "vfx/vfx_shared.h"
-#include "Vfx/Metadata/VfxRegionInfo.h"
+#include "vfx/metadata/VfxRegionInfo.h"
 #include "vfx/particles/PtFxManager.h"
-#include "Vfx/Systems/VfxWeather.h"
-#include "Vfx/VfxHelper.h"
+#include "vfx/systems/VfxWeather.h"
+#include "vfx/VfxHelper.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
