diff --git a/src/dev_ng/game/task/Scenario/info/ScenarioActionManager.h b/src/dev_ng/game/task/Scenario/info/ScenarioActionManager.h
--- a/src/dev_ng/game/task/Scenario/info/ScenarioActionManager.h
+++ b/src/dev_ng/game/task/Scenario/info/ScenarioActionManager.h
@@ -2,10 +2,10 @@
 #define SCENARIO_ACTION_MANAGER_H
 
 // Rage headers
-#include "Data/Base.h"
+#include "data/base.h"
 
 // Game headers
-#include "Task/Scenario/Info/ScenarioAction.h"
+#include "task/Scenario/info/ScenarioAction.h"
 
 
 ////////////////////////////////////////////////////////////////////////////////
