diff --git a/src/dev_ng/game/weapons/info/VehicleWeaponInfo.cpp b/src/dev_ng/game/weapons/info/VehicleWeaponInfo.cpp
--- a/src/dev_ng/game/weapons/info/VehicleWeaponInfo.cpp
+++ b/src/dev_ng/game/weapons/info/VehicleWeaponInfo.cpp
@@ -5,10 +5,10 @@
 //
 
 // File header
-#include "Weapons/Info/VehicleWeaponInfo.h"
+#include "weapons/info/VehicleWeaponInfo.h"
 
 // Parser headers
-#include "VehicleWeaponInfo_parser.h"
+#include "vehicleweaponinfo_parser.h"
 
 // Macro to disable optimisations if set
 WEAPON_OPTIMISATIONS()
