diff --git a/src/dev_ng/rage/scaleform/Include/GFxPlayer.h b/src/dev_ng/rage/scaleform/Include/GFxPlayer.h
index 9ac959bfc..74f5c3f37 100644
--- a/src/dev_ng/rage/scaleform/Include/GFxPlayer.h
+++ b/src/dev_ng/rage/scaleform/Include/GFxPlayer.h
@@ -30,7 +30,7 @@ THE WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR ANY PURPOSE.
 
 #include "GFxPlayerStats.h"
 
-#include "vector\color32.h"
+#include "vector/color32.h"
 
 #include <ctype.h>  // for wchar_t
 #include <stdarg.h> // for va_list args
