diff --git a/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/BikeSyncNodes.h b/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/BikeSyncNodes.h
--- a/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/BikeSyncNodes.h
+++ b/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/BikeSyncNodes.h
@@ -7,7 +7,7 @@
 #ifndef BIKE_SYNC_NODES_H
 #define BIKE_SYNC_NODES_H
 
-#include "network/objects/synchronisation/SyncNodes/VehicleSyncNodes.h"
+#include "network/Objects/Synchronisation/SyncNodes/VehicleSyncNodes.h"
 
 ///////////////////////////////////////////////////////////////////////////////////////
 // IBikeNodeDataAccessor
