diff --git a/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/DynamicEntitySyncNodes.cpp b/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/DynamicEntitySyncNodes.cpp
--- a/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/DynamicEntitySyncNodes.cpp
+++ b/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/DynamicEntitySyncNodes.cpp
@@ -4,7 +4,7 @@
 // written by:    John Gurney
 //
 
-#include "network/objects/synchronisation/syncnodes/DynamicEntitySyncNodes.h"
+#include "network/Objects/Synchronisation/SyncNodes/DynamicEntitySyncNodes.h"
 #include "scene/world/GameWorld.h"
 
 NETWORK_OPTIMISATIONS()
