diff --git a/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/EntitySyncNodes.h b/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/EntitySyncNodes.h
--- a/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/EntitySyncNodes.h
+++ b/src/dev_ng/game/network/Objects/Synchronisation/SyncNodes/EntitySyncNodes.h
@@ -7,9 +7,9 @@
 #ifndef ENTITY_SYNC_NODES_H
 #define ENTITY_SYNC_NODES_H
 
-#include "network/objects/synchronisation/SyncNodes/BaseSyncNodes.h"
+#include "network/Objects/Synchronisation/SyncNodes/BaseSyncNodes.h"
 #include "fwnet/netobject.h"
-#include "script/handlers/GameScriptIds.h"
+#include "script/Handlers/GameScriptIds.h"
 
 class CEntityScriptGameStateDataNode;
 class CEntityOrientationDataNode;
