diff --git a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkScenarioBlockingAreaWorldState.h b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkScenarioBlockingAreaWorldState.h
--- a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkScenarioBlockingAreaWorldState.h
+++ b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkScenarioBlockingAreaWorldState.h
@@ -7,9 +7,9 @@
 #ifndef NETWORK_SCENARIO_BLOCKING_AREA_WORLD_STATE_H
 #define NETWORK_SCENARIO_BLOCKING_AREA_WORLD_STATE_H
 
-#include "fwtl/pool.h"
-#include "network/general/scriptworldstate/NetworkScriptWorldStateManager.h"
-#include "network/general/scriptworldstate/NetworkScriptWorldStateTypes.h"
+#include "fwtl/Pool.h"
+#include "network/General/ScriptWorldState/NetworkScriptWorldStateManager.h"
+#include "network/General/ScriptWorldState/NetworkScriptWorldStateTypes.h"
 
 //PURPOSE
 // Describes a region of the map in which scenarios have been blocked by a script
