diff --git a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkEntityAreaWorldState.cpp b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkEntityAreaWorldState.cpp
--- a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkEntityAreaWorldState.cpp
+++ b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkEntityAreaWorldState.cpp
@@ -9,14 +9,14 @@
 #include "bank/bkmgr.h"
 #include "bank/bank.h"
 #include "vector/geometry.h"
-#include "phbound/BoundBox.h"
+#include "phbound/boundbox.h"
 
-#include "fwnet/NetLogUtils.h"
+#include "fwnet/netlogutils.h"
 
-#include "Network/NetworkInterface.h"
-#include "Network/Events/NetworkEventTypes.h"
-#include "peds/Ped.h"
-#include "Script/Script.h"
+#include "network/NetworkInterface.h"
+#include "network/Events/NetworkEventTypes.h"
+#include "Peds/ped.h"
+#include "script/script.h"
 
 NETWORK_OPTIMISATIONS()
 
