diff --git a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkRoadNodeWorldState.cpp b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkRoadNodeWorldState.cpp
--- a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkRoadNodeWorldState.cpp
+++ b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkRoadNodeWorldState.cpp
@@ -9,11 +9,11 @@
 #include "bank/bank.h"
 #include "vector/geometry.h"
 
-#include "fwnet/NetLogUtils.h"
+#include "fwnet/netlogutils.h"
 
-#include "Network/NetworkInterface.h"
-#include "peds/Ped.h"
-#include "VehicleAI/PathFind.h"
+#include "network/NetworkInterface.h"
+#include "Peds/ped.h"
+#include "vehicleAi/pathfind.h"
 
 NETWORK_OPTIMISATIONS()
 
