diff --git a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkVehiclePlayerLockingWorldState.h b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkVehiclePlayerLockingWorldState.h
--- a/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkVehiclePlayerLockingWorldState.h
+++ b/src/dev_ng/game/network/General/ScriptWorldState/WorldStates/NetworkVehiclePlayerLockingWorldState.h
@@ -7,9 +7,9 @@
 #ifndef NETWORK_VEHICLE_PLAYER_LOCKING_WORLD_STATE_H
 #define NETWORK_VEHICLE_PLAYER_LOCKING_WORLD_STATE_H
 
-#include "fwtl/pool.h"
-#include "network/general/scriptworldstate/NetworkScriptWorldStateManager.h"
-#include "network/general/scriptworldstate/NetworkScriptWorldStateTypes.h"
+#include "fwtl/Pool.h"
+#include "network/General/ScriptWorldState/NetworkScriptWorldStateManager.h"
+#include "network/General/ScriptWorldState/NetworkScriptWorldStateTypes.h"
 
 //PURPOSE
 // Describes which vehicles and which vehicle instances are locked to which players...
