diff --git a/src/dev_ng/rage/framework/src/vfx/decal/decalbucket.h b/src/dev_ng/rage/framework/src/vfx/decal/decalbucket.h
--- a/src/dev_ng/rage/framework/src/vfx/decal/decalbucket.h
+++ b/src/dev_ng/rage/framework/src/vfx/decal/decalbucket.h
@@ -31,7 +31,7 @@ namespace rage {
 // Not using the MULTIPLE_RENDER_THREADS define here, since it can be switched
 // off, instead DECAL_MULTI_THREADED_RENDERING is true for platforms that will
 // generally support MULTIPLE_RENDER_THREADS.
-#define DECAL_MULTI_THREADED_RENDERING          (RSG_DURANGO || RSG_ORBIS || (RSG_PC && __D3D11))
+#define DECAL_MULTI_THREADED_RENDERING          (RSG_DURANGO || RSG_ORBIS || ((RSG_PC || RSG_LINUX) && __D3D11))
 
 #if DECAL_MULTI_THREADED_RENDERING
 #	define DECAL_MULTI_THREADED_RENDERING_ONLY(...)     __VA_ARGS__
