diff --git a/src/dev_ng/rage/framework/src/vfx/decal/decalmanager.h b/src/dev_ng/rage/framework/src/vfx/decal/decalmanager.h
--- a/src/dev_ng/rage/framework/src/vfx/decal/decalmanager.h
+++ b/src/dev_ng/rage/framework/src/vfx/decal/decalmanager.h
@@ -74,7 +74,7 @@ namespace rage {
 //
 #if RSG_DURANGO || RSG_ORBIS
 	typedef u8      decalCbufIdx;
-#elif RSG_PC
+#elif RSG_PC || RSG_LINUX
 	typedef u32     decalCbufIdx;
 #endif
 
@@ -489,7 +489,7 @@ class decalManager : public decalManagerAsync
 			unsigned                     m_staticIdxBufLastAdded[NUM_PER_CONTEXT_BUFFERS];
 			decalCbufIdx                *m_staticConstBufIdxData[NUM_PER_CONTEXT_BUFFERS];
 
-#		elif RSG_PC && __D3D11
+#		elif (RSG_PC || RSG_LINUX) && __D3D11
 			// Vertex declarations
 			grcVertexDeclaration        *m_staticVtxDecl;
 #			if DECAL_STORE_UNCOMPRESSED_VERTICES
