diff --git a/src/dev_ng/game/animation/AnimScene/AnimScene.cpp b/src/dev_ng/game/animation/AnimScene/AnimScene.cpp
--- a/src/dev_ng/game/animation/AnimScene/AnimScene.cpp
+++ b/src/dev_ng/game/animation/AnimScene/AnimScene.cpp
@@ -17,13 +17,13 @@
 #include "animation/AnimScene/events/AnimScenePlayAnimEvent.h"
 #include "animation/AnimScene/AnimSceneEntityIterator.h"
 #include "animation/AnimScene/AnimSceneEventIterator.h"
-#include "peds/ped.h"
-#include "Peds\PedIntelligence.h"
+#include "Peds/ped.h"
+#include "Peds/PedIntelligence.h"
 #include "task/Animation/TaskAnims.h"
-#include "task\Scenario\Types\TaskUseScenario.h"
-#include "vehicles/vehicle.h"
-#include "vehicleAi\VehicleIntelligence.h"
-#include "objects/object.h"
+#include "task/Scenario/Types/TaskUseScenario.h"
+#include "Vehicles/vehicle.h"
+#include "vehicleAi/VehicleIntelligence.h"
+#include "Objects/object.h"
 #if __BANK
 #include "camera/base/BaseCamera.h"
 #include "camera/system/CameraManager.h"
