diff --git a/src/dev_ng/game/task/Helpers/PavementFloodFillHelper.h b/src/dev_ng/game/task/Helpers/PavementFloodFillHelper.h
--- a/src/dev_ng/game/task/Helpers/PavementFloodFillHelper.h
+++ b/src/dev_ng/game/task/Helpers/PavementFloodFillHelper.h
@@ -1,7 +1,7 @@
 #ifndef _PAVEMENT_FLOODFILL_HELPER_H
 #define _PAVEMENT_FLOODFILL_HELPER_H
 
-#include "Pathserver/PathServer.h"
+#include "pathserver/PathServer.h"
 
 class CPed;
 
