diff --git a/src/dev_ng/game/modelinfo/CompEntityModelInfo.cpp b/src/dev_ng/game/modelinfo/CompEntityModelInfo.cpp
--- a/src/dev_ng/game/modelinfo/CompEntityModelInfo.cpp
+++ b/src/dev_ng/game/modelinfo/CompEntityModelInfo.cpp
@@ -8,7 +8,7 @@
 // C headers
 // Rage headers
 // Game headers
-#include "modelInfo\CompEntityModelInfo.h"
+#include "modelinfo/CompEntityModelInfo.h"
 
 // --- Defines ------------------------------------------------------------------
 
