diff --git a/src/dev_ng/game/modelinfo/MloModelInfo.cpp b/src/dev_ng/game/modelinfo/MloModelInfo.cpp
--- a/src/dev_ng/game/modelinfo/MloModelInfo.cpp
+++ b/src/dev_ng/game/modelinfo/MloModelInfo.cpp
@@ -14,11 +14,11 @@
 
 // Game headers
 #include "audio/northaudioengine.h"
-#include "modelinfo/mlomodelinfo.h"
-#include "scene/loader/mapdata_entities.h"
-#include "scene/loader/mapdata_interiors.h"
-#include "scene/portals/interiorinst.h"
-#include "scene/world/gameworld.h"
+#include "modelinfo/MloModelInfo.h"
+#include "scene/loader/MapData_Entities.h"
+#include "scene/loader/MapData_Interiors.h"
+#include "scene/portals/InteriorInst.h"
+#include "scene/world/GameWorld.h"
 #include "streaming/streaming.h"
 
 #include "diag/art_channel.h"
