diff --git a/src/dev_ng/game/modelinfo/ModelSeatInfo.h b/src/dev_ng/game/modelinfo/ModelSeatInfo.h
--- a/src/dev_ng/game/modelinfo/ModelSeatInfo.h
+++ b/src/dev_ng/game/modelinfo/ModelSeatInfo.h
@@ -7,14 +7,14 @@
 #ifndef INC_SEAT_MODELINFO_H_
 #define INC_SEAT_MODELINFO_H_
 
-#include "ai\debug\types\VehicleDebugInfo.h"
-#include "Animation/AnimBones.h"
+#include "ai/debug/types/VehicleDebugInfo.h"
+#include "animation/AnimBones.h"
 #include "modelinfo/modelinfo_channel.h"
 #include "scene/RegdRefTypes.h"
 #include "vector/quaternion.h"
 #include "Vehicles/Metadata/VehicleDebug.h"
 #include "Vehicles/Metadata/VehicleEnterExitFlags.h"
-#include "vehicles/VehicleDefines.h"
+#include "Vehicles/VehicleDefines.h"
 
 //forward definitions
 
