diff --git a/src/dev_ng/game/modelinfo/VehicleModelInfo.cpp b/src/dev_ng/game/modelinfo/VehicleModelInfo.cpp
--- a/src/dev_ng/game/modelinfo/VehicleModelInfo.cpp
+++ b/src/dev_ng/game/modelinfo/VehicleModelInfo.cpp
@@ -17,15 +17,15 @@
 #include "crskeleton/skeletondata.h"
 #include "diag/art_channel.h"
 #include "file/default_paths.h"
-#include "fragment/typeGroup.h"
+#include "fragment/typegroup.h"
 #include "parser/manager.h"
 #include "parser/psofile.h"
 #include "parser/restparserservices.h"
-#include "phbound/BoundBox.h"
-#include "phbound/BoundCapsule.h"
-#include "phbound/BoundComposite.h"
+#include "phbound/boundbox.h"
+#include "phbound/boundcapsule.h"
+#include "phbound/boundcomposite.h"
 #include "phbound/boundbvh.h"
-#include "phBound/support.h"
+#include "phbound/support.h"
 #include "system/memory.h"
 #include "system/nelem.h"
 
@@ -47,15 +47,15 @@
 
 // Game headers
 #include "audio/radioaudioentity.h"
-#include "core/game.h"
+#include "Core/game.h"
 #include "debug/DebugScene.h"
 #include "frontend/Scaleform/ScaleFormStore.h"
 #include "frontend/NewHud.h"
 #include "frontend/UIReticule.h"
 #include "game/config.h"
-#include "game/modelindices.h"
-#include "ModelInfo/ModelInfo.h"
-#include "ModelInfo/ModelInfo_Factories.h"
+#include "game/ModelIndices.h"
+#include "modelinfo/ModelInfo.h"
+#include "modelinfo/ModelInfo_Factories.h"
 #include "modelinfo/VehicleModelInfoColors.h"
 #include "modelinfo/VehicleModelInfoExtensions.h"
 #include "modelinfo/VehicleModelInfoPlates.h"
@@ -63,11 +63,11 @@
 #include "modelinfo/VehicleModelInfoVariation.h"
 #include "modelinfo/VehicleModelInfo.h"
 #include "modelinfo/VehicleModelInfoData_parser.h"
-#include "objects/CoverTuning.h"
+#include "Objects/CoverTuning.h"
 #include "pathserver/ExportCollision.h"
 #include "pathserver/NavGenParam.h"
-#include "physics/gtainst.h"
-#include "physics/floater.h"
+#include "physics/gtaInst.h"
+#include "physics/Floater.h"
 #include "renderer/HierarchyIds.h"
 #include "renderer/RenderPhases/RenderPhaseReflection.h"
 #include "renderer/DrawLists/drawList.h"
@@ -80,28 +80,28 @@
 #include "streaming/streaming.h"
 #include "streaming/streamingvisualize.h"
 #include "system/FileMgr.h"
-#include "Task/Vehicle/TaskCarUtils.h"
-#include "Task/Vehicle/TaskInVehicle.h"
-#include "Task/Vehicle/TaskEnterVehicle.h"
-#include "Task/Combat/Cover/Cover.h"
-#include "vehicles/cargen.h"
-#include "vehicles/Metadata/VehicleMetadataManager.h"
-#include "vehicles/Metadata/VehicleSeatInfo.h"
-#include "vehicles/Metadata/VehicleEntryPointInfo.h"
-#include "vehicles/Metadata/VehicleLayoutInfo.h"
-#include "vehicles/Metadata/VehicleScenarioLayoutInfo.h"
-#include "vehicles/HandlingMgr.h"
-#include "vehicles/Planes.h"
-#include "vehicles/Train.h"
-#include "vehicles/Vehicle.h"
-#include "vehicles/VehicleFactory.h"
-#include "vehicles/vehicleLightSwitch.h"
-#include "vehicles/wheel.h"
+#include "task/Vehicle/TaskCarUtils.h"
+#include "task/Vehicle/TaskInVehicle.h"
+#include "task/Vehicle/TaskEnterVehicle.h"
+#include "task/Combat/Cover/cover.h"
+#include "Vehicles/cargen.h"
+#include "Vehicles/Metadata/VehicleMetadataManager.h"
+#include "Vehicles/Metadata/VehicleSeatInfo.h"
+#include "Vehicles/Metadata/VehicleEntryPointInfo.h"
+#include "Vehicles/Metadata/VehicleLayoutInfo.h"
+#include "Vehicles/Metadata/VehicleScenarioLayoutInfo.h"
+#include "Vehicles/handlingMgr.h"
+#include "Vehicles/Planes.h"
+#include "Vehicles/train.h"
+#include "Vehicles/vehicle.h"
+#include "Vehicles/VehicleFactory.h"
+#include "Vehicles/vehicleLightSwitch.h"
+#include "Vehicles/wheel.h"
 #include "vfx/vehicleglass/VehicleGlassManager.h"
 #include "scene/EntityIterator.h"
-#include "stats/StatsMgr.h"
-#include "peds/ped.h"
-#include "peds/PedHelmetComponent.h"
+#include "Stats/StatsMgr.h"
+#include "Peds/ped.h"
+#include "Peds/PedHelmetComponent.h"
 
 #if __BANK
 #include "grcore/debugdraw.h"
@@ -2692,7 +2692,7 @@ void CVehicleModelInfo::RequestDials(const sVehicleDashboardData& params)
 						CScaleformMgr::AddParamString(""); // tuning
 						CScaleformMgr::AddParamString(FILTER_UNKNOWN(stationStr));
 
-	#if RSG_PC
+	#if RSG_PC && !RSG_LINUX
 						if(isUserTrack)
 						{
 							CScaleformMgr::AddParamString(audRadioStation::GetUserRadioTrackManager()->GetTrackArtist(audRadioTrack::GetUserTrackIndexFromTextId(trackId)));
@@ -2703,7 +2703,7 @@ void CVehicleModelInfo::RequestDials(const sVehicleDashboardData& params)
 	#endif		
 						CScaleformMgr::AddParamString(FILTER_UNKNOWN(artistStr));
 						CScaleformMgr::AddParamString(FILTER_UNKNOWN(trackStr));
-	#if RSG_PC
+	#if RSG_PC && !RSG_LINUX
 						}
 	#endif // RSG_PC
 						CScaleformMgr::EndMethod();
