diff --git a/src/dev_ng/game/modelinfo/VehicleModelInfo.h b/src/dev_ng/game/modelinfo/VehicleModelInfo.h
--- a/src/dev_ng/game/modelinfo/VehicleModelInfo.h
+++ b/src/dev_ng/game/modelinfo/VehicleModelInfo.h
@@ -23,12 +23,12 @@
 #include "modelinfo/VehicleModelInfoEnums.h"
 #include "modelinfo/VehicleModelInfoFlags.h"
 #include "modelinfo/VehicleModelInfoPlates.h"
-#include "renderer/Color.h"
+#include "renderer/color.h"
 #include "renderer/HierarchyIds.h"
 #include "streaming/streamingrequest.h"
-#include "task/Combat/Cover/Cover.h" // For the CCoverPointInfo class
-#include "vehicles/VehicleDefines.h"
-#include "vehicles/vehicle_channel.h"
+#include "task/Combat/Cover/cover.h" // For the CCoverPointInfo class
+#include "Vehicles/VehicleDefines.h"
+#include "Vehicles/vehicle_channel.h"
 
 #if !__SPU
 	// Rage headers
@@ -37,8 +37,8 @@
 	// Game headers
 	#include "scene/loader/MapData_Extensions.h"
 	#include "scene/loader/MapData_Misc.h"
-	#include "fwtl/pool.h"
-	#include "vfx/metadata/vfxvehicleinfo.h"
+	#include "fwtl/Pool.h"
+	#include "vfx/metadata/VfxVehicleInfo.h"
 #endif //!__SPU...
 
 
