diff --git a/src/dev_ng/game/modelinfo/WeaponModelInfo.cpp b/src/dev_ng/game/modelinfo/WeaponModelInfo.cpp
--- a/src/dev_ng/game/modelinfo/WeaponModelInfo.cpp
+++ b/src/dev_ng/game/modelinfo/WeaponModelInfo.cpp
@@ -10,14 +10,14 @@
 #include "fwanimation/animmanager.h"
 #include "rmcore/drawable.h"
 #include "phbound/BoundSphere.h"
-#include "phbound/BoundComposite.h"
-#include "phcore/MaterialMgr.h"
+#include "phbound/boundcomposite.h"
+#include "phcore/materialmgr.h"
 #include "phcore/phmath.h"
 #include "physics/archetype.h"
 #include "physics/tunable.h"
 
 // Framework headers
-#include "fwanimation\expressionsets.h"
+#include "fwanimation/expressionsets.h"
 #include "vfx/ptfx/ptfxmanager.h"
 
 // game headers
@@ -30,7 +30,7 @@
 #include "shaders/CustomShaderEffectBase.h"
 #include "shaders/CustomShaderEffectWeapon.h"
 #include "scene/texLod.h"
-#include "weapons/weaponchannel.h"
+#include "weapons/WeaponChannel.h"
 
 WEAPON_OPTIMISATIONS()
 
