diff --git a/src/dev_ng/game/animation/AnimScene/entities/AnimSceneEntities.cpp b/src/dev_ng/game/animation/AnimScene/entities/AnimSceneEntities.cpp
--- a/src/dev_ng/game/animation/AnimScene/entities/AnimSceneEntities.cpp
+++ b/src/dev_ng/game/animation/AnimScene/entities/AnimSceneEntities.cpp
@@ -3,7 +3,7 @@
 #include "AnimSceneEntities_parser.h"
 
 // rage includes
-#include "cutfile\cutfevent.h" // required for AnimatedModelEntity below
+#include "cutfile/cutfevent.h" // required for AnimatedModelEntity below
 #include "grcore/debugdraw.h"
 
 // framework includes
@@ -14,7 +14,7 @@
 #include "animation/AnimScene/AnimSceneEntityIterator.h"
 #include "animation/AnimScene/entities/AnimSceneEntityInitialisers.h"
 #include "cutscene/AnimatedModelEntity.h" // required for weapon helper functions
-#include "Peds/Ped.h"
+#include "Peds/ped.h"
 #include "Peds/PedFactory.h"
 #include "Peds/PedIntelligence.h"
 #if __BANK
@@ -24,8 +24,8 @@
 #include "Objects/object.h"
 #include "Objects/objectpopulation.h"
 #include "streaming/streaming.h"
-#include "vehicles/vehicle.h"
-#include "vehicles/VehicleFactory.h"
+#include "Vehicles/vehicle.h"
+#include "Vehicles/VehicleFactory.h"
 
 ANIM_OPTIMISATIONS()
 
