diff --git a/src/dev_ng/rage/base/src/audiosoundtypes/ifsound.cpp b/src/dev_ng/rage/base/src/audiosoundtypes/ifsound.cpp
--- a/src/dev_ng/rage/base/src/audiosoundtypes/ifsound.cpp
+++ b/src/dev_ng/rage/base/src/audiosoundtypes/ifsound.cpp
@@ -5,7 +5,7 @@
 //
 
 #include "ifsound.h"
-#include "audiohardware\channel.h"
+#include "audiohardware/channel.h"
 #include "audioengine/engine.h"
 #include "audioengine/soundfactory.h"
 
