diff --git a/src/dev_ng/rage/base/src/diag/diagerrorcodes.h b/src/dev_ng/rage/base/src/diag/diagerrorcodes.h
--- a/src/dev_ng/rage/base/src/diag/diagerrorcodes.h
+++ b/src/dev_ng/rage/base/src/diag/diagerrorcodes.h
@@ -1,7 +1,7 @@
 #ifndef _DIAGERRORCODES_H_
 #define _DIAGERRORCODES_H_
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 #include "atl/map.h"
 #include "diag/errorcodes.h"
 #include "system/language.h"
