diff --git a/src/dev_ng/rage/base/src/diag/tracker_remote2.cpp b/src/dev_ng/rage/base/src/diag/tracker_remote2.cpp
--- a/src/dev_ng/rage/base/src/diag/tracker_remote2.cpp
+++ b/src/dev_ng/rage/base/src/diag/tracker_remote2.cpp
@@ -115,7 +115,7 @@ diagTrackerRemote2::diagTrackerRemote2(const bool startWorker, int customPort)
 			strcat(clientInfo.m_SymbolFile,".cmp");
 	}
 
-#elif __WIN32PC // __XENON
+#elif __WIN32PC && !__LINUX // __XENON
 	// sysStack::InitClass() hasn't been called yet, we'll have to figure out the
 	// location of the symbol file ourselves.
 //	safecpy(clientInfo.m_SymbolFile, sysStack::GetSymbolFilename());
