diff --git a/src/dev_ng/game/replaycoordinator/storage/Montage.cpp b/src/dev_ng/game/replaycoordinator/storage/Montage.cpp
--- a/src/dev_ng/game/replaycoordinator/storage/Montage.cpp
+++ b/src/dev_ng/game/replaycoordinator/storage/Montage.cpp
@@ -17,10 +17,10 @@
 
 // game
 #include "audio/frontendaudioentity.h"
-#include "control/replay/Replay.h"
+#include "control/replay/replay.h"
 #include "control/replay/ReplaySettings.h"
 #include "debug/Debug.h"
-#include "FrontEnd/WarningScreen.h"
+#include "frontend/WarningScreen.h"
 #include "frontend/VideoEditor/Core/VideoEditorUtil.h"
 #include "replaycoordinator/ReplayAudioTrackProvider.h"
 #include "replaycoordinator/replay_coordinator_channel.h"
