diff --git a/src/dev_ng/game/network/Network.cpp b/src/dev_ng/game/network/Network.cpp
--- a/src/dev_ng/game/network/Network.cpp
+++ b/src/dev_ng/game/network/Network.cpp
@@ -34,12 +34,12 @@
 // framework headers
 #include "fwlocalisation/languagePack.h"
 #include "fwanimation/directorcomponentsyncedscene.h"
-#include "fwnet/NetLogUtils.h"
-#include "fwnet/NetPlayerMessages.h"
+#include "fwnet/netlogutils.h"
+#include "fwnet/netplayermessages.h"
 #include "fwnet/netchannel.h"
-#include "fwnet/NetEventMgr.h"
+#include "fwnet/neteventmgr.h"
 #include "fwnet/netremotelog.h"
-#include "fwnet/netscgamerdatamgr.h"
+#include "fwnet/netscgamerDataMgr.h"
 #include "fwnet/netutils.h"
 #include "fwrenderer/renderthread.h"
 #include "fwscene/stores/staticboundsstore.h"
@@ -52,72 +52,72 @@
 #include "camera/CamInterface.h"
 #include "camera/gameplay/follow/FollowVehicleCamera.h"
 #include "control/gamelogic.h"
-#include "core/app.h"
-#include "core/game.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Core/app.h"
+#include "Core/game.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "event/EventDamage.h"
 #include "event/EventGroup.h"
 #include "event/EventNetwork.h"
 #include "frontend/MiniMap.h"
-#include "Frontend/MultiplayerGamerTagHud.h"
-#include "Frontend/MultiplayerChat.h"
+#include "frontend/MultiplayerGamerTagHud.h"
+#include "frontend/MultiplayerChat.h"
 #include "game/BackgroundScripts/BackgroundScripts.h"
 #include "game/weather.h"
-#include "network/arrays/NetworkArrayMgr.h"
+#include "network/Arrays/NetworkArrayMgr.h"
 #include "network/Bandwidth/NetworkBandwidthManager.h"
 #include "network/Cloud/UserContentManager.h"
 #include "network/Crews/NetworkCrewDataMgr.h"
 #include "network/Debug/NetworkBot.h"
 #include "network/Debug/NetworkDebug.h"
 #include "network/Debug/NetworkSoakTests.h"
-#include "network/events/NetworkEventTypes.h"
-#include "Network/General/NetworkAssetVerifier.h"
-#include "network/general/NetworkFakePlayerNames.h"
-#include "Network/General/NetworkHasherUtil.h"
-#include "Network/General/NetworkScratchpad.h"
-#include "network/general/NetworkStreamingRequestManager.h"
+#include "network/Events/NetworkEventTypes.h"
+#include "network/General/NetworkAssetVerifier.h"
+#include "network/General/NetworkFakePlayerNames.h"
+#include "network/General/NetworkHasherUtil.h"
+#include "network/General/NetworkScratchpad.h"
+#include "network/General/NetworkStreamingRequestManager.h"
 #include "network/Live/livemanager.h"
-#include "Network/Live/NetworkTelemetry.h"
-#include "Network/Objects/NetworkObjectMgr.h"
-#include "Network/Objects/NetworkObjectPopulationMgr.h"
-#include "Network/Objects/Entities/NetObjPlayer.h"
-#include "Network/Party/NetworkParty.h"
-#include "network/players/NetworkPlayerMgr.h"
-#include "network/roaming/RoamingBubbleMgr.h"
-#include "Network/Sessions/NetworkSession.h"
-#include "Network/Sessions/NetworkVoiceSession.h"
-#include "Network/Shop/Catalog.h"
-#include "Network/SocialClubServices/GamePresenceEvents.h"
-#include "Network/Stats/networkleaderboardsessionmgr.h"
-#include "Network/Voice/NetworkVoice.h"
-#include "Network/SocialClubServices/SocialClubCommunityEventMgr.h"
+#include "network/Live/NetworkTelemetry.h"
+#include "network/Objects/NetworkObjectMgr.h"
+#include "network/Objects/NetworkObjectPopulationMgr.h"
+#include "network/Objects/Entities/NetObjPlayer.h"
+#include "network/Party/NetworkParty.h"
+#include "network/Players/NetworkPlayerMgr.h"
+#include "network/Roaming/RoamingBubbleMgr.h"
+#include "network/Sessions/NetworkSession.h"
+#include "network/Sessions/NetworkVoiceSession.h"
+#include "network/Shop/Catalog.h"
+#include "network/SocialClubServices/GamePresenceEvents.h"
+#include "network/Stats/NetworkLeaderboardSessionMgr.h"
+#include "network/Voice/NetworkVoice.h"
+#include "network/SocialClubServices/SocialClubCommunityEventMgr.h"
 #include "network/Live/NetworkLegalRestrictionsManager.h"
-#include "peds/PedPopulation.h"
-#include "peds/Ped.h"
-#include "peds/PedIntelligence.h"
-#include "peds/PlayerInfo.h"
+#include "Peds/pedpopulation.h"
+#include "Peds/ped.h"
+#include "Peds/PedIntelligence.h"
+#include "Peds/PlayerInfo.h"
 #include "pickups/PickupManager.h"
 #include "scene/DataFileMgr.h"
 #include "scene/FileLoader.h"
 #include "scene/streamer/StreamVolume.h"
-#include "scene/texlod.h"
+#include "scene/texLod.h"
 #include "scene/world/GameWorld.h"
 #include "script/commands_datafile.h"
 #include "script/commands_network.h"
 #include "script/script_hud.h"
 #include "streaming/IslandHopper.h"
-#include "streaming/PopulationStreaming.h"
+#include "streaming/populationstreaming.h"
 #include "system/controlMgr.h"
 #include "system/memvisualize.h"
 #include "system/memmanager.h"
 #include "system/InfoState.h"
-#include "vehicles/vehiclepopulation.h"
+#include "Vehicles/vehiclepopulation.h"
 #include "task/Physics/TaskNM.h"
 #include "task/Combat/TaskCombatMelee.h"
 #include "task/Combat/Subtasks/TaskVehicleCombat.h"
 #include "task/Default/TaskIncapacitated.h"
 #include "task/Default/TaskPlayer.h"
-#include "Task/Motion/Locomotion/TaskInWater.h"
+#include "task/Motion/Locomotion/TaskInWater.h"
 #include "task/Movement/TaskGetUp.h"
 #include "task/Scenario/Types/TaskUseScenario.h"
 #include "task/Weapons/WeaponTarget.h"
@@ -126,21 +126,21 @@
 #include "Vehicles/Bike.h"
 #include "Vehicles/cargen.h"
 #include "Vehicles/train.h"
-#include "vehicles/vehicle.h"
+#include "Vehicles/vehicle.h"
 #include "vfx/misc/MovieManager.h"
 #include "vfx/vehicleglass/VehicleGlassComponent.h"
-#include "Weapons/Projectiles/Projectile.h"
-#include "Weapons/Projectiles/ProjectileManager.h"
+#include "weapons/projectiles/Projectile.h"
+#include "weapons/projectiles/ProjectileManager.h"
 #include "weapons/Bullet.h"
 #include "weapons/Weapon.h"
-#include "stats/StatsInterface.h"
-#include "stats/StatsDataMgr.h"
-#include "stats/StatsMgr.h"
-#include "stats/StatsSavesMgr.h"
+#include "Stats/StatsInterface.h"
+#include "Stats/StatsDataMgr.h"
+#include "Stats/StatsMgr.h"
+#include "Stats/StatsSavesMgr.h"
 #include "Objects/Door.h"
-#include "stats/MoneyInterface.h"
+#include "Stats/MoneyInterface.h"
 #include "renderer/RenderTargetMgr.h"
-#include "vehicles/vehicleDamage.h"
+#include "Vehicles/vehicleDamage.h"
 
 #include "frontend/UIWorldIcon.h"
 #include "frontend/GameStreamMgr.h"
@@ -373,7 +373,7 @@ public:
 	}
 };
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 NetworkExitFlow::NetworkExitFlow() 
 	: m_ExitToDesktopState(ETD_NONE)
 	, m_fromMP(false)
@@ -728,7 +728,7 @@ CNetworkScratchpads&                CNetwork::sm_ScriptScratchpads(s_ScriptScrat
 CRoamingBubbleMgr&                  CNetwork::sm_RoamingBubbleMgr(s_RoamingBubbleMgr);
 CNetworkVoice&						CNetwork::sm_NetworkVoice(s_NetworkVoice);
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 CNetworkTextChat&					CNetwork::sm_NetworkTextChat(s_NetworkTextChat);
 NetworkExitFlow						CNetwork::sm_ExitToDesktopFlow;
 #endif
@@ -1828,7 +1828,7 @@ CNetwork::Init(unsigned initMode)
 
         ASSERT_ONLY(CQueriableInterface::ValidateTaskInfos());
 
-        #if __WIN32PC
+		#if __WIN32PC && !__LINUX
 		sm_ExitToDesktopFlow.ResetExitFlowState();
 		#endif
 	}
@@ -1900,7 +1900,9 @@ CNetwork::Shutdown(unsigned shutdownMode)
 
         netLog::StopLoggingThread();
 
-		WIN32PC_ONLY(sm_ExitToDesktopFlow.ResetExitFlowState());
+		#if __WIN32PC && !__LINUX
+		sm_ExitToDesktopFlow.ResetExitFlowState();
+		#endif
     }
 	else if(shutdownMode == SHUTDOWN_CORE)
 	{
@@ -2426,7 +2428,7 @@ CNetwork::Update(bool fromLoadingCode)
                 }
 			}
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 			sm_ExitToDesktopFlow.Update();
 #endif // __WIN32PC
 
@@ -3010,7 +3012,7 @@ void CNetwork::UpdateEventTriggerCounters()
 
 void CNetwork::UpdateScuiHotkeys()
 {
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 	CControl& control = CControlMgr::GetPlayerMappingControl();
 
 	// If the SCUI is disalbed for shutdown, do not attempt to show the UI.
@@ -3544,7 +3546,7 @@ bool CNetwork::IsReadyToCheckNetworkAccess(const eNetworkAccessArea OUTPUT_ONLY(
 	}
 #endif
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 	// On PC, wait for sign-in to complete.
 	if (g_rlPc.GetProfileManager()->IsSigningIn())
 	{
