diff --git a/src/dev_ng/game/debug/DebugDraw/DebugVolume.cpp b/src/dev_ng/game/debug/DebugDraw/DebugVolume.cpp
--- a/src/dev_ng/game/debug/DebugDraw/DebugVolume.cpp
+++ b/src/dev_ng/game/debug/DebugDraw/DebugVolume.cpp
@@ -13,8 +13,8 @@
 
 #include "renderer/Util/Util.h" // sadly, this relies on old Vector3 utility functions
 #include "renderer/color.h"
-#include "debug/debugdraw/debugvolume.h"
-#include "debug/debugdraw/debugvolume_kDOP18.h"
+#include "debug/Debugdraw/debugvolume.h"
+#include "debug/Debugdraw/debugvolume_kDOP18.h"
 
 RENDER_OPTIMISATIONS()
 
