diff --git a/src/dev_ng/game/audio/ambience/water/audshoreline.cpp b/src/dev_ng/game/audio/ambience/water/audshoreline.cpp
--- a/src/dev_ng/game/audio/ambience/water/audshoreline.cpp
+++ b/src/dev_ng/game/audio/ambience/water/audshoreline.cpp
@@ -12,8 +12,8 @@
 #include "audio/northaudioengine.h" 
 #include "audio/frontendaudioentity.h"
 #include "audio/gameobjects.h"
-#include "debug/vectormap.h"
-#include "renderer/water.h"
+#include "debug/VectorMap.h"
+#include "renderer/Water.h"
 #include "game/weather.h"
 #include "vfx/VfxHelper.h"
 #include "audio/weatheraudioentity.h"
