diff --git a/src/dev_ng/rage/base/src/rline/rlpresence.cpp b/src/dev_ng/rage/base/src/rline/rlpresence.cpp
--- a/src/dev_ng/rage/base/src/rline/rlpresence.cpp
+++ b/src/dev_ng/rage/base/src/rline/rlpresence.cpp
@@ -2892,7 +2892,7 @@ rlPresence::RefreshSigninState(const int gamerIdx)
                   handleStr);
 #endif
 
-#if !__FINAL
+#if !__SKIPDEBUGAUTH
         //Make sure we're preventing people from seeing secret
         //stuff in our games during dev.
         rlGamerInfo::CheckDevPrivacyPrivilegesAndBail(newGamerInfo.GetLocalIndex());
