diff --git a/src/dev_ng/game/animation/AnimBones.cpp b/src/dev_ng/game/animation/AnimBones.cpp
--- a/src/dev_ng/game/animation/AnimBones.cpp
+++ b/src/dev_ng/game/animation/AnimBones.cpp
@@ -7,10 +7,10 @@
 #include "AnimBones.h"
 
 // Rage headers
-#include "crSkeleton/SkeletonData.h"
+#include "crskeleton/skeletondata.h"
 
 // Gta headers
-#include "animation/anim_channel.h "
+#include "animation/anim_channel.h"
 
 ANIM_OPTIMISATIONS()
 
