diff --git a/src/dev_ng/rage/suite/src/bink/movie.cpp b/src/dev_ng/rage/suite/src/bink/movie.cpp
--- a/src/dev_ng/rage/suite/src/bink/movie.cpp
+++ b/src/dev_ng/rage/suite/src/bink/movie.cpp
@@ -1565,7 +1565,7 @@ void bwMovie::ShaderEndDraw(bwMovie* movie){
     if (movie)
 	{
 		movie->m_bRenderInProgress = false;
-#if !RSG_PC
+#if !RSG_PC && !RSG_LINUX
 		if (movie->m_textureFence[!movie->m_bFenceIndex])
 		{
 			GRCDEVICE.GpuFreeFence(movie->m_textureFence[!movie->m_bFenceIndex]);
