diff --git a/src/dev_ng/game/control/replay/Game/ReplayInterfaceGame.cpp b/src/dev_ng/game/control/replay/Game/ReplayInterfaceGame.cpp
--- a/src/dev_ng/game/control/replay/Game/ReplayInterfaceGame.cpp
+++ b/src/dev_ng/game/control/replay/Game/ReplayInterfaceGame.cpp
@@ -2,14 +2,14 @@
 
 #if GTA_REPLAY
 
-#include "game/clock.h"
+#include "game/Clock.h"
 #include "game/weather.h"
 
 #include "../ReplayController.h"
 #include "../Misc/ReplayPacket.h"
 #include "../Misc/WaterPacket.h"
 #include "../ReplayLightingManager.h"
-#include "../audio/AmbientAudioPacket.h"
+#include "../Audio/AmbientAudioPacket.h"
 #include "../Misc/ClothPacket.h"
 
 #include "renderer/RenderPhases/RenderPhaseFX.h"
