diff --git a/src/dev_ng/game/security/plugins/dllnamecheckplugin.cpp b/src/dev_ng/game/security/plugins/dllnamecheckplugin.cpp
--- a/src/dev_ng/game/security/plugins/dllnamecheckplugin.cpp
+++ b/src/dev_ng/game/security/plugins/dllnamecheckplugin.cpp
@@ -14,16 +14,16 @@
 #include "system/threadtype.h"
 
 //Game headers
-#include "Network/Live/NetworkTelemetry.h"
-#include "Network/NetworkInterface.h"
-#include "Network/Events/NetworkEventTypes.h"
-#include "Network/General/NetworkAssetVerifier.h"
-#include "Network/Sessions/NetworkSession.h"
+#include "network/Live/NetworkTelemetry.h"
+#include "network/NetworkInterface.h"
+#include "network/Events/NetworkEventTypes.h"
+#include "network/General/NetworkAssetVerifier.h"
+#include "network/Sessions/NetworkSession.h"
 #include "script/script.h"
 #include "system/InfoState.h"
-#include "system/memorycheck.h"
+#include "system/MemoryCheck.h"
 #include "streaming/defragmentation.h"
-#include "stats/StatsInterface.h"
+#include "Stats/StatsInterface.h"
 
 // Security headers
 #include "net/status.h"
