diff --git a/src/dev_ng/game/security/plugins/playernamemonitorplugin.cpp b/src/dev_ng/game/security/plugins/playernamemonitorplugin.cpp
--- a/src/dev_ng/game/security/plugins/playernamemonitorplugin.cpp
+++ b/src/dev_ng/game/security/plugins/playernamemonitorplugin.cpp
@@ -10,9 +10,9 @@
 #include "security/ragesecplugin.h"
 #include "security/ragesecenginetasks.h"
 #include "stats/StatsInterface.h"
-#include "Peds\ped.h"
-#include "Peds\PlayerInfo.h"
-#include "rline\rlgamerinfo.h "
+#include "Peds/ped.h"
+#include "Peds/PlayerInfo.h"
+#include "rline/rlgamerinfo.h "
 #include "security/ragesechashutility.h"
 
 using namespace rage;
