diff --git a/src/dev_ng/rage/base/src/audiohardware/device_pc.cpp b/src/dev_ng/rage/base/src/audiohardware/device_pc.cpp
--- a/src/dev_ng/rage/base/src/audiohardware/device_pc.cpp
+++ b/src/dev_ng/rage/base/src/audiohardware/device_pc.cpp
@@ -4,7 +4,7 @@
 // Copyright (C) 1999-2008 Rockstar Games.  All Rights Reserved.
 //
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 
 #include "system/xtl.h"
 #include "driver.h"
