diff --git a/src/dev_ng/rage/base/src/audiohardware/wmfstreamreader.h b/src/dev_ng/rage/base/src/audiohardware/wmfstreamreader.h
--- a/src/dev_ng/rage/base/src/audiohardware/wmfstreamreader.h
+++ b/src/dev_ng/rage/base/src/audiohardware/wmfstreamreader.h
@@ -5,7 +5,7 @@
 // 
 #ifndef AUD_WMFSTREAMREADER_H
 #define AUD_WMFSTREAMREADER_H
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 
 #include "mediareader.h"
 
