diff --git a/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.h b/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.h
--- a/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.h
+++ b/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.h
@@ -7,7 +7,7 @@
 #ifndef CINEMATIC_WATER_CRASH_CAMERA_H
 #define CINEMATIC_WATER_CRASH_CAMERA_H
 
-#include "Camera/Base/BaseCamera.h"
+#include "camera/base/BaseCamera.h"
 #include "camera/system/CameraMetadata.h"
 
 class camCinematicWaterCrashCameraMetadata;
