diff --git a/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicGroupCamera.cpp b/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicGroupCamera.cpp
--- a/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicGroupCamera.cpp
+++ b/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicGroupCamera.cpp
@@ -5,10 +5,10 @@
 //
 
 #include "CinematicGroupCamera.h"
-#include "camera/cinematic/cinematicdirector.h"
+#include "camera/cinematic/CinematicDirector.h"
 #include "camera/CamInterface.h"
 #include "camera/helpers/Collision.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "physics/gtaArchetype.h"
 #include "scene/Entity.h"
 
