diff --git a/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.cpp b/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.cpp
--- a/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.cpp
+++ b/src/dev_ng/game/camera/cinematic/camera/tracking/CinematicWaterCrashCamera.cpp
@@ -13,9 +13,9 @@
 #include "camera/system/CameraMetadata.h"
 #include "camera/cinematic/context/CinematicWaterCrashContext.h"
 #include "fwmaths/random.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "scene/Entity.h"
-#include "vehicles/vehicle.h"
+#include "Vehicles/vehicle.h"
 
 INSTANTIATE_RTTI_CLASS(camCinematicWaterCrashCamera,0x84F7B3E8)
 
