diff --git a/src/dev_ng/game/scene/portals/InteriorProxy.cpp b/src/dev_ng/game/scene/portals/InteriorProxy.cpp
--- a/src/dev_ng/game/scene/portals/InteriorProxy.cpp
+++ b/src/dev_ng/game/scene/portals/InteriorProxy.cpp
@@ -17,25 +17,25 @@
 #include "fwscene/stores/mapdatastore.h"
 #include "fwscene/stores/staticboundsstore.h"
 #include "fwsys/metadatastore.h"
-#include "fwutil/keygen.h"
+#include "fwutil/KeyGen.h"
 #include "streaming/streamingvisualize.h"
 #include "fwnet/netchannel.h"
 
 // game headers
 #include "audio/northaudioengine.h"
 #include "control/replay/replay.h"
-#include "modelInfo/MloModelInfo.h"
+#include "modelinfo/MloModelInfo.h"
 #include "network/General/NetworkUtil.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "scene/scene_channel.h"
 #include "scene/loader/mapFileMgr.h"
-#include "scene/lod/lodMgr.h"
+#include "scene/lod/LodMgr.h"
 #include "scene/portals/InteriorInst.h"
-#include "scene/portals/interiorgroupmanager.h"
+#include "scene/portals/InteriorGroupManager.h"
 #include "scene/portals/Portal.h"
 #include "scene/world/GameWorld.h"
-#include "vehicles/vehicle.h"
-#include "vehicles/vehiclepopulation.h"
+#include "Vehicles/vehicle.h"
+#include "Vehicles/vehiclepopulation.h"
 #include "scene/dlc_channel.h"
 
 STREAMING_OPTIMISATIONS()
