diff --git a/src/dev_ng/game/ik/IkManager.cpp b/src/dev_ng/game/ik/IkManager.cpp
--- a/src/dev_ng/game/ik/IkManager.cpp
+++ b/src/dev_ng/game/ik/IkManager.cpp
@@ -38,13 +38,13 @@
 #include "ik/solvers/TorsoVehicleSolver.h"
 #include "ik/solvers/RootSlopeFixupSolver.h"
 #include "Peds/Horse/horseComponent.h"
-#include "Peds/Ped.h"
+#include "Peds/ped.h"
 #include "Peds/PedTuning.h"
 #include "Peds/PedIKSettings.h"
 #include "Peds/PedIntelligence.h"
 #include "scene/world/GameWorld.h"
-#include "Task/System/Tuning.h"
-#include "Task/Vehicle/TaskInVehicle.h"
+#include "task/System/Tuning.h"
+#include "task/Vehicle/TaskInVehicle.h"
 
 #if __IK_DEV
 #include "scene/world/GameWorld.h"
