diff --git a/src/dev_ng/game/control/replay/Effects/ParticleWaterFxPacket.h b/src/dev_ng/game/control/replay/Effects/ParticleWaterFxPacket.h
--- a/src/dev_ng/game/control/replay/Effects/ParticleWaterFxPacket.h
+++ b/src/dev_ng/game/control/replay/Effects/ParticleWaterFxPacket.h
@@ -6,13 +6,13 @@
 #ifndef INC_WATERFXPACKET_H_
 #define INC_WATERFXPACKET_H_
 
-#include "Control/replay/ReplaySettings.h"
+#include "control/replay/ReplaySettings.h"
 
 // Replay only setup for win32
 #if GTA_REPLAY
 
-#include "control\replay\PacketBasics.h"
-#include "control\replay\Effects\ParticlePacket.h"
+#include "control/replay/PacketBasics.h"
+#include "control/replay/Effects/ParticlePacket.h"
 
 namespace rage
 { class ptxEffectInst; }
