diff --git a/src/dev_ng/game/control/replay/Effects/ParticleScriptedFxPacket.h b/src/dev_ng/game/control/replay/Effects/ParticleScriptedFxPacket.h
--- a/src/dev_ng/game/control/replay/Effects/ParticleScriptedFxPacket.h
+++ b/src/dev_ng/game/control/replay/Effects/ParticleScriptedFxPacket.h
@@ -6,13 +6,13 @@
 #ifndef INC_SCRIPTFXPACKET_H_
 #define INC_SCRIPTFXPACKET_H_
 
-#include "Control/replay/ReplaySettings.h"
+#include "control/replay/ReplaySettings.h"
 
 // Replay only setup for win32
 #if GTA_REPLAY
 
-#include "control\replay\Effects\ParticlePacket.h"
-#include "vector\matrix34.h"
+#include "control/replay/Effects/ParticlePacket.h"
+#include "vector/matrix34.h"
 
 #if !RSG_ORBIS
 #pragma warning(push)
