diff --git a/src/dev_ng/game/control/replay/Effects/ParticleBloodFxPacket.h b/src/dev_ng/game/control/replay/Effects/ParticleBloodFxPacket.h
--- a/src/dev_ng/game/control/replay/Effects/ParticleBloodFxPacket.h
+++ b/src/dev_ng/game/control/replay/Effects/ParticleBloodFxPacket.h
@@ -6,14 +6,14 @@
 #ifndef INC_BLOODFXPACKET_H_
 #define INC_BLOODFXPACKET_H_
 
-#include "Control/replay/ReplaySettings.h"
+#include "control/replay/ReplaySettings.h"
 
 // Replay only setup for win32
 #if GTA_REPLAY
 
-#include "control\replay\Effects\ParticlePacket.h"
-#include "control\replay\packetbasics.h"
-#include "vfx\decal\decalsettings.h"
+#include "control/replay/Effects/ParticlePacket.h"
+#include "control/replay/PacketBasics.h"
+#include "vfx/decal/decalsettings.h"
 
 class CPed;
 class CObject;
