diff --git a/src/dev_ng/game/control/replay/Effects/ParticleRippleFxPacket.h b/src/dev_ng/game/control/replay/Effects/ParticleRippleFxPacket.h
--- a/src/dev_ng/game/control/replay/Effects/ParticleRippleFxPacket.h
+++ b/src/dev_ng/game/control/replay/Effects/ParticleRippleFxPacket.h
@@ -6,14 +6,14 @@
 #ifndef INC_RIPPLEFXPACKET_H_
 #define INC_RIPPLEFXPACKET_H_
 
-#include "Control/replay/ReplaySettings.h"
+#include "control/replay/ReplaySettings.h"
 
 // Replay only setup for win32
 #if GTA_REPLAY
 
-#include "control\replay\Effects\ParticlePacket.h"
-#include "control\replay\Misc\ReplayPacket.h"
-#include "vector\vector3.h"
+#include "control/replay/Effects/ParticlePacket.h"
+#include "control/replay/Misc/ReplayPacket.h"
+#include "vector/vector3.h"
 
 #if !RSG_ORBIS
 #pragma warning(push)
