diff --git a/src/dev_ng/game/control/replay/Effects/ParticleVehicleFxPacket.cpp b/src/dev_ng/game/control/replay/Effects/ParticleVehicleFxPacket.cpp
--- a/src/dev_ng/game/control/replay/Effects/ParticleVehicleFxPacket.cpp
+++ b/src/dev_ng/game/control/replay/Effects/ParticleVehicleFxPacket.cpp
@@ -10,8 +10,8 @@
 #include "control/replay/ReplayInternal.h"
 #include "control/replay/ReplayInterfaceVeh.h"
 #include "control/replay/Misc/ReplayPacket.h"
-#include "Timecycle/TimeCycle.h"
-#include "vehicles/heli.h"
+#include "timecycle/TimeCycle.h"
+#include "Vehicles/Heli.h"
 #include "vfx/ptfx/ptfxattach.h"
 #include "vfx/ptfx/ptfxmanager.h"
 #include "vfx/systems/VfxVehicle.h"
