diff --git a/src/dev_ng/game/control/replay/Vehicle/VehicleBikePacket.h b/src/dev_ng/game/control/replay/Vehicle/VehicleBikePacket.h
--- a/src/dev_ng/game/control/replay/Vehicle/VehicleBikePacket.h
+++ b/src/dev_ng/game/control/replay/Vehicle/VehicleBikePacket.h
@@ -8,7 +8,7 @@
 #ifndef VEHBIKEPACKET_H
 #define VEHBIKEPACKET_H
 
-#include "Control/replay/ReplaySettings.h"
+#include "control/replay/ReplaySettings.h"
 
 // Replay only setup for win32
 #if GTA_REPLAY
