diff --git a/src/dev_ng/game/control/replay/Vehicle/WheelPacket.cpp b/src/dev_ng/game/control/replay/Vehicle/WheelPacket.cpp
--- a/src/dev_ng/game/control/replay/Vehicle/WheelPacket.cpp
+++ b/src/dev_ng/game/control/replay/Vehicle/WheelPacket.cpp
@@ -3,7 +3,7 @@
 
 #if GTA_REPLAY
 
-#include "Control/replay/ReplayExtensions.h"
+#include "control/replay/ReplayExtensions.h"
 
 u32 CPacketWheel::ms_RecordSession = 1;
 
