diff --git a/src/dev_ng/game/ai/ambient/AmbientModelSetManager.cpp b/src/dev_ng/game/ai/ambient/AmbientModelSetManager.cpp
--- a/src/dev_ng/game/ai/ambient/AmbientModelSetManager.cpp
+++ b/src/dev_ng/game/ai/ambient/AmbientModelSetManager.cpp
@@ -1,5 +1,5 @@
 // File header
-#include "AI/Ambient/AmbientModelSetManager.h"
+#include "ai/ambient/AmbientModelSetManager.h"
 
 // Rage headers
 #include "ai/aichannel.h"
