diff --git a/src/dev_ng/game/text/messages.cpp b/src/dev_ng/game/text/messages.cpp
--- a/src/dev_ng/game/text/messages.cpp
+++ b/src/dev_ng/game/text/messages.cpp
@@ -24,18 +24,18 @@
 #include "Control/GameLogic.h"
 #include "control/replay/replay.h"
 #include "core/game.h"
-#include "cutscene\CutSceneManagerNew.h"
-#include "debug\Debug.h"
-#include "frontend\BusySpinner.h"
-#include "frontend\PauseMenu.h"
+#include "cutscene/CutSceneManagerNew.h"
+#include "debug/Debug.h"
+#include "frontend/BusySpinner.h"
+#include "frontend/PauseMenu.h"
 #include "frontend/WarningScreen.h"
-#include "frontend\NewHud.h"
+#include "frontend/NewHud.h"
 #include "Frontend/Scaleform/ScaleFormStore.h"
 #include "frontend/VideoEditor/VideoEditorInterface.h"
 #include "frontend/ui_channel.h"
-#include "game\Clock.h"
+#include "game/Clock.h"
 #include "peds/Ped.h"
-#include "Stats\StatsMgr.h"
+#include "Stats/StatsMgr.h"
 #include "renderer/DrawLists/DrawListMgr.h"
 #include "SaveLoad/savegame_autoload.h"
 #include "SaveLoad/savegame_messages.h"
@@ -43,13 +43,13 @@
 #include "script/script_channel.h"
 #include "script/script_hud.h"
 #include "system/controlMgr.h"
-#include "Text\Text.h"
-#include "Text\TextFormat.h"
-#include "Text\TextConversion.h"
+#include "Text/Text.h"
+#include "Text/TextFormat.h"
+#include "Text/TextConversion.h"
 
 // Parser headers
 #include "parser/manager.h"
-#include "Text\dialogueCharacters_parser.h"
+#include "Text/dialogueCharacters_parser.h"
 
 
 #include "data/aes_init.h"
