diff --git a/src/dev_ng/game/text/TextFile.cpp b/src/dev_ng/game/text/TextFile.cpp
--- a/src/dev_ng/game/text/TextFile.cpp
+++ b/src/dev_ng/game/text/TextFile.cpp
@@ -14,28 +14,28 @@
 #include <stdio.h>
 
 // Rage headers
-#include "string\stringhash.h"
+#include "string/stringhash.h"
 #include "system/criticalsection.h"
 
 // Game headers
-#include "Text\TextConversion.h"
-#include "Text\TextFile.h"
-#include "Text\Text.h"
+#include "text/TextConversion.h"
+#include "text/TextFile.h"
+#include "text/text.h"
 #include "chunk.h"
-#include "cutscene/cutscene_channel.h"
-#include "cutscene/cutscenemanagernew.h"
-#include "cutscene/CutSceneAssetManager.h"
-#include "debug\Debug.h"
-#include "frontend\pausemenu.h"
+#include "Cutscene/cutscene_channel.h"
+#include "Cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneAssetManager.h"
+#include "debug/Debug.h"
+#include "frontend/PauseMenu.h"
 #include "SaveLoad/GenericGameStorage.h"
-#include "scene/datafilemgr.h"
-#include "script\script.h"
-#include "system\filemgr.h"
+#include "scene/DataFileMgr.h"
+#include "script/script.h"
+#include "system/FileMgr.h"
 #include "system/param.h"
 #include "system/memory.h"
 #include "system/system.h"
 #include "fwsys/timer.h"
-#include "text\messages.h"
+#include "text/messages.h"
 
 
 RAGE_DEFINE_CHANNEL(text);
@@ -278,7 +278,7 @@ const char* CTextFile::GetLanguageFile(char *buffer, size_t bufferSize, bool bUs
 
 	PARAM_langfilesuffix.Get(suffix);
 
-#if __FINAL
+#if __PACKEDDEBUG
 	suffix = "_rel";
 #endif
 
