diff --git a/src/dev_ng/game/task/Service/Medic/TaskAmbulancePatrol.cpp b/src/dev_ng/game/task/Service/Medic/TaskAmbulancePatrol.cpp
--- a/src/dev_ng/game/task/Service/Medic/TaskAmbulancePatrol.cpp
+++ b/src/dev_ng/game/task/Service/Medic/TaskAmbulancePatrol.cpp
@@ -3,35 +3,35 @@
 // CREATED : 25-05-2009
 
 // File header
-#include "Task/Service/Medic/TaskAmbulancePatrol.h"
+#include "task/Service/Medic/TaskAmbulancePatrol.h"
 
 // Framework headers
 #include "ai/task/taskchannel.h"
 
 // Game headers
-#include "Game/Dispatch/Incidents.h"
-#include "Game/Dispatch/Orders.h"
-#include "Peds/Ped.h"
+#include "game/Dispatch/Incidents.h"
+#include "game/Dispatch/Orders.h"
+#include "Peds/ped.h"
 #include "Peds/PedGeometryAnalyser.h"
 #include "Peds/PedIntelligence.h"
 #include "Peds/HealthConfig.h"
-#include "Physics/WorldProbe/worldprobe.h"
-#include "Task/Combat/TaskDamageDeath.h"
-#include "Task/Default/TaskUnalerted.h"
-#include "Task/Default/TaskWander.h"
-#include "Task/Movement/TaskGoToPoint.h"
-#include "Task/Movement/TaskNavMesh.h"
-#include "Task/Scenario/Types/TaskUseScenario.h"
-#include "Task/Scenario/Info/ScenarioInfoManager.h"
-#include "Task/Vehicle/TaskCar.h"
-#include "Task/Vehicle/TaskEnterVehicle.h"
-#include "Task/Vehicle/TaskExitVehicle.h"
+#include "physics/WorldProbe/worldprobe.h"
+#include "task/Combat/TaskDamageDeath.h"
+#include "task/Default/TaskUnalerted.h"
+#include "task/Default/TaskWander.h"
+#include "task/Movement/TaskGoToPoint.h"
+#include "task/Movement/TaskNavMesh.h"
+#include "task/Scenario/Types/TaskUseScenario.h"
+#include "task/Scenario/info/ScenarioInfoManager.h"
+#include "task/Vehicle/TaskCar.h"
+#include "task/Vehicle/TaskEnterVehicle.h"
+#include "task/Vehicle/TaskExitVehicle.h"
 #include "Vehicles/Automobile.h"
-#include "VehicleAI/driverpersonality.h"
-#include "VehicleAI/task/TaskVehicleGoToAutomobile.h"
-#include "VehicleAI/task/TaskVehicleCruise.h"
-#include "VehicleAi/task/TaskVehiclePark.h"
-#include "Vfx/Misc/Fire.h"
+#include "vehicleAi/driverpersonality.h"
+#include "vehicleAi/task/TaskVehicleGoToAutomobile.h"
+#include "vehicleAi/task/TaskVehicleCruise.h"
+#include "vehicleAi/task/TaskVehiclePark.h"
+#include "vfx/misc/Fire.h"
 
 AI_OPTIMISATIONS()
 
