diff --git a/src/dev_ng/game/task/System/TaskTreeMotion.cpp b/src/dev_ng/game/task/System/TaskTreeMotion.cpp
--- a/src/dev_ng/game/task/System/TaskTreeMotion.cpp
+++ b/src/dev_ng/game/task/System/TaskTreeMotion.cpp
@@ -1,7 +1,7 @@
 //Game headers.
-#include "Task\System\TaskTreeMotion.h"
+#include "Task/System/TaskTreeMotion.h"
 
-#include "Peds\Ped.h"
+#include "Peds/Ped.h"
 
 AI_OPTIMISATIONS()
 
