diff --git a/src/dev_ng/game/weapons/modifiers/WeaponFallOffModifier.cpp b/src/dev_ng/game/weapons/modifiers/WeaponFallOffModifier.cpp
--- a/src/dev_ng/game/weapons/modifiers/WeaponFallOffModifier.cpp
+++ b/src/dev_ng/game/weapons/modifiers/WeaponFallOffModifier.cpp
@@ -5,10 +5,10 @@
 //
 
 // File header
-#include "Weapons/Modifiers/WeaponFallOffModifier.h"
+#include "weapons/modifiers/WeaponFallOffModifier.h"
 
 // Parser
-#include "WeaponFallOffModifier_parser.h"
+#include "weaponfalloffmodifier_parser.h"
 
 WEAPON_OPTIMISATIONS()
 
