diff --git a/src/dev_ng/game/task/Weapons/TaskWeaponBlocked.h b/src/dev_ng/game/task/Weapons/TaskWeaponBlocked.h
--- a/src/dev_ng/game/task/Weapons/TaskWeaponBlocked.h
+++ b/src/dev_ng/game/task/Weapons/TaskWeaponBlocked.h
@@ -2,10 +2,10 @@
 #define TASK_WEAPON_BLOCKED_H
 
 // Game headers
-#include "Task/System/Task.h"
-#include "Task/Weapons/WeaponController.h"
+#include "task/System/Task.h"
+#include "task/Weapons/WeaponController.h"
 #include "Peds/QueriableInterface.h"
-#include "Task/System/TaskFSMClone.h"
+#include "task/System/TaskFSMClone.h"
 
 // Forward declarations
 class CWeaponController;
