diff --git a/src/dev_ng/game/task/Weapons/TaskProjectile.h b/src/dev_ng/game/task/Weapons/TaskProjectile.h
--- a/src/dev_ng/game/task/Weapons/TaskProjectile.h
+++ b/src/dev_ng/game/task/Weapons/TaskProjectile.h
@@ -2,16 +2,16 @@
 #define TASK_PROJECTILE_H
 
 #include "network/General/NetworkUtil.h"
-#include "peds/QueriableInterface.h"
-#include "Scene/Entity.h"
+#include "Peds/QueriableInterface.h"
+#include "scene/Entity.h"
 #include "scene/RegdRefTypes.h"
-#include "Task/System/Task.h"
-#include "Task\System\TaskFsmClone.h"
-#include "Task\System\TaskTypes.h"
-#include "Task/Weapons/Gun/GunIkInfo.h"
-#include "Task/Weapons/Gun/TaskAimGunOnFoot.h"
-#include "Task/Weapons/WeaponTarget.h"
-#include "fwmaths/Angle.h"
+#include "task/System/Task.h"
+#include "task/System/TaskFSMClone.h"
+#include "task/System/TaskTypes.h"
+#include "task/Weapons/Gun/GunIkInfo.h"
+#include "task/Weapons/Gun/TaskAimGunOnFoot.h"
+#include "task/Weapons/WeaponTarget.h"
+#include "fwmaths/angle.h"
 #if !__FINAL
 #include "atl/string.h"
 #endif
