diff --git a/src/dev_ng/game/vfx/metadata/VfxFogVolumeInfo.cpp b/src/dev_ng/game/vfx/metadata/VfxFogVolumeInfo.cpp
--- a/src/dev_ng/game/vfx/metadata/VfxFogVolumeInfo.cpp
+++ b/src/dev_ng/game/vfx/metadata/VfxFogVolumeInfo.cpp
@@ -23,14 +23,14 @@
 
 // framework
 #include "fwscene/stores/psostore.h"
-#include "fwsys/fileexts.h"
+#include "fwsys/fileExts.h"
 #include "vfx/channel.h"
 #include "vfx/vfxwidget.h"
 
 // game
-#include "Camera/CamInterface.h"
-#include "Scene/DataFileMgr.h"
-#include "Renderer/Water.h"
+#include "camera/CamInterface.h"
+#include "scene/DataFileMgr.h"
+#include "renderer/Water.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
