diff --git a/src/dev_ng/game/vfx/metadata/VfxInteriorInfo.cpp b/src/dev_ng/game/vfx/metadata/VfxInteriorInfo.cpp
--- a/src/dev_ng/game/vfx/metadata/VfxInteriorInfo.cpp
+++ b/src/dev_ng/game/vfx/metadata/VfxInteriorInfo.cpp
@@ -21,11 +21,11 @@
 
 // framework
 #include "fwscene/stores/psostore.h"
-#include "fwsys/fileexts.h"
+#include "fwsys/fileExts.h"
 #include "vfx/channel.h"	
 
 // game
-#include "Scene/DataFileMgr.h"
+#include "scene/DataFileMgr.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
