diff --git a/src/dev_ng/game/vfx/metadata/VfxRegionInfo.cpp b/src/dev_ng/game/vfx/metadata/VfxRegionInfo.cpp
--- a/src/dev_ng/game/vfx/metadata/VfxRegionInfo.cpp
+++ b/src/dev_ng/game/vfx/metadata/VfxRegionInfo.cpp
@@ -21,15 +21,15 @@
 
 // framework
 #include "fwscene/stores/psostore.h"
-#include "fwsys/fileexts.h"
+#include "fwsys/fileExts.h"
 #include "vfx/channel.h"
 
 // game
-#include "Game/Clock.h"
-#include "Game/Weather.h"
+#include "game/Clock.h"
+#include "game/weather.h"
 #include "Peds/PopCycle.h"
-#include "Scene/DataFileMgr.h"
-#include "Vfx/VfxHelper.h"
+#include "scene/DataFileMgr.h"
+#include "vfx/VfxHelper.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
