diff --git a/src/dev_ng/game/vfx/metadata/VfxVehicleInfo.cpp b/src/dev_ng/game/vfx/metadata/VfxVehicleInfo.cpp
--- a/src/dev_ng/game/vfx/metadata/VfxVehicleInfo.cpp
+++ b/src/dev_ng/game/vfx/metadata/VfxVehicleInfo.cpp
@@ -21,13 +21,13 @@
 
 // framework
 #include "fwscene/stores/psostore.h"
-#include "fwsys/fileexts.h"
+#include "fwsys/fileExts.h"
 #include "vfx/channel.h"
 
 // game
-#include "ModelInfo/VehicleModelInfo.h"
-#include "Scene/DataFileMgr.h"
-#include "Vehicles/Vehicle.h"
+#include "modelinfo/VehicleModelInfo.h"
+#include "scene/DataFileMgr.h"
+#include "Vehicles/vehicle.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
