diff --git a/src/dev_ng/game/frontend/Store/StoreAdvertScreen.cpp b/src/dev_ng/game/frontend/Store/StoreAdvertScreen.cpp
--- a/src/dev_ng/game/frontend/Store/StoreAdvertScreen.cpp
+++ b/src/dev_ng/game/frontend/Store/StoreAdvertScreen.cpp
@@ -1,14 +1,14 @@
 
-#include "Frontend/Store/StoreAdvertScreen.h"
-
-#include "Frontend/BusySpinner.h"
-#include "Frontend/Store/StoreMainScreen.h"
-#include "Frontend/Store/StoreScreenMgr.h"
-#include "Frontend/Store/StoreUIChannel.h"
-#include "Frontend/Store/StoreTextureManager.h"
-#include "Frontend/WarningScreen.h"
-#include "Network/Live/livemanager.h"
-#include "network/commerce/CommerceManager.h"
+#include "frontend/Store/StoreAdvertScreen.h"
+
+#include "frontend/BusySpinner.h"
+#include "frontend/Store/StoreMainScreen.h"
+#include "frontend/Store/StoreScreenMgr.h"
+#include "frontend/Store/StoreUIChannel.h"
+#include "frontend/Store/StoreTextureManager.h"
+#include "frontend/WarningScreen.h"
+#include "network/Live/livemanager.h"
+#include "network/Commerce/CommerceManager.h"
 #include "system/controlMgr.h"
 #include "system/pad.h"
 
@@ -443,7 +443,7 @@ void cStoreAdvertScreen::Render()
 		CScaleformMgr::RenderMovie( m_MovieId, fTimeStep, true );
 	}
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 	// hide instructional buttons if SCUI is showing ...hints are confusing
 	if (!g_rlPc.IsUiShowing())
 #endif
