diff --git a/src/dev_ng/game/frontend/Store/StoreMainScreen.cpp b/src/dev_ng/game/frontend/Store/StoreMainScreen.cpp
--- a/src/dev_ng/game/frontend/Store/StoreMainScreen.cpp
+++ b/src/dev_ng/game/frontend/Store/StoreMainScreen.cpp
@@ -1,27 +1,27 @@
-#include "Frontend/Store/StoreMainScreen.h"
+#include "frontend/Store/StoreMainScreen.h"
 
 #include "frontend/hud_colour.h"
 #include "frontend/PauseMenu.h"
 #include "frontend/BusySpinner.h"
-#include "Frontend/FrontendStatsMgr.h"
+#include "frontend/FrontendStatsMgr.h"
 #include "frontend/HudTools.h"
 #include "frontend/NewHud.h"
 #include "frontend/Scaleform/ScaleFormMgr.h"
-#include "Frontend/Store/StoreScreenMgr.h"
-#include "Frontend/Store/StoreUIChannel.h"
-#include "Frontend/Store/StoreTextureManager.h"
-#include "FrontEnd/WarningScreen.h"
+#include "frontend/Store/StoreScreenMgr.h"
+#include "frontend/Store/StoreUIChannel.h"
+#include "frontend/Store/StoreTextureManager.h"
+#include "frontend/WarningScreen.h"
 #include "game/Clock.h"
-#include "Network/Cloud/Tunables.h"
-#include "Network/Live/livemanager.h"
-#include "Network/Live/NetworkTransactionTelemetry.h"
-#include "Network/Live/NetworkTelemetry.h"
-#include "Network/commerce/CommerceManager.h"
-#include "Network/Stats//NetworkLeaderboardSessionMgr.h"
+#include "network/Cloud/Tunables.h"
+#include "network/Live/livemanager.h"
+#include "network/Live/NetworkTransactionTelemetry.h"
+#include "network/Live/NetworkTelemetry.h"
+#include "network/Commerce/CommerceManager.h"
+#include "network/Stats//NetworkLeaderboardSessionMgr.h"
 #include "Peds/rendering/PedHeadshotManager.h"
 #include "SaveLoad/savegame_users_and_devices.h"
 #include "scene/ExtraContent.h"
-#include "stats/StatsInterface.h"
+#include "Stats/StatsInterface.h"
 #include "system/controlMgr.h"
 #include "system/pad.h"
 #include "Stats/MoneyInterface.h"
@@ -1149,7 +1149,7 @@ void cStoreMainScreen::Render()
         return;
     }
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 	// hide instructional buttons if SCUI is showing ...hints are confusing
 	if (!g_rlPc.IsUiShowing())
 #endif
