diff --git a/src/dev_ng/game/scene/volume/VolumeAggregate.cpp b/src/dev_ng/game/scene/volume/VolumeAggregate.cpp
--- a/src/dev_ng/game/scene/volume/VolumeAggregate.cpp
+++ b/src/dev_ng/game/scene/volume/VolumeAggregate.cpp
@@ -8,19 +8,19 @@
 #if VOLUME_SUPPORT
 
 //rage includes
-#include "math\amath.h"
-#include "vector\geometry.h"
-#include "vector\color32.h"
-#include "grcore\im.h"
-#include "file\stream.h"
-#include "bank\bkmgr.h"
+#include "math/amath.h"
+#include "vector/geometry.h"
+#include "vector/color32.h"
+#include "grcore/im.h"
+#include "file/stream.h"
+#include "bank/bkmgr.h"
 
 //framework includes
 
 //game includes
-#include "scene\volume\VolumeAggregate.h"
-#include "scene\volume\VolumeManager.h"
-#include "scene\volume\VolumeTool.h"
+#include "scene/volume/VolumeAggregate.h"
+#include "scene/volume/VolumeManager.h"
+#include "scene/volume/VolumeTool.h"
 
 FW_INSTANTIATE_CLASS_POOL(CVolumeAggregate, 256, atHashString("VolumeAggregate",0xf4197375));
 INSTANTIATE_RTTI_CLASS(CVolumeAggregate,0x14DCF76D)
