diff --git a/src/dev_ng/game/scene/volume/VolumeSphere.cpp b/src/dev_ng/game/scene/volume/VolumeSphere.cpp
--- a/src/dev_ng/game/scene/volume/VolumeSphere.cpp
+++ b/src/dev_ng/game/scene/volume/VolumeSphere.cpp
@@ -3,23 +3,23 @@
 // Started	:	2/5/11
 //
 
-#include "scene\volume\VolumeSphere.h"
+#include "scene/volume/VolumeSphere.h"
 #if VOLUME_SUPPORT
 
 //rage includes
-#include "math\amath.h"
-#include "vector\geometry.h"
-#include "vector\color32.h"
-#include "grcore\debugdraw.h"
-#include "grcore\im.h"
-#include "file\stream.h"
-#include "bank\bkmgr.h"
+#include "math/amath.h"
+#include "vector/geometry.h"
+#include "vector/color32.h"
+#include "grcore/debugdraw.h"
+#include "grcore/im.h"
+#include "file/stream.h"
+#include "bank/bkmgr.h"
 
 //framework includes
 
 //game includes
-#include "scene\volume\VolumeManager.h"
-#include "scene\volume\VolumeTool.h"
+#include "scene/volume/VolumeManager.h"
+#include "scene/volume/VolumeTool.h"
 
 FW_INSTANTIATE_CLASS_POOL(CVolumeSphere, 256, atHashString("VolumeSphere",0xa84af05c));
 INSTANTIATE_RTTI_CLASS(CVolumeSphere,0xD52CDBA5)
