diff --git a/src/dev_ng/game/renderer/RenderPhases/RenderPhaseStdNY.cpp b/src/dev_ng/game/renderer/RenderPhases/RenderPhaseStdNY.cpp
--- a/src/dev_ng/game/renderer/RenderPhases/RenderPhaseStdNY.cpp
+++ b/src/dev_ng/game/renderer/RenderPhases/RenderPhaseStdNY.cpp
@@ -12,7 +12,7 @@
 #include "grcore/debugdraw.h"
 #include "grcore/quads.h"
 #include "grcore/fastquad.h"
-#include "profile\profiler.h"
+#include "profile/profiler.h"
 #include "profile/timebars.h"
 // Game headers
 #include "camera/viewports/ViewportManager.h"
@@ -21,19 +21,19 @@
 #include "frontend/BusySpinner.h"
 #include "FrontEnd/HudTools.h"
 #include "FrontEnd/WarningScreen.h"
-#include "Renderer\occlusion.h"
+#include "Renderer/occlusion.h"
 #include "renderer/PostProcessFX.h"
 #include "renderer/Water.h"
-#include "renderer\renderListGroup.h"
+#include "renderer/renderListGroup.h"
 #include "renderer/ScreenshotManager.h"
 #include "frontend/Credits.h"
-#include "Frontend\MobilePhone.h"
-#include "Frontend\PauseMenu.h"
-#include "game\clock.h"
+#include "Frontend/MobilePhone.h"
+#include "Frontend/PauseMenu.h"
+#include "game/clock.h"
 #include "SaveLoad/savegame_photo_manager.h"
-#include "Scene\scene.h"
-#include "Script\script.h"
-#include "Vfx\VisualEffects.h"
+#include "Scene/scene.h"
+#include "Script/script.h"
+#include "Vfx/VisualEffects.h"
 #if RSG_XENON
 #include "grcore/wrapper_d3d.h"
 #endif
