diff --git a/src/dev_ng/game/camera/gameplay/aim/ThirdPersonPedAimInCoverCamera.cpp b/src/dev_ng/game/camera/gameplay/aim/ThirdPersonPedAimInCoverCamera.cpp
--- a/src/dev_ng/game/camera/gameplay/aim/ThirdPersonPedAimInCoverCamera.cpp
+++ b/src/dev_ng/game/camera/gameplay/aim/ThirdPersonPedAimInCoverCamera.cpp
@@ -16,7 +16,7 @@
 #include "camera/helpers/FrameInterpolator.h"
 #include "camera/system/CameraFactory.h"
 #include "camera/system/CameraMetadata.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "scene/Entity.h"
 #include "task/Combat/Cover/cover.h"
 #include "task/Combat/Cover/TaskCover.h"
