diff --git a/src/dev_ng/game/audio/environment/audwarpmanager.h b/src/dev_ng/game/audio/environment/audwarpmanager.h
--- a/src/dev_ng/game/audio/environment/audwarpmanager.h
+++ b/src/dev_ng/game/audio/environment/audwarpmanager.h
@@ -4,7 +4,7 @@
 #include "scene/world/gameworld.h"
 #include "peds/ped.h"
 #include "vehicles/vehicle.h" 
-#include "game\Clock.h"
+#include "game/Clock.h"
 #include "camera/CamInterface.h"
 #include "camera/debug/DebugDirector.h"
 
