diff --git a/src/dev_ng/game/audio/environment/audwarpmanager.h b/src/dev_ng/game/audio/environment/audwarpmanager.h
--- a/src/dev_ng/game/audio/environment/audwarpmanager.h
+++ b/src/dev_ng/game/audio/environment/audwarpmanager.h
@@ -1,10 +1,10 @@
 #ifndef AUD_WARP_MANAGER_H_
 #define AUD_WARP_MANAGER_H_
 
-#include "scene/world/gameworld.h"
-#include "peds/ped.h"
-#include "vehicles/vehicle.h" 
-#include "game\Clock.h"
+#include "scene/world/GameWorld.h"
+#include "Peds/ped.h"
+#include "Vehicles/vehicle.h" 
+#include "game/Clock.h"
 #include "camera/CamInterface.h"
 #include "camera/debug/DebugDirector.h"
 
