diff --git a/src/dev_ng/game/task/Movement/TaskGotoPointAvoidance.cpp b/src/dev_ng/game/task/Movement/TaskGotoPointAvoidance.cpp
--- a/src/dev_ng/game/task/Movement/TaskGotoPointAvoidance.cpp
+++ b/src/dev_ng/game/task/Movement/TaskGotoPointAvoidance.cpp
@@ -3,12 +3,12 @@
 // 
 #include "Peds/PedDebugVisualiser.h"
 #include "Peds/PedIntelligence.h"
-#include "Task/Movement/TaskGotoPoint.h"
-#include "Task/Movement/TaskGotoPointAvoidance.h"
+#include "task/Movement/TaskGotoPoint.h"
+#include "task/Movement/TaskGotoPointAvoidance.h"
 
 
 // Rage headers
-#include "Math/angmath.h"
+#include "math/angmath.h"
 
 //#pragma optimize("", off)
 AI_OPTIMISATIONS()
