diff --git a/src/dev_ng/game/task/Movement/TaskMoveToTacticalPoint.h b/src/dev_ng/game/task/Movement/TaskMoveToTacticalPoint.h
--- a/src/dev_ng/game/task/Movement/TaskMoveToTacticalPoint.h
+++ b/src/dev_ng/game/task/Movement/TaskMoveToTacticalPoint.h
@@ -11,9 +11,9 @@
 #include "ai/AITarget.h"
 #include "scene/RegdRefTypes.h"
 #include "task/Movement/TaskNavBase.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/Tuning.h"
+#include "task/System/Task.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
 
 // Game forward declarations
 class CCoverPoint;
