diff --git a/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.cpp b/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.cpp
--- a/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.cpp
+++ b/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.cpp
@@ -1,10 +1,10 @@
-#include "Task/Movement/TaskGenericMoveToPoint.h"
+#include "task/Movement/TaskGenericMoveToPoint.h"
 
 
-#include "Task/Motion/TaskMotionBase.h"
-#include "Task/Movement/TaskGoToPoint.h"
-#include "Task/Movement/TaskNavMesh.h"
-#include "Task/Movement/TaskFlyToPoint.h"
+#include "task/Motion/TaskMotionBase.h"
+#include "task/Movement/TaskGotoPoint.h"
+#include "task/Movement/TaskNavMesh.h"
+#include "task/Movement/TaskFlyToPoint.h"
 #include "Peds/NavCapabilities.h"
 #include "Peds/PedIntelligence.h"
 
