diff --git a/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.h b/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.h
--- a/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.h
+++ b/src/dev_ng/game/task/Movement/TaskGenericMoveToPoint.h
@@ -7,10 +7,10 @@
 
 #include "TaskGoTo.h"
 
-#include "AI/AITarget.h"
-#include "Peds/PedDefines.h"
-#include "Task/System/Task.h"
-#include "Task/General/TaskBasic.h"
+#include "ai/AITarget.h"
+#include "Peds/pedDefines.h"
+#include "task/System/Task.h"
+#include "task/General/TaskBasic.h"
 
 
 //***************************************
