diff --git a/src/dev_ng/game/script/commands_path.cpp b/src/dev_ng/game/script/commands_path.cpp
--- a/src/dev_ng/game/script/commands_path.cpp
+++ b/src/dev_ng/game/script/commands_path.cpp
@@ -4,18 +4,18 @@
 
 // Game headers
 #include "control/gps.h"
-#include "Network/Network.h"
-#include "network/networkinterface.h"
+#include "network/Network.h"
+#include "network/NetworkInterface.h"
 #include "pathserver/pathserver.h"
-#include "peds/pedpopulation.h"
-#include "physics/WorldProbe/worldprobe.h"
-#include "scene/world/gameworld.h"
+#include "Peds/pedpopulation.h"
+#include "physics/WorldProbe/WorldProbe.h"
+#include "scene/world/GameWorld.h"
 #include "scene/world/gameworldheightmap.h"
 #include "script/script.h"
 #include "script/script_helper.h"
-#include "vehicleai/junctions.h"
-#include "vehicleai/pathfind.h"
-#include "vehicleai/task/TaskVehicleThreePointTurn.h"
+#include "vehicleAi/junctions.h"
+#include "vehicleAi/pathfind.h"
+#include "vehicleAi/task/TaskVehicleThreePointTurn.h"
 #include "ai/debug/system/AIDebugLogManager.h"
 
 SCRIPT_OPTIMISATIONS ()
