diff --git a/src/dev_ng/game/script/commands_cutscene.cpp b/src/dev_ng/game/script/commands_cutscene.cpp
--- a/src/dev_ng/game/script/commands_cutscene.cpp
+++ b/src/dev_ng/game/script/commands_cutscene.cpp
@@ -5,12 +5,12 @@
 #include "Cutscene/CutSceneManagerNew.h"
 #include "cutscene/AnimatedModelEntity.h"
 #include "script/script_helper.h"
-#include "peds/ped.h"
-#include "peds/rendering/PedVariationPack.h"
+#include "Peds/ped.h"
+#include "Peds/rendering/PedVariationPack.h"
 #include "Handlers/GameScriptEntity.h"
 #include "commands_entity.h"
 #include "cutscene/cutscene_channel.h"
-#include "Cutscene/CutSceneStore.h"
+#include "cutscene/CutSceneStore.h"
 
 ANIM_OPTIMISATIONS();
 
