diff --git a/src/dev_ng/game/script/commands_inventory.cpp b/src/dev_ng/game/script/commands_inventory.cpp
--- a/src/dev_ng/game/script/commands_inventory.cpp
+++ b/src/dev_ng/game/script/commands_inventory.cpp
@@ -1,13 +1,13 @@
 
 // Rage headers
-#include "script\wrapper.h"
+#include "script/wrapper.h"
 // Game headers
 #include "peds/Ped.h"
-#include "peds\pedintelligence.h"
+#include "peds/pedintelligence.h"
 #include "scene/world/GameWorld.h"
-#include "script\script.h"
-#include "vehicles\automobile.h"
-#include "Task\Default\TaskPlayer.h"
+#include "script/script.h"
+#include "vehicles/automobile.h"
+#include "Task/Default/TaskPlayer.h"
 
 namespace inventory_commands
 {
