diff --git a/src/dev_ng/game/script/commands_event.cpp b/src/dev_ng/game/script/commands_event.cpp
--- a/src/dev_ng/game/script/commands_event.cpp
+++ b/src/dev_ng/game/script/commands_event.cpp
@@ -5,14 +5,14 @@
 #include "script/wrapper.h"
 
 // Game headers
-#include "Event/Decision/EventDecisionMakerManager.h"
-#include "Event/EventShocking.h"
-#include "Event/ShockingEvents.h"
-#include "Peds/Ped.h"
+#include "event/decision/EventDecisionMakerManager.h"
+#include "event/EventShocking.h"
+#include "event/ShockingEvents.h"
+#include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
-#include "Script/Handlers/GameScriptResources.h"
-#include "Script/Script.h"
-#include "Script/Script_Channel.h"
+#include "script/Handlers/GameScriptResources.h"
+#include "script/script.h"
+#include "script/Script_Channel.h"
 #include "task/Response/Info/AgitatedManager.h"
 
 // Macro to disable optimisations if set
