diff --git a/src/dev_ng/game/script/streamedscripts.cpp b/src/dev_ng/game/script/streamedscripts.cpp
--- a/src/dev_ng/game/script/streamedscripts.cpp
+++ b/src/dev_ng/game/script/streamedscripts.cpp
@@ -11,13 +11,13 @@
 #include "system/timer.h"
 
 // Game headers
-#include "script\streamedscripts.h"
+#include "script/streamedscripts.h"
 #include "script/script_helper.h"
 #include "streaming/streaming.h"
 #include "streaming/packfilemanager.h"
 #include "streaming/streamingengine.h"
 #include "streaming/streamingmodule.h"
-#include "tools/smoketest.h"
+#include "tools/SmokeTest.h"
 #include "fwsys/fileExts.h" 
 
 PARAM(validatescripts, "[streamedscripts] validate all scripts for unrecognised native commands by loading them all");
