diff --git a/src/dev_ng/game/camera/helpers/FramePropagator.cpp b/src/dev_ng/game/camera/helpers/FramePropagator.cpp
--- a/src/dev_ng/game/camera/helpers/FramePropagator.cpp
+++ b/src/dev_ng/game/camera/helpers/FramePropagator.cpp
@@ -24,7 +24,7 @@
 #include "physics/WorldProbe/WorldProbe.h"
 #include "scene/DynamicEntity.h"
 #include "timeCycle/TimeCycle.h"
-#include "Renderer/RenderPhases/RenderPhaseLensDistortion.h"
+#include "renderer/RenderPhases/RenderPhaseLensDistortion.h"
 #include "input/headtracking.h"
 
 CAMERA_OPTIMISATIONS()
