diff --git a/src/dev_ng/game/vfx/misc/BrightLights.cpp b/src/dev_ng/game/vfx/misc/BrightLights.cpp
--- a/src/dev_ng/game/vfx/misc/BrightLights.cpp
+++ b/src/dev_ng/game/vfx/misc/BrightLights.cpp
@@ -26,16 +26,16 @@
 #include "vfx/vfxwidget.h"
 
 // Game headers
-#include "Camera/CamInterface.h"
-#include "Core/Game.h"
-#include "Renderer/Renderer.h"
-#include "Renderer/RenderThread.h"
+#include "camera/CamInterface.h"
+#include "Core/game.h"
+#include "renderer/Renderer.h"
+#include "renderer/RenderThread.h"
 #include "scene/lod/LodScale.h"
-#include "Shaders/ShaderLib.h"
-#include "TimeCycle/TimeCycle.h"
-#include "Timecycle/TimeCycleConfig.h"
-#include "Vfx/Misc/Coronas.h"
-#include "Vfx/Systems/VfxPed.h"
+#include "shaders/ShaderLib.h"
+#include "timecycle/TimeCycle.h"
+#include "timecycle/TimeCycleConfig.h"
+#include "vfx/misc/Coronas.h"
+#include "vfx/systems/VfxPed.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
