diff --git a/src/dev_ng/game/vfx/misc/DistantLights.h b/src/dev_ng/game/vfx/misc/DistantLights.h
--- a/src/dev_ng/game/vfx/misc/DistantLights.h
+++ b/src/dev_ng/game/vfx/misc/DistantLights.h
@@ -30,10 +30,10 @@
 
 // Framework headers
 #include "entity/archetypemanager.h"
-#include "fwmaths/Random.h"
+#include "fwmaths/random.h"
 
 // Game headers
-#include "VehicleAI/PathFind.h"
+#include "vehicleAi/pathfind.h"
 #include "vfx/VisualEffects.h"
 #include "DistantLights2.h"
 #include "DistantLightsVisibility.h"
