diff --git a/src/dev_ng/game/vfx/misc/Fire.h b/src/dev_ng/game/vfx/misc/Fire.h
--- a/src/dev_ng/game/vfx/misc/Fire.h
+++ b/src/dev_ng/game/vfx/misc/Fire.h
@@ -23,11 +23,11 @@
 #include "script/thread.h"
 
 // game
-#include "Audio/FireAudioEntity.h"
-#include "Network/General/NetworkFXId.h"
-#include "Pathserver/PathServer_DataTypes.h"
-#include "Scene/Entity.h"
-#include "Scene/RegdRefTypes.h"
+#include "audio/fireaudioentity.h"
+#include "network/General/NetworkFXId.h"
+#include "pathserver/PathServer_DataTypes.h"
+#include "scene/Entity.h"
+#include "scene/RegdRefTypes.h"
 #include "fwscene/search/Search.h"
 
 #endif // !__SPU
