diff --git a/src/dev_ng/game/vfx/misc/LensFlare.cpp b/src/dev_ng/game/vfx/misc/LensFlare.cpp
--- a/src/dev_ng/game/vfx/misc/LensFlare.cpp
+++ b/src/dev_ng/game/vfx/misc/LensFlare.cpp
@@ -5,8 +5,8 @@
 //
 
 // INCLUDES
-#include "Vfx/misc/LensFlare.h"
-#include "Vfx/misc/LensFlare_parser.h"
+#include "vfx/misc/LensFlare.h"
+#include "vfx/misc/LensFlare_parser.h"
 
 #if __BANK
 #include "bank/widget.h"
@@ -26,7 +26,7 @@
 #include "fwsys/timer.h"
 #include "fwmaths/vectorutil.h"
 #include "vfx/channel.h"
-#include "vfx/vfxhelper.h"
+#include "vfx/VfxHelper.h"
 #include "shaders/ShaderLib.h"
 #include "renderer/sprite2d.h"
 #include "renderer/PostProcessFX.h"
