diff --git a/src/dev_ng/game/vfx/misc/Markers.cpp b/src/dev_ng/game/vfx/misc/Markers.cpp
--- a/src/dev_ng/game/vfx/misc/Markers.cpp
+++ b/src/dev_ng/game/vfx/misc/Markers.cpp
@@ -31,24 +31,24 @@
 #include "vfx/vfxwidget.h"
 
 // game
-#include "Camera/CamInterface.h"
-#include "Control/GameLogic.h"
+#include "camera/CamInterface.h"
+#include "control/GameLogic.h"
 #include "Core/Game.h"
 #include "Cutscene/CutSceneManagerNew.h"
 #include "FrontEnd/MobilePhone.h"
-#include "ModelInfo/ModelInfo.h"
-#include "ModelInfo/ModelInfo_Factories.h"
+#include "modelinfo/ModelInfo.h"
+#include "modelinfo/ModelInfo_Factories.h"
 #include "Peds/PlayerInfo.h"
 #include "Peds/ped.h"
-#include "Renderer/PostProcessFX.h"
-#include "Renderer/PostProcessFXHelper.h"
-#include "Renderer/RenderTargets.h"
-#include "Renderer/Water.h"
-#include "Renderer/Util/Util.h"
-#include "Scene/World/GameWorld.h"
-#include "Streaming/Streaming.h"
-#include "Vfx/Particles/PtFxManager.h"
-#include "Vfx/VfxHelper.h"
+#include "renderer/PostProcessFX.h"
+#include "renderer/PostProcessFXHelper.h"
+#include "renderer/RenderTargets.h"
+#include "renderer/Water.h"
+#include "renderer/Util/Util.h"
+#include "scene/world/GameWorld.h"
+#include "streaming/Streaming.h"
+#include "vfx/Particles/PtFxManager.h"
+#include "vfx/VfxHelper.h"
 
 ///////////////////////////////////////////////////////////////////////////////
 //  OPTIMISATIONS - TURN ON IN OPTIMISATIONS.H
