diff --git a/src/dev_ng/game/vfx/misc/MovieManager.cpp b/src/dev_ng/game/vfx/misc/MovieManager.cpp
--- a/src/dev_ng/game/vfx/misc/MovieManager.cpp
+++ b/src/dev_ng/game/vfx/misc/MovieManager.cpp
@@ -63,11 +63,11 @@ namespace {
 #include "vfx/vfxwidget.h"
 
 // game
-#include "Core/Game.h"
-#include "Renderer/RenderThread.h"
-#include "System/Device_XContent.h"
-#include "System/FileMgr.h"
-#include "Camera/CamInterface.h"
+#include "Core/game.h"
+#include "renderer/RenderThread.h"
+#include "system/device_xcontent.h"
+#include "system/FileMgr.h"
+#include "camera/CamInterface.h"
 #include "vfx/misc/TVPlaylistManager.h"
 #include "control/replay/Misc/MoviePacket.h"
 
