diff --git a/src/dev_ng/game/Peds/Action/ActionManager.h b/src/dev_ng/game/Peds/Action/ActionManager.h
--- a/src/dev_ng/game/Peds/Action/ActionManager.h
+++ b/src/dev_ng/game/Peds/Action/ActionManager.h
@@ -22,12 +22,12 @@
 #include "vector/vector2.h"
 #include "atl/array.h"
 #include "scene/Entity.h"
-#include "peds/peddefines.h"
+#include "Peds/pedDefines.h"
 #include "system/control.h"
 #include "animation/AnimBones.h"
 #include "crskeleton/skeleton.h"
-#include "renderer\HierarchyIds.h"
-#include "vehicles\vehicle.h"
+#include "renderer/HierarchyIds.h"
+#include "Vehicles/vehicle.h"
 #include "atl/hashstring.h"
 #include "atl/array.h"
 #include "parser/macros.h"
