diff --git a/src/dev_ng/game/VS_Project/CMakeLists.txt b/src/dev_ng/game/VS_Project/CMakeLists.txt
new file mode 100644
--- /dev/null
+++ b/src/dev_ng/game/VS_Project/CMakeLists.txt
@@ -0,0 +1,63 @@
+string(TOLOWER game_win64_${GAME_CONFIG} BIN_NAME)
+project(${BIN_NAME})
+set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT ${PROJECT_NAME})
+
+set(TARGET_TYPE "Exec")
+include(${CMAKE_SOURCE_DIR}/cmake/BuildOptions.cmake)
+
+if (MSVC)
+    add_compile_options(/FI${CMAKE_SOURCE_DIR}/game/basetypes.h)
+    add_compile_options(/FI${CMAKE_SOURCE_DIR}/game/game_config.h)
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /SUBSYSTEM:WINDOWS /ENTRY:mainCRTStartup")
+    add_link_options(/MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /MACHINE:X64 /OPT:REF /INCREMENTAL:NO /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /OPT:ICF /ERRORREPORT:PROMPT /NOLOGO /TLBID:1)
+else()
+    add_compile_options(-include${CMAKE_SOURCE_DIR}/game/basetypes.h)
+    add_compile_options(-include${CMAKE_SOURCE_DIR}/game/game_config.h)
+    add_definitions(-Dlinux)
+    remove_definitions(-DWIN64)
+endif()
+
+set(SRC_FILES
+    "../Core/main.cpp"
+    "game.rc"
+)
+
+add_executable(${PROJECT_NAME} WIN32 ${SRC_FILES})
+
+target_include_directories(${PROJECT_NAME} PRIVATE
+    ${CMAKE_SOURCE_DIR}/game
+    ${CMAKE_SOURCE_DIR}/rage/3rdparty
+    ${CMAKE_SOURCE_DIR}/rage/3rdparty/AMD/AMD_ShadowFX/amd_lib/shared/common/inc
+    ${CMAKE_SOURCE_DIR}/rage/base/src
+    ${CMAKE_SOURCE_DIR}/rage/framework/src
+    ${CMAKE_SOURCE_DIR}/rage/naturalmotion/include
+    ${CMAKE_SOURCE_DIR}/rage/naturalmotion/src
+    ${CMAKE_SOURCE_DIR}/rage/script/src
+    ${CMAKE_SOURCE_DIR}/rage/suite/src
+)
+
+target_link_directories(${PROJECT_NAME} PRIVATE
+    "${CMAKE_SOURCE_DIR}/rage/lib"
+)
+
+target_link_libraries(${PROJECT_NAME} PRIVATE
+    game1_lib
+    game2_lib
+    game3_lib
+    game4_lib
+    NaturalMotion
+    network
+    RageAudio
+    RageCore
+    RageCreature
+    RageFramework
+    RageGraphics
+    RageMisc
+    RageNet
+    RagePhysics
+    RageScript
+    RageSec
+    RageSuiteCreature
+    ScaleformGfx
+)
+add_dependencies(${PROJECT_NAME} game_lib_psc)
