diff --git a/src/dev_ng/game/camera/syncedscene/SyncedSceneDirector.cpp b/src/dev_ng/game/camera/syncedscene/SyncedSceneDirector.cpp
--- a/src/dev_ng/game/camera/syncedscene/SyncedSceneDirector.cpp
+++ b/src/dev_ng/game/camera/syncedscene/SyncedSceneDirector.cpp
@@ -6,12 +6,12 @@
 
 #include "camera/syncedscene/SyncedSceneDirector.h"
 
-#include "camera/base/baseCamera.h"
+#include "camera/base/BaseCamera.h"
 #include "camera/cutscene/AnimatedCamera.h"
 #include "camera/system/CameraMetadata.h"
 #include "camera/system/CameraFactory.h"
-#include "fwAnimation/animhelpers.h"
-#include "fwAnimation/animdirector.h"
+#include "fwanimation/animhelpers.h"
+#include "fwanimation/animdirector.h"
 #include "fwanimation/directorcomponentsyncedscene.h"
 
 CAMERA_OPTIMISATIONS()
