diff --git a/src/dev_ng/game/Vehicles/Metadata/VehicleEntryPointInfo.h b/src/dev_ng/game/Vehicles/Metadata/VehicleEntryPointInfo.h
--- a/src/dev_ng/game/Vehicles/Metadata/VehicleEntryPointInfo.h
+++ b/src/dev_ng/game/Vehicles/Metadata/VehicleEntryPointInfo.h
@@ -5,15 +5,15 @@
 #include "atl/hashstring.h"
 #include "atl/string.h"
 #include "fwanimation/animdefines.h"
-#include "fwtl/pool.h"
+#include "fwtl/Pool.h"
 #include "fwutil/Flags.h"
 #include "parser/macros.h"
 #include "vector/vector3.h"
 
 // Game headers
 #include "ai/ambient/ConditionalAnims.h"
-#include "Renderer/HierarchyIds.h"
-#include "Task/Scenario/info/ScenarioInfoConditions.h"
+#include "renderer/HierarchyIds.h"
+#include "task/Scenario/info/ScenarioInfoConditions.h"
 #include "Vehicles/Metadata/VehicleLayoutInfo.h"
 #include "Vehicles/vehicle_channel.h"
 
