diff --git a/src/dev_ng/game/task/Movement/Climbing/TaskRappel.h b/src/dev_ng/game/task/Movement/Climbing/TaskRappel.h
--- a/src/dev_ng/game/task/Movement/Climbing/TaskRappel.h
+++ b/src/dev_ng/game/task/Movement/Climbing/TaskRappel.h
@@ -8,8 +8,8 @@
 //Game headers
 #include "parser/manager.h"
 #include "Peds/QueriableInterface.h"
-#include "Task/System/TaskFSMClone.h"
-#include "Task/System/Tuning.h"
+#include "task/System/TaskFSMClone.h"
+#include "task/System/Tuning.h"
 
 #include "diag/channel.h"
 
