diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotReachForWound.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotReachForWound.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotReachForWound.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotReachForWound.cpp
@@ -17,11 +17,11 @@
 #include "NmRsCharacter.h"
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 
 #include "NmRsCBU_TaskManager.h"
-#include "NmRsCBU_CatchFall.h"
-#include "NmRsCBU_Rollup.h"
+#include "NmRsCBU_Catchfall.h"
+#include "NmRsCBU_RollUp.h"
 
 namespace ART
 {
