diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Electrocute.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Electrocute.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Electrocute.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Electrocute.cpp
@@ -20,7 +20,7 @@
 #include "NmRsCharacter.h"
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 
 #include "NmRsCBU_TaskManager.h"
 
