diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SoftKeyframe.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SoftKeyframe.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SoftKeyframe.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SoftKeyframe.cpp
@@ -23,7 +23,7 @@
 #include "NmRsCharacter.h"
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 #include "nmutils/TypeUtils.h"
 //#define LOADCRAWLFROMFILE
 
