diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SmartFall.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SmartFall.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SmartFall.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_SmartFall.cpp
@@ -26,7 +26,7 @@
 #include "NmRsCharacter.h"
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 
 #include "NmRsCBU_TaskManager.h"
 #include "NmRsCBU_HeadLook.h"
