diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Flinch.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Flinch.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Flinch.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_Flinch.cpp
@@ -23,7 +23,7 @@
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
 #include "NmRsEngine.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 
 #include "NmRsCBU_TaskManager.h"
 #include "NmRsCBU_SpineTwist.h" 
