diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCharacterMasking.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCharacterMasking.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCharacterMasking.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCharacterMasking.cpp
@@ -19,7 +19,7 @@
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
 
-#include "art/MessageParams.h"
+#include "art/messageparams.h"
 
 namespace ART
 {
