diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotJustWhenFallen.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotJustWhenFallen.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotJustWhenFallen.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_ShotJustWhenFallen.cpp
@@ -17,14 +17,14 @@
 #include "NmRsCharacter.h"
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 
 #include "NmRsCBU_TaskManager.h"
 #include "NmRsCBU_BalancerCollisionsReaction.h"
 #include "NmRsCBU_DynamicBalancer.h"
 #include "NmRsCBU_RollUp.h"
 #include "NmRsCBU_RollDownStairs.h"
-#include "NmRsCBU_CatchFall.h"
+#include "NmRsCBU_Catchfall.h"
 #include "NmRsCBU_SmartFall.h"
 
 namespace ART
