diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_FallOverWall.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_FallOverWall.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_FallOverWall.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_FallOverWall.cpp
@@ -26,7 +26,7 @@
 #include "NmRsCBU_Catchfall.h" 
 #endif//useNewFallOverWall
 
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 
 namespace ART
 {
