diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsUtils.h b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsUtils.h
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsUtils.h
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsUtils.h
@@ -22,7 +22,7 @@
 #ifndef NM_RS_UTILS_H
 #define NM_RS_UTILS_H
 
-#include "NMutils/NMTypes.h"
+#include "nmutils/nmtypes.h"
 #include "physics/inst.h"
 #include "vector/geometry.h"
 
