diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_RollOver.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_RollOver.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_RollOver.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_RollOver.cpp
@@ -31,7 +31,7 @@
 #include "NmRsCharacter.h"
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 
 #include "NmRsCBU_TaskManager.h"
 #include "NmRsCBU_BodyFoetal.h" 
