diff --git a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_OnFire.cpp b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_OnFire.cpp
--- a/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_OnFire.cpp
+++ b/src/dev_ng/rage/naturalmotion/src/rockstar/NmRsCBU_OnFire.cpp
@@ -16,7 +16,7 @@
 #include "NmRsCBU_OnFire.h"
 #include "NmRsEffectors.h"
 #include "NmRsGenericPart.h"
-#include "ART/ARTFeedback.h"
+#include "art/artfeedback.h"
 #include "NmRsCBU_TaskManager.h"
 
 #include "NmRsCBU_ArmsWindmill.h"
