diff --git a/src/dev_ng/game/task/Crimes/TaskReactToPursuit.cpp b/src/dev_ng/game/task/Crimes/TaskReactToPursuit.cpp
--- a/src/dev_ng/game/task/Crimes/TaskReactToPursuit.cpp
+++ b/src/dev_ng/game/task/Crimes/TaskReactToPursuit.cpp
@@ -5,31 +5,31 @@
 //
 
 // File header
-#include "Task/crimes/TaskReactToPursuit.h"
+#include "task/Crimes/TaskReactToPursuit.h"
 
 // Rage Headers
 #include "math/angmath.h"
 
 // Game Headers
-#include "Event/System/EventData.h"
-#include "Game/Dispatch/IncidentManager.h"
-#include "Peds/Ped.h"
+#include "event/system/EventData.h"
+#include "game/Dispatch/IncidentManager.h"
+#include "Peds/ped.h"
 #include "Peds/PedFactory.h"
 #include "Peds/PedIntelligence.h"
-#include "Task/Crimes/TaskPursueCriminal.h"
+#include "task/Crimes/TaskPursueCriminal.h"
 #include "task/Combat/TaskNewCombat.h"
-#include "Task/Combat/TaskThreatResponse.h"
-#include "Task/General/TaskBasic.h"
-#include "Task/Movement/TaskGotoPoint.h"
-#include "Task/Response/TaskFlee.h"
-#include "Task/Service/Police/TaskPolicePatrol.h"
-#include "Task/Vehicle/TaskCar.h"
-#include "Task/Vehicle/TaskEnterVehicle.h"
-#include "Task/Vehicle/TaskExitVehicle.h"
+#include "task/Combat/TaskThreatResponse.h"
+#include "task/General/TaskBasic.h"
+#include "task/Movement/TaskGotoPoint.h"
+#include "task/Response/TaskFlee.h"
+#include "task/Service/Police/TaskPolicePatrol.h"
+#include "task/Vehicle/TaskCar.h"
+#include "task/Vehicle/TaskEnterVehicle.h"
+#include "task/Vehicle/TaskExitVehicle.h"
 #include "vehicleAi/VehicleIntelligence.h"
-#include "VehicleAi/Task/TaskVehicleCruise.h"
-#include "VehicleAi/Task/TaskVehicleGoTo.h"
-#include "vehicleAi/Task/TaskVehiclePark.h"
+#include "vehicleAi/task/TaskVehicleCruise.h"
+#include "vehicleAi/task/TaskVehicleGoTo.h"
+#include "vehicleAi/task/TaskVehiclePark.h"
 
 AI_OPTIMISATIONS()
 
