diff --git a/src/dev_ng/game/Objects/DummyObject.cpp b/src/dev_ng/game/Objects/DummyObject.cpp
--- a/src/dev_ng/game/Objects/DummyObject.cpp
+++ b/src/dev_ng/game/Objects/DummyObject.cpp
@@ -3,7 +3,7 @@
 // Started	:	20/04/2000
 
 // Framework headers
-#include "fwscene/world/worldmgr.h"
+#include "fwscene/world/WorldMgr.h"
 #include "fwscene/stores/mapdatastore.h"
 
 // Rage headers
@@ -14,8 +14,8 @@
 // Game headers 
 #include "DummyObject.h"
 #include "Door.h"
-#include "Object.h"
-#include "PathServer/ExportCollision.h"
+#include "object.h"
+#include "pathserver/ExportCollision.h"
 #include "scene/world/GameWorld.h"
 #include "script/script.h"
 #include "script/script_brains.h"
