diff --git a/src/dev_ng/rage/base/src/system/buddyallocator_config.h b/src/dev_ng/rage/base/src/system/buddyallocator_config.h
--- a/src/dev_ng/rage/base/src/system/buddyallocator_config.h
+++ b/src/dev_ng/rage/base/src/system/buddyallocator_config.h
@@ -9,7 +9,7 @@
 
 #define RESOURCE_POOLS 0		// was PS3 || XENON
 
-#define ENABLE_BUDDY_ALLOCATOR RSG_PC
+#define ENABLE_BUDDY_ALLOCATOR (RSG_PC || RSG_LINUX)
 
 #if defined(__RGSC_DLL) && __RGSC_DLL
 	// the sparse allocator breaks the social club DLL
