diff --git a/src/dev_ng/rage/base/src/system/cache.h b/src/dev_ng/rage/base/src/system/cache.h
--- a/src/dev_ng/rage/base/src/system/cache.h
+++ b/src/dev_ng/rage/base/src/system/cache.h
@@ -13,9 +13,9 @@
 #include <spu_intrinsics.h>
 #elif __PPU
 #include <ppu_intrinsics.h>
-#elif __WIN32PC
+#elif __WIN32PC && !__LINUX
 #include <emmintrin.h>
-#elif RSG_ORBIS
+#elif RSG_ORBIS || RSG_LUNUX
 #include <emmintrin.h>
 #include <xmmintrin.h>
 #elif RSG_DURANGO
