diff --git a/src/dev_ng/rage/base/src/system/namedpipe.cpp b/src/dev_ng/rage/base/src/system/namedpipe.cpp
--- a/src/dev_ng/rage/base/src/system/namedpipe.cpp
+++ b/src/dev_ng/rage/base/src/system/namedpipe.cpp
@@ -13,7 +13,7 @@
 #include "system/ipc.h"
 #include "system/timer.h"
 
-#if __PPU || RSG_ORBIS
+#if __PPU || RSG_ORBIS || RSG_LINUX
 #include <netinet/in.h>
 #include <netinet/tcp.h>
 #include <sys/socket.h>
@@ -26,7 +26,7 @@
 #include "grprofile/pix.h" // for PROFILE_BUILD  define before xtl.h include. makes unity builds happy.
 #include "system/xtl.h"
 
-#elif __WIN32PC
+#elif __WIN32PC && !__LINUX
 
 #define STRICT
 #pragma warning(disable: 4668)
@@ -150,7 +150,7 @@ void sysNamedPipe::Close() {
 #define TCP_NODELAY SCE_NET_TCP_NODELAY
 #endif
 
-#if RSG_ORBIS
+#if RSG_ORBIS || RSG_LINUX
 #define SOCKET int
 #define SOCKET_ERROR -1
 #endif
