diff --git a/src/dev_ng/game/camera/cinematic/context/CinematicFallFromHeliContext.cpp b/src/dev_ng/game/camera/cinematic/context/CinematicFallFromHeliContext.cpp
--- a/src/dev_ng/game/camera/cinematic/context/CinematicFallFromHeliContext.cpp
+++ b/src/dev_ng/game/camera/cinematic/context/CinematicFallFromHeliContext.cpp
@@ -6,11 +6,11 @@
 
 #include "camera/cinematic/context/CinematicFallFromHeliContext.h"
 #include "camera/system/CameraMetadata.h"
-#include "peds/ped.h"
-#include "peds/PedIntelligence.h"
+#include "Peds/ped.h"
+#include "Peds/PedIntelligence.h"
 #include "task/Combat/TaskDamageDeath.h"
 #include "task/Vehicle/TaskExitVehicle.h"
-#include "vehicles/metadata/VehicleSeatInfo.h"
+#include "Vehicles/Metadata/VehicleSeatInfo.h"
 
 INSTANTIATE_RTTI_CLASS(camCinematicFallFromHeliContext,0x13E7C9E9)
 
