diff --git a/src/dev_ng/game/weapons/gadgets/GadgetJetpack.h b/src/dev_ng/game/weapons/gadgets/GadgetJetpack.h
--- a/src/dev_ng/game/weapons/gadgets/GadgetJetpack.h
+++ b/src/dev_ng/game/weapons/gadgets/GadgetJetpack.h
@@ -2,8 +2,8 @@
 #define INC_GADGET_JETPACK
 
 // Game headers
-#include "Weapons/Gadgets/Gadget.h"
-#include "Task/Movement/JetpackObject.h"
+#include "weapons/gadgets/Gadget.h"
+#include "task/Movement/JetpackObject.h"
 
 #if ENABLE_JETPACK
 
