diff --git a/src/dev_ng/game/weapons/gadgets/GadgetParachute.h b/src/dev_ng/game/weapons/gadgets/GadgetParachute.h
--- a/src/dev_ng/game/weapons/gadgets/GadgetParachute.h
+++ b/src/dev_ng/game/weapons/gadgets/GadgetParachute.h
@@ -2,7 +2,7 @@
 #define GADGET_PARACHUTE_H
 
 // Game headers
-#include "Weapons/Gadgets/Gadget.h"
+#include "weapons/Gadgets/Gadget.h"
 
 //////////////////////////////////////////////////////////////////////////
 // CGadgetParachute
