diff --git a/src/dev_ng/game/animation/AnimScene/events/AnimSceneEvents.h b/src/dev_ng/game/animation/AnimScene/events/AnimSceneEvents.h
--- a/src/dev_ng/game/animation/AnimScene/events/AnimSceneEvents.h
+++ b/src/dev_ng/game/animation/AnimScene/events/AnimSceneEvents.h
@@ -6,7 +6,7 @@
 #include "streaming/streamingrequest.h"
 
 // game includes
-#include "animation\AnimScene\AnimScene.h"
+#include "animation/AnimScene/AnimScene.h"
 #include "animation/AnimScene/AnimSceneEvent.h"
 #include "animation/AnimScene/AnimSceneHelpers.h"
 #include "peds/PedMotionState.h"
