diff --git a/src/dev_ng/game/animation/AnimScene/events/AnimSceneEventInitialisers.cpp b/src/dev_ng/game/animation/AnimScene/events/AnimSceneEventInitialisers.cpp
--- a/src/dev_ng/game/animation/AnimScene/events/AnimSceneEventInitialisers.cpp
+++ b/src/dev_ng/game/animation/AnimScene/events/AnimSceneEventInitialisers.cpp
@@ -1,10 +1,10 @@
 #if __BANK
 
 #include "AnimSceneEventInitialisers.h"
-#include "AnimSceneEventInitialisers_Parser.h"
+#include "AnimSceneEventInitialisers_parser.h"
 
 #include "animation/AnimScene/AnimSceneEditor.h"
-#include "animation/animscene/AnimSceneEvent.h"
+#include "animation/AnimScene/AnimSceneEvent.h"
 #include "animation/debug/AnimDebug.h"
 
 
