diff --git a/src/dev_ng/game/camera/cinematic/shot/CinematicFallFromHeliShot.cpp b/src/dev_ng/game/camera/cinematic/shot/CinematicFallFromHeliShot.cpp
--- a/src/dev_ng/game/camera/cinematic/shot/CinematicFallFromHeliShot.cpp
+++ b/src/dev_ng/game/camera/cinematic/shot/CinematicFallFromHeliShot.cpp
@@ -9,11 +9,11 @@
 #include "camera/cinematic/camera/mounted/CinematicMountedCamera.h"
 #include "camera/cinematic/camera/mounted/CinematicMountedPartCamera.h"
 #include "camera/cinematic/context/CinematicFallFromHeliContext.h"
-#include "camera\cinematic\camera\tracking\CinematicVehicleTrackingCamera.h"
-#include "peds/ped.h"
-#include "peds/PedIntelligence.h"
-#include "vehicles/Metadata/VehicleEntryPointInfo.h"
-#include "vehicles/trailer.h"
+#include "camera/cinematic/camera/tracking/CinematicVehicleTrackingCamera.h"
+#include "Peds/ped.h"
+#include "Peds/PedIntelligence.h"
+#include "Vehicles/Metadata/VehicleEntryPointInfo.h"
+#include "Vehicles/Trailer.h"
 #include "task/Vehicle/TaskExitVehicle.h"
 
 
