diff --git a/src/dev_ng/game/debug/TextureViewer/TextureViewerDebugging.cpp b/src/dev_ng/game/debug/TextureViewer/TextureViewerDebugging.cpp
--- a/src/dev_ng/game/debug/TextureViewer/TextureViewerDebugging.cpp
+++ b/src/dev_ng/game/debug/TextureViewer/TextureViewerDebugging.cpp
@@ -3,7 +3,7 @@
 // (c) 2010 RockstarNorth
 // ==============================================
 
-#include "debug/textureviewer/textureviewerprivate.h"
+#include "debug/TextureViewer/TextureViewerPrivate.h"
 
 #if __BANK
 
@@ -20,14 +20,14 @@
 #include "fwscene/stores/mapdatastore.h"
 
 // Game headers
-#include "core/game.h"
+#include "Core/game.h"
 #include "debug/DebugArchetypeProxy.h"
-#include "modelinfo/modelinfo.h"
+#include "modelinfo/ModelInfo.h"
 #include "scene/loader/MapData.h"
 
-#include "debug/textureviewer/textureviewersearch.h"
-#include "debug/textureviewer/textureviewerutil.h"
-#include "debug/textureviewer/textureviewerdebugging.h"
+#include "debug/TextureViewer/TextureViewersearch.h"
+#include "debug/TextureViewer/TextureViewerutil.h"
+#include "debug/TextureViewer/TextureViewerdebugging.h"
 
 DEBUG_TEXTURE_VIEWER_OPTIMISATIONS()
 
