diff --git a/src/dev_ng/game/debug/TextureViewer/TextureViewerEntry.h b/src/dev_ng/game/debug/TextureViewer/TextureViewerEntry.h
--- a/src/dev_ng/game/debug/TextureViewer/TextureViewerEntry.h
+++ b/src/dev_ng/game/debug/TextureViewer/TextureViewerEntry.h
@@ -6,13 +6,13 @@
 #ifndef _DEBUG_TEXTUREVIEWER_TEXTUREVIEWERENTRY_H_
 #define _DEBUG_TEXTUREVIEWER_TEXTUREVIEWERENTRY_H_
 
-#include "debug/textureviewer/textureviewerprivate.h"
+#include "debug/TextureViewer/TextureViewerPrivate.h"
 
 #if __BANK
 
 #include "atl/string.h"                              // atString
 #include "paging/ref.h"                              // pgRef
-#include "debug/textureviewer/textureviewersearch.h" // CAssetRef,CTxdRef,CTextureRef
+#include "debug/TextureViewer/TextureViewersearch.h" // CAssetRef,CTxdRef,CTextureRef
 #include "scene/RegdRefTypes.h"                      // RegdEnt
 
 namespace rage { template <typename,int,typename> class atArray; }
