diff --git a/src/dev_ng/game/Peds/rendering/PedHeadshotManager.cpp b/src/dev_ng/game/Peds/rendering/PedHeadshotManager.cpp
--- a/src/dev_ng/game/Peds/rendering/PedHeadshotManager.cpp
+++ b/src/dev_ng/game/Peds/rendering/PedHeadshotManager.cpp
@@ -26,9 +26,9 @@
 #include "bank/bkmgr.h"
 #include "bank/bank.h"
 #include "bank/combo.h"
-#include "crSkeleton/Skeleton.h"
-#include "crSkeleton/SkeletonData.h"
-#include "crSkeleton/BoneData.h"
+#include "crskeleton/skeleton.h"
+#include "crskeleton/SkeletonData.h"
+#include "crskeleton/BoneData.h"
 #include "streaming/streamingengine.h"
 #include "system/memory.h"
 
@@ -41,21 +41,21 @@
 #include "fwdrawlist/drawlist.h"
 
 // Game headers:
-#include "animation/animBones.h"
+#include "animation/AnimBones.h"
 #include "debug/MarketingTools.h"
 #include "camera/CamInterface.h"
-#include "Network/NetworkInterface.h"
+#include "network/NetworkInterface.h"
 #include "Peds/ped.h"
 #include "Peds/PedFactory.h"
 #include "Peds/PlayerPed.h"
 #include "Peds/rendering/pedVariationPack.h"
-#include "peds/rendering/PedVariationDebug.h"
-#include "renderer/DrawLists/drawlistMgr.h"
+#include "Peds/rendering/PedVariationDebug.h"
+#include "renderer/DrawLists/drawListMgr.h"
 #include "scene/EntityTypes.h"
-#include "scene/world/gameWorld.h"
+#include "scene/world/GameWorld.h"
 #include "script/thread.h"
 #include "script/script.h"
-#include "Shaders/CustomShaderEffectPed.h"
+#include "shaders/CustomShaderEffectPed.h"
 #include "Stats/StatsInterface.h"
 #include "streaming/streaming.h"
 #include "system/interlocked.h"
@@ -69,7 +69,7 @@
 #include "renderer/Util/ShaderUtil.h"
 #include "renderer/Util/Util.h"	 
 #include "renderer/Lights/lights.h"
-#include "renderer/Sprite2d.h"
+#include "renderer/sprite2d.h"
 #include "renderer/PostProcessFX.h"
 
 #if __XENON
