diff --git a/src/dev_ng/game/Peds/rendering/PedProps.cpp b/src/dev_ng/game/Peds/rendering/PedProps.cpp
--- a/src/dev_ng/game/Peds/rendering/PedProps.cpp
+++ b/src/dev_ng/game/Peds/rendering/PedProps.cpp
@@ -1,16 +1,16 @@
 //rage headers
 #if !__SPU
 
-#include "Peds\rendering\pedprops.h"
+#include "Peds/rendering/PedProps.h"
 
 #if __BANK
-	#include "bank\bkmgr.h"
-	#include "bank\bank.h"
-	#include "bank\button.h"
-	#include "bank\combo.h"
-	#include "bank\slider.h"
-	#include "bank\toggle.h"
-	#include "bank\text.h"
+	#include "bank/bkmgr.h"
+	#include "bank/bank.h"
+	#include "bank/button.h"
+	#include "bank/combo.h"
+	#include "bank/slider.h"
+	#include "bank/toggle.h"
+	#include "bank/text.h"
 #endif //__BANK
 #include "crskeleton/skeleton.h"
 #include "crskeleton/skeletondata.h"
@@ -22,22 +22,22 @@
 
 //game headers
 #include "animation/CreatureMetadata.h"
-#include "Control/Replay/Replay.h"
-#include "ModelInfo/PedModelInfo.h"
-#include "Peds\ped.h"
-#include "Peds\rendering\pedVariationPack.h"
-#include "Peds\rendering\pedVariationStream.h"
-#include "peds/rendering/PedVariationDebug.h"
-#include "peds/rendering/pedvariationstream.h"
-#include "scene\fileLoader.h"
-#include "scene\world\gameworld.h"
+#include "control/replay/replay.h"
+#include "modelinfo/PedModelInfo.h"
+#include "Peds/ped.h"
+#include "Peds/rendering/PedVariationPack.h"
+#include "Peds/rendering/PedVariationStream.h"
+#include "Peds/rendering/PedVariationDebug.h"
+#include "Peds/rendering/PedVariationStream.h"
+#include "scene/FileLoader.h"
+#include "scene/world/GameWorld.h"
 #include "script/script.h"
 #include "shaders/CustomShaderEffectPed.h"
 #include "streaming/ScriptPreload.h"
-#include "streaming\streaming.h"
-#include "streaming\streamingmodule.h"
-#include "system\filemgr.h"
-#include "renderer\renderer.h"
+#include "streaming/streaming.h"
+#include "streaming/streamingmodule.h"
+#include "system/FileMgr.h"
+#include "renderer/Renderer.h"
 #include "scene/ExtraMetadataMgr.h"
 #if GTA_REPLAY
 #include "control/replay/ReplayExtensions.h"
@@ -235,7 +235,7 @@ u8 CPedPropInfo::GetMaxNumPropsTex(eAnchorPoints anchorPointIdx, u32 propId) con
 #endif//__BANK
 
 #else  //!__SPU
-	#include "Peds\pedprops.h"
+	#include "Peds/pedprops.h"
 #endif //__SPU
 
 // EJ: Memory Optimization
