diff --git a/src/dev_ng/game/task/Physics/TaskNMOnFire.h b/src/dev_ng/game/task/Physics/TaskNMOnFire.h
--- a/src/dev_ng/game/task/Physics/TaskNMOnFire.h
+++ b/src/dev_ng/game/task/Physics/TaskNMOnFire.h
@@ -8,7 +8,7 @@
 
 // Game headers
 #include "Task/System/Task.h"
-#include "Task\System\TaskTypes.h"
+#include "Task/System/TaskTypes.h"
 #include "Task/Physics/TaskNM.h"
 
 // ------------------------------------------------------------------------------
