diff --git a/src/dev_ng/game/task/Physics/TaskNMDrunk.cpp b/src/dev_ng/game/task/Physics/TaskNMDrunk.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMDrunk.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMDrunk.cpp
@@ -1,28 +1,28 @@
 // Filename   :	TaskNMDrunk.h
 // Description:	Code version of original script drunk routine.
 
-#include "Task/Physics/TaskNMDrunk.h"
+#include "task/Physics/TaskNMDrunk.h"
 
 #if ENABLE_DRUNK
 
 //---- Include Files ---------------------------------------------------------------------------------------------------------------------------------
 // Rage headers
-#include "crskeleton\Skeleton.h"
-#include "fragment\Cache.h"
-#include "fragment\Instance.h"
-#include "fragment\Type.h"
-#include "fragment\TypeChild.h"
-#include "fragmentnm\messageparams.h"
+#include "crskeleton/skeleton.h"
+#include "fragment/Cache.h"
+#include "fragment/instance.h"
+#include "fragment/Type.h"
+#include "fragment/TypeChild.h"
+#include "fragmentnm/messageparams.h"
 #include "pharticulated/articulatedcollider.h"
 #include "physics/shapetest.h"
 
 // Framework headers
-//#include "fwmaths\Angle.h"
+//#include "fwmaths/angle.h"
 
 // Game headers
-#include "debug\DebugScene.h"
+#include "debug/DebugScene.h"
 #include "Peds/ped.h"
-#include "Task/motion/TaskMotionBase.h"
+#include "task/Motion/TaskMotionBase.h"
 //----------------------------------------------------------------------------------------------------------------------------------------------------
 
 AI_OPTIMISATIONS()
