diff --git a/src/dev_ng/game/task/Physics/TaskNMDangle.cpp b/src/dev_ng/game/task/Physics/TaskNMDangle.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMDangle.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMDangle.cpp
@@ -6,15 +6,15 @@
 
 // C headers
 // Rage headers
-#include "crskeleton\Skeleton.h"
-#include "fragment\Cache.h"
-#include "fragment\Instance.h"
-#include "fragment\Type.h"
-#include "fragment\TypeChild.h"
-#include "fragmentnm\messageparams.h"
+#include "crskeleton/skeleton.h"
+#include "fragment/cache.h"
+#include "fragment/instance.h"
+#include "fragment/type.h"
+#include "fragment/typechild.h"
+#include "fragmentnm/messageparams.h"
 #include "pharticulated/articulatedcollider.h"
 #include "physics/shapetest.h"
-#include "Physics/constrainthinge.h"
+#include "physics/constrainthinge.h"
 #include "physics/simulator.h"
 #include "phbound/boundcomposite.h"
 
@@ -22,10 +22,10 @@
 #include "grcore/debugdraw.h"
 
 // Game headers
-#include "Peds\Ped.h"
-#include "Task/Physics/NmDebug.h"
-#include "Task/Physics/TaskNMDangle.h"
-#include "Physics/physics.h"
+#include "Peds/ped.h"
+#include "task/Physics/NmDebug.h"
+#include "task/Physics/TaskNMDangle.h"
+#include "physics/physics.h"
 #include "physics/RagdollConstraints.h"
 
 AI_OPTIMISATIONS()
