diff --git a/src/dev_ng/game/task/Physics/TaskNMDraggingToSafety.cpp b/src/dev_ng/game/task/Physics/TaskNMDraggingToSafety.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMDraggingToSafety.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMDraggingToSafety.cpp
@@ -5,7 +5,7 @@
 // --- Include Files ------------------------------------------------------------
 
 // Rage headers
-#include "pharticulated\articulatedcollider.h"
+#include "pharticulated/articulatedcollider.h"
 #include "phbound/boundcomposite.h"
 #include "physics/constraintdistance.h"
 #include "physics/constraintfixed.h"
@@ -13,7 +13,7 @@
 // Game headers
 #include "Peds/ped.h"
 #include "physics/physics.h"
-#include "Task/Physics/TaskNMDraggingToSafety.h"
+#include "task/Physics/TaskNMDraggingToSafety.h"
 
 AI_OPTIMISATIONS()
 
