diff --git a/src/dev_ng/game/task/Physics/TaskNMBuoyancy.cpp b/src/dev_ng/game/task/Physics/TaskNMBuoyancy.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMBuoyancy.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMBuoyancy.cpp
@@ -3,23 +3,23 @@
 
 //---- Include Files ---------------------------------------------------------------------------------------------------------------------------------
 // Rage headers
-#include "crskeleton\Skeleton.h"
-#include "fragment\Cache.h"
-#include "fragment\Instance.h"
-#include "fragment\Type.h"
-#include "fragmentnm\messageparams.h"
+#include "crskeleton/Skeleton.h"
+#include "fragment/Cache.h"
+#include "fragment/Instance.h"
+#include "fragment/Type.h"
+#include "fragmentnm/messageparams.h"
 
 // Framework headers
 #include "grcore/debugdraw.h"
 
 // Game headers
 #include "Peds/ped.h"
-#include "Task/Motion/Locomotion/TaskInWater.h"
-#include "Task/Physics/TaskNMBuoyancy.h"
-#include "Task/Physics/TaskNMRiverRapids.h"
-#include "Task/Physics/TaskNMShot.h"
-#include "Task/Physics/NmDebug.h"
-#include "Renderer/Water.h"
+#include "task/Motion/Locomotion/TaskInWater.h"
+#include "task/Physics/TaskNMBuoyancy.h"
+#include "task/Physics/TaskNMRiverRapids.h"
+#include "task/Physics/TaskNMShot.h"
+#include "task/Physics/NmDebug.h"
+#include "renderer/Water.h"
 
 //----------------------------------------------------------------------------------------------------------------------------------------------------
 
