diff --git a/src/dev_ng/game/task/Physics/TaskNMStumble.cpp b/src/dev_ng/game/task/Physics/TaskNMStumble.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMStumble.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMStumble.cpp
@@ -5,12 +5,12 @@
 
 //---- Include Files ---------------------------------------------------------------------------------------------------------------------------------
 // Rage headers
-#include "crskeleton\Skeleton.h"
-#include "fragment\Cache.h"
-#include "fragment\Instance.h"
-#include "fragment\Type.h"
-#include "fragment\TypeChild.h"
-#include "fragmentnm\messageparams.h"
+#include "crskeleton/Skeleton.h"
+#include "fragment/Cache.h"
+#include "fragment/Instance.h"
+#include "fragment/Type.h"
+#include "fragment/TypeChild.h"
+#include "fragmentnm/messageparams.h"
 #include "pharticulated/articulatedcollider.h"
 #include "phbound/boundcomposite.h"
 #include "physics/shapetest.h"
@@ -18,15 +18,15 @@
 
 // Framework headers
 #include "grcore/debugdraw.h"
-//#include "fwmaths\Angle.h"
+//#include "fwmaths/angle.h"
 
 // Game headers
 #include "Peds/ped.h"
-#include "Task/Physics/TaskNMStumble.h"
-#include "Task/Physics/NmDebug.h"
-#include "Weapons/Components/WeaponComponentClip.h"
-#include "Weapons/Components/WeaponComponentScope.h"
-#include "Weapons/Components/WeaponComponentSuppressor.h"
+#include "task/Physics/TaskNMStumble.h"
+#include "task/Physics/NmDebug.h"
+#include "weapons/components/WeaponComponentClip.h"
+#include "weapons/components/WeaponComponentScope.h"
+#include "weapons/components/WeaponComponentSuppressor.h"
 
 //----------------------------------------------------------------------------------------------------------------------------------------------------
 
