diff --git a/src/dev_ng/game/task/Physics/TaskNMShot.h b/src/dev_ng/game/task/Physics/TaskNMShot.h
--- a/src/dev_ng/game/task/Physics/TaskNMShot.h
+++ b/src/dev_ng/game/task/Physics/TaskNMShot.h
@@ -7,11 +7,11 @@
 // --- Include Files ------------------------------------------------------------
 
 // Game headers
-#include "Task/System/Task.h"
-#include "Task\System\TaskTypes.h"
-#include "Task/Physics/TaskNM.h"
-#include "Task/Physics/TaskNMBrace.h"
-#include "Task/Physics/ThreatenedHelper.h"
+#include "task/System/Task.h"
+#include "task/System/TaskTypes.h"
+#include "task/Physics/TaskNM.h"
+#include "task/Physics/TaskNMBrace.h"
+#include "task/Physics/ThreatenedHelper.h"
 
 // ------------------------------------------------------------------------------
 
