diff --git a/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp b/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp
@@ -3,13 +3,13 @@
 
 //---- Include Files ---------------------------------------------------------------------------------------------------------------------------------
 // Rage headers
-#include "crskeleton\Skeleton.h"
-#include "fragment\Cache.h"
-#include "fragment\Instance.h"
-#include "fragment\Type.h"
-#include "fragment\TypeChild.h"
-#include "fragmentnm\messageparams.h"
-#include "pharticulated\articulatedbody.h"
+#include "crskeleton/Skeleton.h"
+#include "fragment/Cache.h"
+#include "fragment/Instance.h"
+#include "fragment/Type.h"
+#include "fragment/TypeChild.h"
+#include "fragmentnm/messageparams.h"
+#include "pharticulated/articulatedbody.h"
 
 // Framework headers
 #include "grcore/debugdraw.h"
