diff --git a/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp b/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMInjuredOnGround.cpp
@@ -3,13 +3,13 @@
 
 //---- Include Files ---------------------------------------------------------------------------------------------------------------------------------
 // Rage headers
-#include "crskeleton\Skeleton.h"
-#include "fragment\Cache.h"
-#include "fragment\Instance.h"
-#include "fragment\Type.h"
-#include "fragment\TypeChild.h"
-#include "fragmentnm\messageparams.h"
-#include "pharticulated\articulatedbody.h"
+#include "crskeleton/skeleton.h"
+#include "fragment/Cache.h"
+#include "fragment/instance.h"
+#include "fragment/Type.h"
+#include "fragment/TypeChild.h"
+#include "fragmentnm/messageparams.h"
+#include "pharticulated/articulatedbody.h"
 
 // Framework headers
 #include "grcore/debugdraw.h"
@@ -17,8 +17,8 @@
 // Game headers
 #include "event/Events.h"
 #include "Peds/ped.h"
-#include "Task/Physics/TaskNMInjuredOnGround.h"
-#include "Task/Physics/NmDebug.h"
+#include "task/Physics/TaskNMInjuredOnGround.h"
+#include "task/Physics/NmDebug.h"
 
 //----------------------------------------------------------------------------------------------------------------------------------------------------
 
