diff --git a/src/dev_ng/game/task/Physics/TaskNMPose.h b/src/dev_ng/game/task/Physics/TaskNMPose.h
--- a/src/dev_ng/game/task/Physics/TaskNMPose.h
+++ b/src/dev_ng/game/task/Physics/TaskNMPose.h
@@ -7,11 +7,11 @@
 // --- Include Files ------------------------------------------------------------
 
 // Game headers
-#include "Task/System/Task.h"
-#include "Task\System\TaskTypes.h"
-#include "Task/Physics/TaskNM.h"
-#include "Task/Physics/TaskNMRelax.h"
-#include "Network/General/NetworkUtil.h"
+#include "task/System/Task.h"
+#include "task/System/TaskTypes.h"
+#include "task/Physics/TaskNM.h"
+#include "task/Physics/TaskNMRelax.h"
+#include "network/General/NetworkUtil.h"
 // ------------------------------------------------------------------------------
 
 //
