diff --git a/src/dev_ng/game/task/Physics/TaskNMSimple.cpp b/src/dev_ng/game/task/Physics/TaskNMSimple.cpp
--- a/src/dev_ng/game/task/Physics/TaskNMSimple.cpp
+++ b/src/dev_ng/game/task/Physics/TaskNMSimple.cpp
@@ -3,24 +3,24 @@
 
 //---- Include Files ---------------------------------------------------------------------------------------------------------------------------------
 // Rage headers
-#include "crskeleton\Skeleton.h"
-#include "fragment\Cache.h"
-#include "fragment\Instance.h"
-#include "fragment\Type.h"
-#include "fragment\TypeChild.h"
-#include "fragmentnm\messageparams.h"
-#include "pharticulated\articulatedbody.h"
+#include "crskeleton/skeleton.h"
+#include "fragment/Cache.h"
+#include "fragment/instance.h"
+#include "fragment/Type.h"
+#include "fragment/TypeChild.h"
+#include "fragmentnm/messageparams.h"
+#include "pharticulated/articulatedbody.h"
 
 // Framework headers
 #include "grcore/debugdraw.h"
 
 // Game headers
-#include "Event/EventDamage.h"
-#include "Peds/Ped.h"
+#include "event/EventDamage.h"
+#include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
-#include "Task/Combat/TaskWrithe.h"
-#include "Task/Physics/TaskNMSimple.h"
-#include "Task/Physics/NmDebug.h"
+#include "task/Combat/TaskWrithe.h"
+#include "task/Physics/TaskNMSimple.h"
+#include "task/Physics/NmDebug.h"
 
 //----------------------------------------------------------------------------------------------------------------------------------------------------
 
