diff --git a/src/dev_ng/game/task/Physics/TaskNM.h b/src/dev_ng/game/task/Physics/TaskNM.h
--- a/src/dev_ng/game/task/Physics/TaskNM.h
+++ b/src/dev_ng/game/task/Physics/TaskNM.h
@@ -16,13 +16,13 @@
 
 // Game headers
 #include "animation/MovePed.h"
-#include "peds/QueriableInterface.h"
-#include "Task/Physics/AnimPoseHelper.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskFSMClone.h"
-#include "Task/System/TaskTypes.h"
+#include "Peds/QueriableInterface.h"
+#include "task/Physics/AnimPoseHelper.h"
+#include "task/System/Task.h"
+#include "task/System/TaskFSMClone.h"
+#include "task/System/TaskTypes.h"
 #include "system/FileMgr.h"
-#include "Physics/gtaInst.h"
+#include "physics/gtaInst.h"
 
 // ------------------------------------------------------------------------------
 
