diff --git a/src/dev_ng/game/task/Physics/TaskNMStumble.h b/src/dev_ng/game/task/Physics/TaskNMStumble.h
--- a/src/dev_ng/game/task/Physics/TaskNMStumble.h
+++ b/src/dev_ng/game/task/Physics/TaskNMStumble.h
@@ -9,9 +9,9 @@
 // --- Include Files ------------------------------------------------------------
 
 // Game headers
-#include "Task/System/Task.h"
-#include "Task\System\TaskTypes.h"
-#include "Task/Physics/TaskNM.h"
+#include "task/System/Task.h"
+#include "task/System/TaskTypes.h"
+#include "task/Physics/TaskNM.h"
 
 #define NUM_GRAB_POINTS 4
 
