diff --git a/src/dev_ng/game/task/Physics/ThreatenedHelper.h b/src/dev_ng/game/task/Physics/ThreatenedHelper.h
--- a/src/dev_ng/game/task/Physics/ThreatenedHelper.h
+++ b/src/dev_ng/game/task/Physics/ThreatenedHelper.h
@@ -8,7 +8,7 @@
 
 // Game headers
 #include "Task/System/Task.h"
-#include "Task\System\TaskTypes.h"
+#include "Task/System/TaskTypes.h"
 #include "Task/Physics/TaskNM.h"
 
 // This class is NOT a task, it is NOT derived from CTaskNMBehaviour, it's just a helper
