diff --git a/src/dev_ng/game/task/Physics/ThreatenedHelper.h b/src/dev_ng/game/task/Physics/ThreatenedHelper.h
--- a/src/dev_ng/game/task/Physics/ThreatenedHelper.h
+++ b/src/dev_ng/game/task/Physics/ThreatenedHelper.h
@@ -7,9 +7,9 @@
 // --- Include Files ------------------------------------------------------------
 
 // Game headers
-#include "Task/System/Task.h"
-#include "Task\System\TaskTypes.h"
-#include "Task/Physics/TaskNM.h"
+#include "task/System/Task.h"
+#include "task/System/TaskTypes.h"
+#include "task/Physics/TaskNM.h"
 
 // This class is NOT a task, it is NOT derived from CTaskNMBehaviour, it's just a helper
 // that can manage the threatened behavior. It does have methods for handling messages,
