diff --git a/src/dev_ng/game/renderer/Mirrors.cpp b/src/dev_ng/game/renderer/Mirrors.cpp
--- a/src/dev_ng/game/renderer/Mirrors.cpp
+++ b/src/dev_ng/game/renderer/Mirrors.cpp
@@ -31,7 +31,7 @@
 // Framework headers
 #include "fwdebug/debugdraw.h"
 #include "fwdrawlist/drawlistmgr.h"
-#include "fwmaths/vector.h"
+#include "fwmaths/Vector.h"
 #include "fwscene/scan/Scan.h"
 #include "fwscene/scan/ScanDebug.h" // [MIRROR FLIP]
 #include "fwscene/world/InteriorSceneGraphNode.h"
@@ -53,7 +53,7 @@
 #include "renderer/Mirrors.h"
 #include "renderer/Renderer.h"
 #include "renderer/rendertargets.h"
-#include "renderer/renderthread.h"
+#include "renderer/RenderThread.h"
 #include "renderer/sprite2d.h"
 #include "renderer/Util/ShaderUtil.h"
 #include "renderer/Water.h"
