diff --git a/src/dev_ng/game/renderer/occlusion.cpp b/src/dev_ng/game/renderer/occlusion.cpp
--- a/src/dev_ng/game/renderer/occlusion.cpp
+++ b/src/dev_ng/game/renderer/occlusion.cpp
@@ -16,9 +16,9 @@
 #include "grcore/viewport_inline.h"
 #include "bank/bkmgr.h"
 #include "profile/timebars.h"
-#include "phsolver\contactmgr.h" // for physics scratchpad memory
-#include "physics\contact.h"
-#include "phcore\pool.h"
+#include "phsolver/contactmgr.h" // for physics scratchpad memory
+#include "physics/contact.h"
+#include "phcore/pool.h"
 #include "system/cache.h"
 #include "vector/colors.h"
 #include "softrasterizer/softrasterizer.h"
@@ -40,15 +40,15 @@
 // game includes
 #include "camera/CamInterface.h"
 #include "camera/viewports/ViewportManager.h"
-#include "core/game.h"
-#include "Renderer/RenderPhases/RenderPhaseCascadeShadows.h"
-#include "Renderer/RenderPhases/RenderPhaseParaboloidShadows.h"
-#include "Renderer/RenderPhases/RenderPhaseWaterReflection.h"
-#include "Renderer/Lights/lights.h"
-#include "scene/entity.h"
+#include "Core/game.h"
+#include "renderer/RenderPhases/RenderPhaseCascadeShadows.h"
+#include "renderer/RenderPhases/RenderPhaseParaboloidShadows.h"
+#include "renderer/RenderPhases/RenderPhaseWaterReflection.h"
+#include "renderer/Lights/lights.h"
+#include "scene/Entity.h"
 #include "scene/portals/FrustumDebug.h"
-#include "peds/PlayerInfo.h"
-#include "debug/vectormap.h"
+#include "Peds/PlayerInfo.h"
+#include "debug/VectorMap.h"
 #include "timecycle/TimeCycle.h"
 
 #include "fwsys/timer.h"
