diff --git a/src/dev_ng/game/renderer/River.cpp b/src/dev_ng/game/renderer/River.cpp
--- a/src/dev_ng/game/renderer/River.cpp
+++ b/src/dev_ng/game/renderer/River.cpp
@@ -1,5 +1,5 @@
 // game headers
-#include "river.h"
+#include "River.h"
 
 #include "audio/northaudioengine.h"
 #include "game/Clock.h"
@@ -18,11 +18,11 @@
 #include "scene/RegdRefTypes.h"
 #include "spatialdata/aabb.h"
 #include "timecycle/TimeCycle.h"
-#include "physics/WorldProbe/worldprobe.h"
+#include "physics/WorldProbe/WorldProbe.h"
 #include "vfx/sky/Sky.h"
 
-#include "grcore\texture_gnm.h"
-#include "grcore\wrapper_gnm.h"
+#include "grcore/texture_gnm.h"
+#include "grcore/wrapper_gnm.h"
 
 #include "system/xtl.h"
 #if __D3D
