diff --git a/src/dev_ng/game/renderer/zonecull.cpp b/src/dev_ng/game/renderer/zonecull.cpp
--- a/src/dev_ng/game/renderer/zonecull.cpp
+++ b/src/dev_ng/game/renderer/zonecull.cpp
@@ -6,28 +6,28 @@
 // CREATED : 14/1/01
 //
 /////////////////////////////////////////////////////////////////////////////////
-#include "renderer\zonecull.h"
+#include "renderer/zonecull.h"
 
-#include "system\new.h"
+#include "system/new.h"
 
 // Rage headers
 #include "bank/bkmgr.h"
 
 // Framework headers
 #include "grcore/debugdraw.h"
-#include "fwmaths\random.h"
+#include "fwmaths/random.h"
 
 // Game headers
 #include "camera/CamInterface.h"
 #include "camera/viewports/ViewportManager.h"
-#include "scene\FileLoader.h"
-#include "system\filemgr.h"
-#include "modelinfo\ModelInfo.h"
+#include "scene/FileLoader.h"
+#include "system/filemgr.h"
+#include "modelinfo/ModelInfo.h"
 #include "peds/Ped.h"
-#include "renderer\Renderer.h"
-#include "vehicles\Train.h"
-#include "game\Zones.h"
-#include "scene\world\gameWorld.h"
+#include "renderer/Renderer.h"
+#include "vehicles/Train.h"
+#include "game/Zones.h"
+#include "scene/world/gameWorld.h"
 
 
 s32		CCullZones::NumAttributeZones;
