diff --git a/src/dev_ng/game/vfx/sky/Sky.cpp b/src/dev_ng/game/vfx/sky/Sky.cpp
--- a/src/dev_ng/game/vfx/sky/Sky.cpp
+++ b/src/dev_ng/game/vfx/sky/Sky.cpp
@@ -7,7 +7,7 @@
 #include "rmcore/drawable.h"
 
 // INCLUDES
-#include "Vfx/Sky/Sky.h"
+#include "vfx/sky/Sky.h"
 
 // rage
 #include "grmodel/modelfactory.h"
@@ -35,11 +35,11 @@
 // game
 #include "camera/CamInterface.h"
 #include "camera/base/BaseCamera.h"
-#include "Core/Game.h"
+#include "Core/game.h"
 #include "debug/Rendering/DebugDeferred.h"
 #include "renderer/GtaDrawable.h"
-#include "renderer/renderthread.h"
-#include "renderer/lights/lights.h"
+#include "renderer/RenderThread.h"
+#include "renderer/Lights/lights.h"
 #include "renderer/RenderSettings.h"
 #include "renderer/RenderPhases/RenderPhaseReflection.h"
 #include "renderer/RenderPhases/RenderPhaseDebugOverlay.h"
@@ -50,9 +50,9 @@
 #include "streaming/streaming.h"
 #include "timecycle/TimeCycle.h"
 #include "timecycle/TimeCycleConfig.h"
-#include "vfx/vfxHelper.h"
+#include "vfx/VfxHelper.h"
 #include "vfx/vfx_channel.h"
-#include "Vfx/Misc/Coronas.h"
+#include "vfx/misc/Coronas.h"
 #include "vfx/sky/SkySettings.h"
 #include "vfx/VfxSettings.h"
 
