diff --git a/src/dev_ng/rage/framework/src/video/mediaencoder_buffers.cpp b/src/dev_ng/rage/framework/src/video/mediaencoder_buffers.cpp
--- a/src/dev_ng/rage/framework/src/video/mediaencoder_buffers.cpp
+++ b/src/dev_ng/rage/framework/src/video/mediaencoder_buffers.cpp
@@ -17,7 +17,7 @@
 
 #if (USES_MEDIA_ENCODER) && !__RESOURCECOMPILER && !__RGSC_DLL
 
-#if (RSG_PC || RSG_DURANGO)
+#if (RSG_PC || RSG_DURANGO) && !RSG_LINUX
 #pragma warning(push)
 #pragma warning(disable: 4668)
 #include <WinSock2.h>
