diff --git a/src/dev_ng/rage/framework/src/video/mediadecoder_win.cpp b/src/dev_ng/rage/framework/src/video/mediadecoder_win.cpp
--- a/src/dev_ng/rage/framework/src/video/mediadecoder_win.cpp
+++ b/src/dev_ng/rage/framework/src/video/mediadecoder_win.cpp
@@ -12,7 +12,12 @@
 
 #if defined(VIDEO_PLAYBACK_ENABLED) && VIDEO_PLAYBACK_ENABLED
 
-#if ( RSG_DURANGO || RSG_PC ) && !__RESOURCECOMPILER && !__RGSC_DLL
+#if RSG_LINUX
+namespace rage
+{
+	MediaDecoder::MediaDecoder() {}
+} // namespace rage
+#elif ( RSG_DURANGO || RSG_PC ) && !__RESOURCECOMPILER && !__RGSC_DLL
 
 // rage
 #include "atl/array.h"
