diff --git a/src/dev_ng/rage/framework/src/video/media_common.cpp b/src/dev_ng/rage/framework/src/video/media_common.cpp
--- a/src/dev_ng/rage/framework/src/video/media_common.cpp
+++ b/src/dev_ng/rage/framework/src/video/media_common.cpp
@@ -9,6 +9,7 @@
 //
 /////////////////////////////////////////////////////////////////////////////////
 #include "media_common.h"
+#include <climits>
 
 #ifndef __RGSC_DLL
 #define __RGSC_DLL 0
