diff --git a/src/dev_ng/game/pathserver/PathServer_Hierarchical.cpp b/src/dev_ng/game/pathserver/PathServer_Hierarchical.cpp
--- a/src/dev_ng/game/pathserver/PathServer_Hierarchical.cpp
+++ b/src/dev_ng/game/pathserver/PathServer_Hierarchical.cpp
@@ -1,19 +1,19 @@
 
 // Rage headers
-#include "file\device.h"
-#include "file\stream.h"
-#include "data\struct.h"
-#include "file\asset.h"
-#include "paging\rscbuilder.h"
+#include "file/device.h"
+#include "file/stream.h"
+#include "data/struct.h"
+#include "file/asset.h"
+#include "paging/rscbuilder.h"
 #include "vector/geometry.h"
 
 // Framework headers
 #include "ai/navmesh/priqueue.h"
-#include "fwmaths\random.h"
-#include "fwmaths\vector.h"
+#include "fwmaths/random.h"
+#include "fwmaths/vector.h"
 
 // Game headers
-#include "PathServer\PathServer.h"
+#include "PathServer/PathServer.h"
 
 #ifdef GTA_ENGINE
 NAVMESH_OPTIMISATIONS()
