diff --git a/src/dev_ng/game/pathserver/PathServerThread.cpp b/src/dev_ng/game/pathserver/PathServerThread.cpp
--- a/src/dev_ng/game/pathserver/PathServerThread.cpp
+++ b/src/dev_ng/game/pathserver/PathServerThread.cpp
@@ -4,8 +4,8 @@
 #include "profile/telemetry.h"
 #include "profile/timebars.h"
 #include "profile/cputrace.h"
-#include "System/ipc.h"
-#include "System/Xtl.h"
+#include "system/ipc.h"
+#include "system/Xtl.h"
 #include "Vector/Colors.h"
 #include "Vector/Geometry.h"
 #include "system/memory.h"
@@ -14,8 +14,8 @@
 #include "ai/navmesh/priqueue.h"
 #include "fwmaths/Vector.h"
 
-#include "PathServer/PathServer.h"
-#include "PathServer/PathServerThread.h"
+#include "pathserver/PathServer.h"
+#include "pathserver/PathServerThread.h"
 
 #include "ai/navmesh/pathserverthread.h"
 
@@ -25,12 +25,12 @@
 #include "system/threadpriorities.h"
 #include "task/Movement/TaskNavMesh.h"	// Used for tracking cases where an AI task has failed to retrieve a requested path
 #include "task/Movement/TaskMoveWander.h"
-#include "peds/pedpopulation.h"
+#include "Peds/pedpopulation.h"
 #include "system/InfoState.h"
 #endif
 
 #include "security/ragesecgameinterface.h"
-#include "Network/Sessions/NetworkSession.h"
+#include "network/Sessions/NetworkSession.h"
 
 #if __XENON && !__FINAL
 #define __TRACE_PATH_REQUESTS_ON_XENON		1
