diff --git a/src/dev_ng/game/pathserver/PathServer_Cover.cpp b/src/dev_ng/game/pathserver/PathServer_Cover.cpp
--- a/src/dev_ng/game/pathserver/PathServer_Cover.cpp
+++ b/src/dev_ng/game/pathserver/PathServer_Cover.cpp
@@ -1,7 +1,7 @@
-#include "PathServer\PathServer.h"
+#include "PathServer/PathServer.h"
 
 #ifdef GTA_ENGINE
-#include "task\Combat\Cover\Cover.h"
+#include "task/Combat/Cover/Cover.h"
 #include "scene/world/GameWorld.h"
 #include "fwsys/timer.h"
 
