diff --git a/src/dev_ng/game/shaders/CustomShaderEffectCable.cpp b/src/dev_ng/game/shaders/CustomShaderEffectCable.cpp
--- a/src/dev_ng/game/shaders/CustomShaderEffectCable.cpp
+++ b/src/dev_ng/game/shaders/CustomShaderEffectCable.cpp
@@ -22,10 +22,10 @@
 // Game headers
 #include "camera/CamInterface.h"
 #include "camera/viewports/ViewportManager.h"
-#include "debug/debug.h"
+#include "debug/Debug.h"
 #include "Objects/DummyObject.h"
 #include "renderer/PostProcessFX.h"
-#include "renderer/renderphases/RenderPhaseEntitySelect.h"
+#include "renderer/RenderPhases/RenderPhaseEntitySelect.h"
 #include "scene/AnimatedBuilding.h"
 #include "scene/Building.h"
 #include "scene/Entity.h"
