diff --git a/src/dev_ng/game/shaders/CustomShaderEffectBase.cpp b/src/dev_ng/game/shaders/CustomShaderEffectBase.cpp
--- a/src/dev_ng/game/shaders/CustomShaderEffectBase.cpp
+++ b/src/dev_ng/game/shaders/CustomShaderEffectBase.cpp
@@ -13,7 +13,7 @@
 #include "rmcore/drawable.h"
 
 // Game headers:
-#include "debug/debug.h"
+#include "debug/Debug.h"
 #include "CustomShaderEffectBase.h"
 #include "CustomShaderEffectPed.h"
 #include "CustomShaderEffectVehicle.h"
@@ -26,9 +26,9 @@
 #include "CustomShaderEffectMirror.h"
 #include "CustomShaderEffectWeapon.h"
 #include "CustomShaderEffectInterior.h"
-#include "modelinfo/vehicleModelInfo.h"
+#include "modelinfo/VehicleModelInfo.h"
 #include "physics/gtaArchetype.h"
-#include "renderer/DrawLists/drawlist.h"
+#include "renderer/DrawLists/drawList.h"
 #include "renderer/Entities/InstancedEntityRenderer.h"
 
 #if !__FINAL
