diff --git a/src/dev_ng/game/shaders/CustomShaderEffectMirror.cpp b/src/dev_ng/game/shaders/CustomShaderEffectMirror.cpp
--- a/src/dev_ng/game/shaders/CustomShaderEffectMirror.cpp
+++ b/src/dev_ng/game/shaders/CustomShaderEffectMirror.cpp
@@ -4,8 +4,8 @@
 
 // Rage headers
 #include "bank/bkmgr.h"
-#include "grmodel/ShaderFx.h"
-#include "grmodel/Geometry.h"
+#include "grmodel/shaderfx.h"
+#include "grmodel/geometry.h"
 #include "grcore/image.h"
 #include "grcore/texture.h"
 #include "grcore/texturereference.h"
@@ -18,7 +18,7 @@
 #include "fwutil/xmacro.h"
 
 // Game headers
-#include "debug/debug.h"
+#include "debug/Debug.h"
 #include "renderer/Mirrors.h"
 #include "renderer/PostProcessFX.h"
 #include "renderer/RenderPhases/RenderPhaseReflection.h"
