diff --git a/src/dev_ng/game/shaders/CustomShaderEffectAnimUV.cpp b/src/dev_ng/game/shaders/CustomShaderEffectAnimUV.cpp
--- a/src/dev_ng/game/shaders/CustomShaderEffectAnimUV.cpp
+++ b/src/dev_ng/game/shaders/CustomShaderEffectAnimUV.cpp
@@ -5,7 +5,7 @@
 //	19/11/2007	-	Andrzej:	- Update(): anim time synchronization between entities and their lods added;
 //
 // The gta5 'testbed' level has an example of an AnimUV.
-#include "shaders\CustomShaderEffectAnimUV.h"
+#include "shaders/CustomShaderEffectAnimUV.h"
 
 //Rage includes
 #include "cranimation/frame.h"
@@ -21,7 +21,7 @@
 #include "scene/Entity.h"
 #include "shaders/ShaderLib.h"
 #include "streaming/defragmentation.h"
-#include "Control/Replay/Replay.h"
+#include "control/replay/replay.h"
 
 ANIM_OPTIMISATIONS()
 
