diff --git a/src/dev_ng/game/game/Dispatch/RoadBlock.cpp b/src/dev_ng/game/game/Dispatch/RoadBlock.cpp
--- a/src/dev_ng/game/game/Dispatch/RoadBlock.cpp
+++ b/src/dev_ng/game/game/Dispatch/RoadBlock.cpp
@@ -2,7 +2,7 @@
 // Started	:	1/20/2012
 
 // File header
-#include "Game/Dispatch/RoadBlock.h"
+#include "game/Dispatch/RoadBlock.h"
 
 // Rage headers
 #include "fwmaths/random.h"
@@ -19,9 +19,9 @@
 #include "Peds/PedFactory.h"
 #include "Peds/PedIntelligence.h"
 #include "scene/world/GameWorld.h"
-#include "Task/Animation/TaskAnims.h"
+#include "task/Animation/TaskAnims.h"
 #include "vehicleAi/pathfind.h"
-#include "vehicles/Automobile.h"
+#include "Vehicles/Automobile.h"
 #include "Vehicles/vehiclepopulation.h"
 
 AI_OPTIMISATIONS()
