diff --git a/src/dev_ng/game/physics/WorldProbe/WorldProbe.cpp b/src/dev_ng/game/physics/WorldProbe/WorldProbe.cpp
--- a/src/dev_ng/game/physics/WorldProbe/WorldProbe.cpp
+++ b/src/dev_ng/game/physics/WorldProbe/WorldProbe.cpp
@@ -14,27 +14,27 @@
 #include "system/task.h"
 #include "spatialdata/sphere.h"
 #include "fwsys/gameskeleton.h"
-#include "entity\transform.h"
+#include "entity/transform.h"
 
 // Framework headers
 
 // Game headers
-#include "debug/debugscene.h"
-#include "peds/ped.h"
+#include "debug/DebugScene.h"
+#include "Peds/ped.h"
 #include "modelinfo/PedModelInfo.h"
 #include "physics/gtaArchetype.h"
 #include "physics/gtaInst.h"
 #include "physics/gtaMaterialManager.h"
 #include "physics/iterator.h"
 #include "physics/physics.h"
-#include "physics/WorldProbe/worldprobe.h"
+#include "physics/WorldProbe/WorldProbe.h"
 #include "physics/levelnew.h"
-#include "scene/entity.h"
-#include "scene/physical.h"
-#include "vehicles/Automobile.h"
-#include "vehicles/Boat.h"
-#include "vehicles/vehicle.h"
-#include "vehicles/wheel.h"
+#include "scene/Entity.h"
+#include "scene/Physical.h"
+#include "Vehicles/Automobile.h"
+#include "Vehicles/Boat.h"
+#include "Vehicles/vehicle.h"
+#include "Vehicles/wheel.h"
 #include "scene/world/GameWorld.h"
 
 PHYSICS_OPTIMISATIONS()
@@ -56,7 +56,7 @@ PF_TIMER(Unspecified_Probes, Probes);
 
 
 #if RECORD_SHAPE_TEST_TASK_MANAGER_INTERACTION
-	#include "atl\array.h"
+	#include "atl/array.h"
 	#define MAX_SHAPE_TEST_TASK_HANDLES_RECORDED (50) //Only really needs to be MAX_ASYNCHRONOUS_PROCESSING_SHAPE_TESTS, but this will count leakage
 	atFixedArray<sysTaskHandle, MAX_SHAPE_TEST_TASK_HANDLES_RECORDED> g_DebugRecordTaskManagerInteractionShapeTestHandles;
 	u32 g_DebugRecordTaskManagerInteractionShapeTestHandlesHighWaterMark = 0u;
