diff --git a/src/dev_ng/game/frontend/Scaleform/ScaleFormMgr.cpp b/src/dev_ng/game/frontend/Scaleform/ScaleFormMgr.cpp
--- a/src/dev_ng/game/frontend/Scaleform/ScaleFormMgr.cpp
+++ b/src/dev_ng/game/frontend/Scaleform/ScaleFormMgr.cpp
@@ -9,12 +9,12 @@
 
 // Rage headers
 #include "fwscene/stores/txdstore.h"
-#include "fwsys/FileExts.h"
+#include "fwsys/fileExts.h"
 #include "fwsys/timer.h"
 #include "grcore/allocscope.h"
 #include "grcore/effect.h"
 #include "scaleform/renderer.h"
-#include "Scaleform/scaleform.h"
+#include "scaleform/scaleform.h"
 #include "scr_scaleform/scaleform.h"
 #include "script/thread.h"
 #include "streaming/streamingengine.h"
@@ -34,20 +34,20 @@
 // Game headers
 #include "audio/frontendaudioentity.h"
 #include "camera/CamInterface.h"
-#include "Core/Game.h"
-#include "Debug/TextureViewer/TextureViewer.h"
-#include "Frontend/MovieCLipText.h"
-#include "Frontend/Scaleform/ScaleFormMgr.h"
-#include "Frontend/Scaleform/ScaleFormComplexObjectMgr.h"
-#include "Frontend/Scaleform/ScaleFormStore.h"
+#include "Core/game.h"
+#include "debug/TextureViewer/TextureViewer.h"
+#include "frontend/MovieClipText.h"
+#include "frontend/Scaleform/ScaleFormMgr.h"
+#include "frontend/Scaleform/ScaleFormComplexObjectMgr.h"
+#include "frontend/Scaleform/ScaleFormStore.h"
 #if RSG_PC
-#include "Frontend/MultiplayerChat.h"
+#include "frontend/MultiplayerChat.h"
 #endif // RSG_PC
 #include "game/Clock.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "renderer/rendertargets.h"
 #include "scene/world/GameWorld.h"
-#include "Script/Script_Hud.h"
+#include "script/script_hud.h"
 #include "streaming/streaming.h"
 #include "system/controlMgr.h"
 #include "system/param.h"
