diff --git a/src/dev_ng/game/vfx/systems/VfxGadget.cpp b/src/dev_ng/game/vfx/systems/VfxGadget.cpp
--- a/src/dev_ng/game/vfx/systems/VfxGadget.cpp
+++ b/src/dev_ng/game/vfx/systems/VfxGadget.cpp
@@ -18,16 +18,16 @@
 
 // framework
 #include "vfx/channel.h"
-#include "vfx/ptfx/ptfxAttach.h"
+#include "vfx/ptfx/ptfxattach.h"
 #include "vfx/vfxwidget.h"
 
 // game
-#include "Core/Game.h"
-#include "Peds/Ped.h"
-#include "Vfx/Particles/PtFxManager.h"
-#include "Vfx/Systems/VfxTrail.h"
-#include "Vfx/Systems/VfxPed.h"
-#include "Vfx/VfxHelper.h"
+#include "Core/game.h"
+#include "Peds/ped.h"
+#include "vfx/particles/PtFxManager.h"
+#include "vfx/systems/VfxTrail.h"
+#include "vfx/systems/VfxPed.h"
+#include "vfx/VfxHelper.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
