diff --git a/src/dev_ng/game/vfx/systems/VfxProjectile.h b/src/dev_ng/game/vfx/systems/VfxProjectile.h
--- a/src/dev_ng/game/vfx/systems/VfxProjectile.h
+++ b/src/dev_ng/game/vfx/systems/VfxProjectile.h
@@ -19,7 +19,7 @@
 // rage
 
 // game
-#include "physics/WorldProbe/worldprobe.h"
+#include "physics/WorldProbe/WorldProbe.h"
 
 ///////////////////////////////////////////////////////////////////////////////
 //  FORWARD DECLARATIONS
