diff --git a/src/dev_ng/game/vfx/systems/VfxWeather.cpp b/src/dev_ng/game/vfx/systems/VfxWeather.cpp
--- a/src/dev_ng/game/vfx/systems/VfxWeather.cpp
+++ b/src/dev_ng/game/vfx/systems/VfxWeather.cpp
@@ -25,25 +25,25 @@
 #include "vfx/vfxwidget.h"
 
 // game
-#include "Camera/CamInterface.h"
-#include "Camera/Gameplay/Aim/FirstPersonShooterCamera.h"
-#include "Camera/Gameplay/GameplayDirector.h"
+#include "camera/CamInterface.h"
+#include "camera/Gameplay/Aim/FirstPersonShooterCamera.h"
+#include "camera/Gameplay/GameplayDirector.h"
 #include "Core/Game.h"
 #include "Cutscene/CutSceneManagerNew.h"
-#include "Game/Weather.h"
-#include "Game/Wind.h"
-#include "Peds/Ped.h"
+#include "game/Weather.h"
+#include "game/wind.h"
+#include "Peds/ped.h"
 #include "Peds/PopCycle.h"
-#include "physics/WorldProbe/worldprobe.h"
-#include "Scene/Portals/Portal.h"
-#include "Scene/World/GameWorld.h"
-#include "TimeCycle/TimeCycle.h"
-#include "Vehicles/Vehicle.h"
-#include "Vfx/Metadata/VfxRegionInfo.h"
-#include "Vfx/Particles/PtFxManager.h"
-#include "Vfx/Systems/VfxLens.h"
-#include "Vfx/VfxHelper.h"
-#include "Vfx/VfxSettings.h"
+#include "physics/WorldProbe/WorldProbe.h"
+#include "scene/portals/Portal.h"
+#include "scene/world/GameWorld.h"
+#include "timecycle/TimeCycle.h"
+#include "Vehicles/vehicle.h"
+#include "vfx/metadata/VfxRegionInfo.h"
+#include "vfx/Particles/PtFxManager.h"
+#include "vfx/systems/VfxLens.h"
+#include "vfx/VfxHelper.h"
+#include "vfx/VfxSettings.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
