diff --git a/src/dev_ng/game/task/Motion/Locomotion/TaskFishLocomotion.h b/src/dev_ng/game/task/Motion/Locomotion/TaskFishLocomotion.h
--- a/src/dev_ng/game/task/Motion/Locomotion/TaskFishLocomotion.h
+++ b/src/dev_ng/game/task/Motion/Locomotion/TaskFishLocomotion.h
@@ -1,7 +1,7 @@
 #ifndef TASK_FISH_LOCOMOTION_H
 #define TASK_FISH_LOCOMOTION_H
 
-#include "Task/Motion/TaskMotionBase.h"
+#include "task/Motion/TaskMotionBase.h"
 
 //*********************************************************************
 //	CTaskFishLocomotion
