diff --git a/src/dev_ng/game/task/Motion/Locomotion/TaskMotionJetpack.h b/src/dev_ng/game/task/Motion/Locomotion/TaskMotionJetpack.h
--- a/src/dev_ng/game/task/Motion/Locomotion/TaskMotionJetpack.h
+++ b/src/dev_ng/game/task/Motion/Locomotion/TaskMotionJetpack.h
@@ -8,9 +8,9 @@
 #define TASK_MOTION_JETPACK_H
 
 #include "Peds/ped.h"
-#include "Task/Motion/TaskMotionBase.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/Movement/JetpackObject.h"
+#include "task/Motion/TaskMotionBase.h"
+#include "task/System/TaskHelpers.h"
+#include "task/Movement/JetpackObject.h"
 
 #if ENABLE_JETPACK
 
