diff --git a/src/dev_ng/rage/framework/src/fwscene/stores/expressionsdictionarystore.h b/src/dev_ng/rage/framework/src/fwscene/stores/expressionsdictionarystore.h
--- a/src/dev_ng/rage/framework/src/fwscene/stores/expressionsdictionarystore.h
+++ b/src/dev_ng/rage/framework/src/fwscene/stores/expressionsdictionarystore.h
@@ -4,8 +4,8 @@
 
 // --- Include Files ------------------------------------------------------------
 
-#include "fwtl\assetstore.h"
-#include "crextra\expressions.h"
+#include "fwtl/assetstore.h"
+#include "crextra/expressions.h"
 
 
 namespace rage {
