diff --git a/src/dev_ng/game/renderer/Shadows/ParaboloidShadows.cpp b/src/dev_ng/game/renderer/Shadows/ParaboloidShadows.cpp
--- a/src/dev_ng/game/renderer/Shadows/ParaboloidShadows.cpp
+++ b/src/dev_ng/game/renderer/Shadows/ParaboloidShadows.cpp
@@ -4,7 +4,7 @@
 // ======================================
 
 #include "fwutil/xmacro.h"
-#include "fwtl/pool.h"
+#include "fwtl/Pool.h"
 #include "grcore/allocscope.h"
 #include "grcore/debugdraw.h"
 #include "system/typeinfo.h"
@@ -17,20 +17,20 @@
 #include "renderer/Lights/LightEntity.h"
 #include "renderer/Lights/LightCulling.h"
 #include "renderer/Util/Util.h" // SAFE_DELETE, SAFE_RELEASE
-#include "renderer/spupm/spupmmgr.h"
-#include "renderer/shadows/shadowsprivate.h"
-#include "renderer/shadows/paraboloidshadows.h"
+#include "renderer/SpuPM/spupmmgr.h"
+#include "renderer/Shadows/shadowsprivate.h"
+#include "renderer/Shadows/ParaboloidShadows.h"
 #include "scene/portals/Portal.h"
 #include "scene/portals/PortalTracker.h"
 #include "system/SettingsManager.h"
-#include "Renderer/Deferred/DeferredLighting.h"
-#include "Renderer/RenderPhases/RenderPhaseCascadeShadows.h"
+#include "renderer/Deferred/DeferredLighting.h"
+#include "renderer/RenderPhases/RenderPhaseCascadeShadows.h"
 
 #if __BANK
 #include "debug/Rendering/DebugLighting.h"
 #include "debug/Rendering/DebugLights.h"
 #include "debug/BudgetDisplay.h"
-#include "Cutscene/CutSceneDebugManager.h"
+#include "cutscene/CutSceneDebugManager.h"
 #endif
 
 #if RSG_ORBIS
