diff --git a/src/dev_ng/game/renderer/Shadows/ShadowsPrivate.h b/src/dev_ng/game/renderer/Shadows/ShadowsPrivate.h
--- a/src/dev_ng/game/renderer/Shadows/ShadowsPrivate.h
+++ b/src/dev_ng/game/renderer/Shadows/ShadowsPrivate.h
@@ -40,15 +40,15 @@
 
 
 // Game headers
-#include "debug/debugglobals.h"
+#include "debug/Debugglobals.h"
 #include "camera/viewports/viewportmanager.h"
-#include "shaders/shaderlib.h"
-#include "renderer/lights/lights.h"
+#include "shaders/ShaderLib.h"
+#include "renderer/Lights/lights.h"
 #include "renderer/rendertargets.h"
 #include "renderer/sprite2d.h"
 #include "renderer/deferred/deferredlighting.h"
-#include "renderer/drawlists/drawlistmgr.h"
-#include "renderer/shadows/shadows.h"
+#include "renderer/DrawLists/drawListMgr.h"
+#include "renderer/Shadows/Shadows.h"
 
 // ================================================================================================
 
