diff --git a/src/dev_ng/rage/base/src/physics/colliderdispatch.h b/src/dev_ng/rage/base/src/physics/colliderdispatch.h
--- a/src/dev_ng/rage/base/src/physics/colliderdispatch.h
+++ b/src/dev_ng/rage/base/src/physics/colliderdispatch.h
@@ -7,8 +7,8 @@
 #ifndef PHYSICS_COLLIDERDISPATCH_H 
 #define PHYSICS_COLLIDERDISPATCH_H 
 
-#include "physics\collider.h"
-#include "pharticulated\articulatedcollider.h"
+#include "physics/collider.h"
+#include "pharticulated/articulatedcollider.h"
 
 namespace rage {
 
