diff --git a/src/dev_ng/game/debug/Rendering/DebugLights.h b/src/dev_ng/game/debug/Rendering/DebugLights.h
--- a/src/dev_ng/game/debug/Rendering/DebugLights.h
+++ b/src/dev_ng/game/debug/Rendering/DebugLights.h
@@ -13,8 +13,8 @@
 // game 
 #include "atl/map.h"
 #include "scene/2dEffect.h"
-#include "renderer/Lights/Lights.h"
-#include "Debug/Rendering/DebugRendering.h"
+#include "renderer/Lights/lights.h"
+#include "debug/Rendering/DebugRendering.h"
 #include "scene/RegdRefTypes.h"
 
 // =============================================================================================== //
