diff --git a/src/dev_ng/game/debug/Rendering/DebugRendering.cpp b/src/dev_ng/game/debug/Rendering/DebugRendering.cpp
--- a/src/dev_ng/game/debug/Rendering/DebugRendering.cpp
+++ b/src/dev_ng/game/debug/Rendering/DebugRendering.cpp
@@ -18,12 +18,12 @@
 #include "fwrenderer/RenderPhaseBase.h"
 
 // game
-#include "Debug/Rendering/DebugRendering.h"
-#include "Debug/Editing/LightEditing.h"
+#include "debug/Rendering/DebugRendering.h"
+#include "debug/Editing/LightEditing.h"
 #include "camera/CamInterface.h"
 #include "camera/debug/DebugDirector.h"
 #include "camera/helpers/Frame.h"
-#include "camera/viewports/viewportmanager.h"
+#include "camera/viewports/ViewportManager.h"
 #include "renderer/Deferred/DeferredLighting.h"
 #include "renderer/Deferred/DeferredConfig.h"
 #include "renderer/Deferred/GBuffer.h"
@@ -32,17 +32,17 @@
 #include "renderer/SpuPM/SpuPmMgr.h"
 #include "renderer/Util/ShaderUtil.h"
 #include "optimisations.h"
-#include "renderer/lights/lights.h"
-#include "Renderer/Lights/LightEntity.h"
+#include "renderer/Lights/lights.h"
+#include "renderer/Lights/LightEntity.h"
 #include "scene/FileLoader.h"
 #include "shaders/ShaderEdit.h"
-#include "peds/Ped.h"
+#include "Peds/ped.h"
 #include "debug/DebugScene.h"
 #include "scene/world/GameWorld.h"
-#include "Debug/Rendering/DebugLights.h"
-#include "Debug/Rendering/DebugView.h"
-#include "Debug/Rendering/DebugDeferred.h"
-#include "Debug/Rendering/DebugLighting.h"
+#include "debug/Rendering/DebugLights.h"
+#include "debug/Rendering/DebugView.h"
+#include "debug/Rendering/DebugDeferred.h"
+#include "debug/Rendering/DebugLighting.h"
 #include "renderer/Util/RenderDocManager.h"
 
 RENDER_OPTIMISATIONS()
