diff --git a/src/dev_ng/game/network/Shop/NetworkShopping.cpp b/src/dev_ng/game/network/Shop/NetworkShopping.cpp
--- a/src/dev_ng/game/network/Shop/NetworkShopping.cpp
+++ b/src/dev_ng/game/network/Shop/NetworkShopping.cpp
@@ -1828,7 +1828,7 @@ void CNetworkShoppingMgr::Init(sysMemAllocator* allocator)
 
 	sysMemAllocator::SetCurrent(*previousallocator);
 
-#if	__DEV
+#if	!__PACKEDDEBUG
 	PARSER.LoadObject("common:/data/debug/networkshop_debug", "meta", *this);
 #else
 	PARSER.LoadObject("common:/data/networkshop", "meta", *this);
