diff --git a/src/dev_ng/game/network/Shop/VehicleCatalog.cpp b/src/dev_ng/game/network/Shop/VehicleCatalog.cpp
--- a/src/dev_ng/game/network/Shop/VehicleCatalog.cpp
+++ b/src/dev_ng/game/network/Shop/VehicleCatalog.cpp
@@ -6,7 +6,7 @@
 // Stores the contents of the transaction server catalog for vehicle mods
 //
 
-#include "Network/Shop/VehicleCatalog.h"
+#include "network/Shop/VehicleCatalog.h"
 #include "network/NetworkTypes.h"
 #include "Stats/StatsTypes.h"
 #include "Stats/StatsInterface.h"
