diff --git a/src/dev_ng/game/network/Live/store/marketplace_psn.cpp b/src/dev_ng/game/network/Live/store/marketplace_psn.cpp
--- a/src/dev_ng/game/network/Live/store/marketplace_psn.cpp
+++ b/src/dev_ng/game/network/Live/store/marketplace_psn.cpp
@@ -22,7 +22,7 @@
 #include "diag/output.h"
 #include "system/memory.h"
 #include "string/unicode.h"
-#include "System/Param.h"
+#include "system/Param.h"
 #include "rline/rldiag.h"
 #include "rline/rl.h"
 
