diff --git a/src/dev_ng/game/physics/Deformable.cpp b/src/dev_ng/game/physics/Deformable.cpp
--- a/src/dev_ng/game/physics/Deformable.cpp
+++ b/src/dev_ng/game/physics/Deformable.cpp
@@ -8,7 +8,7 @@
 #include "Objects/object.h"
 #include "physics/physics_channel.h"
 #include "scene/Entity.h"
-#include "scene/datafilemgr.h"
+#include "scene/DataFileMgr.h"
 
 // Parser headers:
 #include "physics/Deformable_parser.h"
