diff --git a/src/dev_ng/game/physics/gtaMaterialManager.cpp b/src/dev_ng/game/physics/gtaMaterialManager.cpp
--- a/src/dev_ng/game/physics/gtaMaterialManager.cpp
+++ b/src/dev_ng/game/physics/gtaMaterialManager.cpp
@@ -12,11 +12,11 @@
 //  INCLUDES
 ///////////////////////////////////////////////////////////////////////////////
 
-#include "GtaMaterialManager.h"
+#include "gtaMaterialManager.h"
 #include "gtaMaterialManager_parser.h"
 
 // game
-#include "Debug/Debug.h"
+#include "debug/Debug.h"
 
 #include "parser/manager.h"
 
