diff --git a/src/dev_ng/rage/base/src/net/nethardware.cpp b/src/dev_ng/rage/base/src/net/nethardware.cpp
--- a/src/dev_ng/rage/base/src/net/nethardware.cpp
+++ b/src/dev_ng/rage/base/src/net/nethardware.cpp
@@ -306,7 +306,7 @@ netHardware::Update()
 			isOnline = g_rlNp.IsOnline(i);
 #elif __RGSC_DLL
 		isOnline = netHardware::IsLinkConnected();
-#elif RSG_LINUX
+#elif RSG_LINUX && 0 // missing rline for Linux source code
 		isOnline = g_rlGgp.IsOnline();
 #elif RL_SC_PLATFORMS
         for(int i = 0; i < RL_MAX_LOCAL_GAMERS && !isOnline; ++i)
