diff --git a/src/dev_ng/game/task/Motion/TaskMotionBase.cpp b/src/dev_ng/game/task/Motion/TaskMotionBase.cpp
--- a/src/dev_ng/game/task/Motion/TaskMotionBase.cpp
+++ b/src/dev_ng/game/task/Motion/TaskMotionBase.cpp
@@ -7,7 +7,7 @@
 #include "math/angmath.h"
 #include "physics/debugEvents.h"
 #include "system/stack.h"
-#include "vector\matrix34.h"
+#include "vector/matrix34.h"
 
 // framework includes
 #include "fwanimation/animdirector.h"
@@ -17,28 +17,28 @@
 // project includes
 #include "animation/debug/AnimDebug.h"
 #include "modelinfo/PedModelInfo.h"
-#include "ModelInfo/VehicleModelInfo.h"
-#include "Network/Objects/Entities/NetObjPlayer.h"
+#include "modelinfo/VehicleModelInfo.h"
+#include "network/Objects/Entities/NetObjPlayer.h"
 #include "Objects/Door.h"
-#include "Peds\Ped.h"
-#include "Peds\PedIntelligence.h"
+#include "Peds/ped.h"
+#include "Peds/PedIntelligence.h"
 #include "Peds/PedIKSettings.h"
-#include "Physics\Physics.h"
-#include "scene/entity.h"
+#include "physics/physics.h"
+#include "scene/Entity.h"
 #include "scene/Physical.h"
 #include "scene/world/GameWorld.h"
-#include "task/motion/Locomotion/TaskMotionPed.h"
+#include "task/Motion/Locomotion/TaskMotionPed.h"
 #include "task/Motion/Locomotion/TaskMotionAiming.h"
-#include "Task/System/MotionTaskData.h"
-#include "Task/Weapons/TaskProjectile.h"
-#include "vehicles/vehicle.h"
-#include "Weapons/Info/WeaponAnimationsManager.h"
+#include "task/System/MotionTaskData.h"
+#include "task/Weapons/TaskProjectile.h"
+#include "Vehicles/vehicle.h"
+#include "weapons/info/WeaponAnimationsManager.h"
 
 #include "camera/CamInterface.h"
 #include "camera/gameplay/aim/FirstPersonShooterCamera.h"
 #include "camera/gameplay/GameplayDirector.h"
 
-#include "Task/Movement/Jumping/TaskJump.h"
+#include "task/Movement/Jumping/TaskJump.h"
 
 AI_OPTIMISATIONS()
 AI_MOTION_OPTIMISATIONS()
