diff --git a/src/dev_ng/game/ai/debug/types/ScriptHistoryDebugInfo.h b/src/dev_ng/game/ai/debug/types/ScriptHistoryDebugInfo.h
--- a/src/dev_ng/game/ai/debug/types/ScriptHistoryDebugInfo.h
+++ b/src/dev_ng/game/ai/debug/types/ScriptHistoryDebugInfo.h
@@ -1,11 +1,11 @@
 #pragma once
 
 // game headers
-#include "ai\debug\system\AIDebugInfo.h"
+#include "ai/debug/system/AIDebugInfo.h"
 
 #if AI_DEBUG_OUTPUT_ENABLED
 
-#include "scene\RegdRefTypes.h"
+#include "scene/RegdRefTypes.h"
 
 class CScriptHistoryDebugInfo : public CAIDebugInfo
 {
