diff --git a/src/dev_ng/game/ai/debug/types/VehicleTaskDebugInfo.h b/src/dev_ng/game/ai/debug/types/VehicleTaskDebugInfo.h
--- a/src/dev_ng/game/ai/debug/types/VehicleTaskDebugInfo.h
+++ b/src/dev_ng/game/ai/debug/types/VehicleTaskDebugInfo.h
@@ -1,7 +1,7 @@
 #pragma once
 
 // game headers
-#include "ai\debug\types\VehicleDebugInfo.h"
+#include "ai/debug/types/VehicleDebugInfo.h"
 
 #if AI_DEBUG_OUTPUT_ENABLED
 
