diff --git a/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.h b/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.h
--- a/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.h
+++ b/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.h
@@ -1,6 +1,6 @@
 #pragma once
 // game headers
-#include "ai\debug\types\VehicleDebugInfo.h"
+#include "ai/debug/types/VehicleDebugInfo.h"
 
 #if AI_DEBUG_OUTPUT_ENABLED
 
