diff --git a/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.cpp b/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.cpp
--- a/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.cpp
+++ b/src/dev_ng/game/ai/debug/types/VehicleStuckDebugInfo.cpp
@@ -1,9 +1,9 @@
-#include "ai\debug\types\VehicleStuckDebugInfo.h"
+#include "ai/debug/types/VehicleStuckDebugInfo.h"
 
 #if AI_DEBUG_OUTPUT_ENABLED
 
-#include "vehicles\vehicle.h"
-#include "vehicleAi\VehicleIntelligence.h"
+#include "Vehicles/vehicle.h"
+#include "vehicleAi/VehicleIntelligence.h"
 
 CVehicleStuckDebugInfo::CVehicleStuckDebugInfo(const CVehicle* pVeh, DebugInfoPrintParams printParams, s32 iNumberOfLines)
 : CVehicleDebugInfo(pVeh, printParams, iNumberOfLines)
