diff --git a/src/dev_ng/game/ai/debug/types/VehicleNodeListDebugInfo.cpp b/src/dev_ng/game/ai/debug/types/VehicleNodeListDebugInfo.cpp
--- a/src/dev_ng/game/ai/debug/types/VehicleNodeListDebugInfo.cpp
+++ b/src/dev_ng/game/ai/debug/types/VehicleNodeListDebugInfo.cpp
@@ -1,8 +1,8 @@
-#include "ai\debug\types\VehicleNodeListDebugInfo.h"
+#include "ai/debug/types/VehicleNodeListDebugInfo.h"
 
 #if AI_DEBUG_OUTPUT_ENABLED
 
-#include "vehicleAi\VehicleIntelligence.h"
+#include "vehicleAi/VehicleIntelligence.h"
 #include "Task/System/TaskHelpers.h"
 
 CVehicleNodeListDebugInfo::CVehicleNodeListDebugInfo(const CVehicle* pVeh, DebugInfoPrintParams printParams, s32 iNumberOfLines)
