diff --git a/src/dev_ng/game/vfx/particles/PtFxManager.h b/src/dev_ng/game/vfx/particles/PtFxManager.h
--- a/src/dev_ng/game/vfx/particles/PtFxManager.h
+++ b/src/dev_ng/game/vfx/particles/PtFxManager.h
@@ -27,14 +27,14 @@
 
 // framework
 #include "vfx/vfxlist.h"
-#include "vfx/ptfx/ptfxCallbacks.h"
-#include "vfx/ptfx/ptfxManager.h"
+#include "vfx/ptfx/ptfxcallbacks.h"
+#include "vfx/ptfx/ptfxmanager.h"
 
 // game
-#include "Streaming/StreamingRequest.h"
-#include "Vfx/Particles/PtFxCollision.h"
-#include "Vfx/Particles/PtFxDebug.h"
-#include "Vfx/Particles/PtFxEntity.h"
+#include "streaming/streamingrequest.h"
+#include "vfx/particles/PtFxCollision.h"
+#include "vfx/particles/PtFxDebug.h"
+#include "vfx/particles/PtFxEntity.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
