diff --git a/src/dev_ng/game/Vehicles/Boat.h b/src/dev_ng/game/Vehicles/Boat.h
--- a/src/dev_ng/game/Vehicles/Boat.h
+++ b/src/dev_ng/game/Vehicles/Boat.h
@@ -11,12 +11,12 @@
 //
 #ifndef _BOAT_H_
 #define _BOAT_H_
-#include "renderer\hierarchyIds.h"
+#include "renderer/hierarchyIds.h"
 //#include "vehicles/AnchorHelper.h"
-#include "vehicles\door.h"
-#include "vehicles\handlingMgr.h"
-#include "vehicles\vehicle.h"
-#include "network\objects\entities\netObjBoat.h"
+#include "vehicles/door.h"
+#include "vehicles/handlingMgr.h"
+#include "vehicles/vehicle.h"
+#include "network/objects/entities/netObjBoat.h"
 #include "System/PadGestures.h"					// for define
 #include "control/replay/ReplaySettings.h"
 
