diff --git a/src/dev_ng/game/Vehicles/vehiclepopulation_shared.cpp b/src/dev_ng/game/Vehicles/vehiclepopulation_shared.cpp
--- a/src/dev_ng/game/Vehicles/vehiclepopulation_shared.cpp
+++ b/src/dev_ng/game/Vehicles/vehiclepopulation_shared.cpp
@@ -1,7 +1,7 @@
-#include "vehicles/vehiclepopulation.h"
+#include "Vehicles/vehiclepopulation.h"
 
 #include "camera/CamInterface.h"
-#include "vehicles/vehpopulation_channel.h"
+#include "Vehicles/vehpopulation_channel.h"
 
 #include "fwscene/world/WorldLimits.h"
 #include "fwsys/timer.h"
