diff --git a/src/dev_ng/game/Vehicles/VehiclePopulationJob.cpp b/src/dev_ng/game/Vehicles/VehiclePopulationJob.cpp
--- a/src/dev_ng/game/Vehicles/VehiclePopulationJob.cpp
+++ b/src/dev_ng/game/Vehicles/VehiclePopulationJob.cpp
@@ -1,19 +1,19 @@
 // Vehicle population SPU update
 // flavius.alecu: 25/10/2011
 
-#include "math/simpleMath.h"
+#include "math/simplemath.h"
 
 #include "vehiclepopulation.h"
 
-#include "BaseTypes.h"
+#include "basetypes.h"
 
-#include "renderer\Renderer.h"			// JW- had to move this up here to get a successful compile...
+#include "renderer/Renderer.h"			// JW- had to move this up here to get a successful compile...
 
 #include "grcore/viewport.h"
 #include "grcore/viewport_inline.h"
 #include "VehiclePopulationAsync.h"
-#include "vector/Vector4.h"
-#include "scene/world/gameWorld.h"
+#include "vector/vector4.h"
+#include "scene/world/GameWorld.h"
 #include "system/timer.h"
 #include "fwscene/world/WorldLimits.h"
 
