diff --git a/src/dev_ng/game/Vehicles/vehiclestorage.h b/src/dev_ng/game/Vehicles/vehiclestorage.h
--- a/src/dev_ng/game/Vehicles/vehiclestorage.h
+++ b/src/dev_ng/game/Vehicles/vehiclestorage.h
@@ -11,7 +11,7 @@
 #ifndef _VEHICLESTORAGE_H_
 #define _VEHICLESTORAGE_H_
 
-//#include "vehicles\vehicle.h"
+//#include "vehicles/vehicle.h"
 #include "modelinfo/VehicleModelInfoVariation.h"
 
 #include "physics/WorldProbe/worldprobe.h"
