diff --git a/src/dev_ng/game/system/EndUserBenchmark.cpp b/src/dev_ng/game/system/EndUserBenchmark.cpp
--- a/src/dev_ng/game/system/EndUserBenchmark.cpp
+++ b/src/dev_ng/game/system/EndUserBenchmark.cpp
@@ -5,10 +5,10 @@
 #include "time.h"
 #include "fwsys/timer.h"
 #include "grcore/diag_d3d.h"
-#include "grcore\adapter_d3d11.h"
-#include "script\script.h"
-#include "script\streamedscripts.h"
-#include "script\thread.h"
+#include "grcore/adapter_d3d11.h"
+#include "script/script.h"
+#include "script/streamedscripts.h"
+#include "script/thread.h"
 #include "camera/CamInterface.h"
 #include "network/NetworkInterface.h"
 
@@ -18,7 +18,7 @@
 #include "bank/bkmgr.h"
 #endif
 
-#include "Network/Live/NetworkTelemetry.h"
+#include "network/Live/NetworkTelemetry.h"
 
 eFRONTEND_MENU_VERSION					EndUserBenchmarks::m_bMenuTrigger = 0;
 bool									EndUserBenchmarks::m_bActive = false;
