diff --git a/src/dev_ng/rage/base/src/rline/youtube/rlyoutubetasks.cpp b/src/dev_ng/rage/base/src/rline/youtube/rlyoutubetasks.cpp
--- a/src/dev_ng/rage/base/src/rline/youtube/rlyoutubetasks.cpp
+++ b/src/dev_ng/rage/base/src/rline/youtube/rlyoutubetasks.cpp
@@ -631,7 +631,7 @@ void rlYoutubeUploadChunkTask::Finish(const FinishType finishType, const int res
 			if (bytesRangeBufLen > 0 &&								// We read in a byte range
 				!strncmp(bytesRangeBuf,"bytes=0-",8) &&				// The byte Range has the correct format
 				strlen(bytesRangeBuf) >= 9 &&						// something exists at the end of the byte range
-				sscanf_s(bytesRangeBuf + 8, "%" I64FMT "u", &bytesUploaded))	// Read in the byte range
+				sscanf(bytesRangeBuf + 8, "%" I64FMT "u", &bytesUploaded))	// Read in the byte range
 			{
 				// end range is zero-indexed, so the total number of bytes uploaded is the ending byte + 1
 				m_UploadPtr->BytesUploaded = bytesUploaded + 1;
@@ -733,7 +733,7 @@ void rlYoutubeGetUploadStatusTask::Finish(const FinishType finishType, const int
 			if (bytesRangeBufLen > 0 &&								// We read in a byte range
 				!strncmp(bytesRangeBuf,"bytes=0-",8) &&				// The byte Range has the correct format
 				strlen(bytesRangeBuf) >= 9 &&						// something exists at the end of the byte range
-				sscanf_s(bytesRangeBuf + 8, "%" I64FMT "u", &bytesUploaded) == 1)	// Read in the byte range
+				sscanf(bytesRangeBuf + 8, "%" I64FMT "u", &bytesUploaded) == 1)	// Read in the byte range
 			{
 				m_UploadPtr->BytesUploaded = bytesUploaded;
 			}
