diff --git a/src/dev_ng/rage/base/src/input/mouse.cpp b/src/dev_ng/rage/base/src/input/mouse.cpp
--- a/src/dev_ng/rage/base/src/input/mouse.cpp
+++ b/src/dev_ng/rage/base/src/input/mouse.cpp
@@ -15,6 +15,10 @@
 #include "system/wndproc.h"
 #include "system/timer.h"
 #include "math/amath.h"
+#include "system/platformdefs.h"
+#if __LINUX // header from DXVK
+#include <windows.h>
+#endif
 
 using namespace rage;
 
@@ -298,6 +302,44 @@ void ioMouse::Update(bool UNUSED_PARAM(ignoreInput)) {
 #endif // REMOTE_INPUT_SUPPORT
 }
 
+#elif RSG_LINUX
+
+void ioMouse::Begin(bool inWin, bool isExclusive)
+{
+}
+
+bool ioMouse::IsLeftRightButtonSwapped()
+{
+	return false;
+}
+
+void ioMouse::End()
+{
+}
+
+void ioMouse::Update(bool ignoreInput)
+{
+}
+
+void ioMouse::SetCursorVisible(bool visible)
+{
+}
+
+bool ioMouse::SetCursorPosition(float x, float y)
+{
+	return false;
+}
+
+#if !__FINAL
+void rage::ioMouse::SetAbsoluteOnly( bool absoluteOnly )
+{
+}
+#endif // !__FINAL
+
+void rage::ioMouse::RecaptureLostDevices()
+{
+}
+
 #elif RSG_PC
 
 #include "winpriv.h"
