diff --git a/src/dev_ng/game/ik/solvers/TorsoReactSolver.h b/src/dev_ng/game/ik/solvers/TorsoReactSolver.h
--- a/src/dev_ng/game/ik/solvers/TorsoReactSolver.h
+++ b/src/dev_ng/game/ik/solvers/TorsoReactSolver.h
@@ -17,11 +17,11 @@
 
 #include "crbody/iksolverbase.h"
 
-#include "fwtl/pool.h"
+#include "fwtl/Pool.h"
 #include "vector/vector3.h"
 
-#include "Animation/AnimBones.h"
-#include "Animation/debug/AnimDebug.h"
+#include "animation/AnimBones.h"
+#include "animation/debug/AnimDebug.h"
 #include "ik/IkRequest.h"
 #include "Peds/pedDefines.h"
 #include "scene/RegdRefTypes.h"
