diff --git a/src/dev_ng/game/network/General/NetworkVehicleModelDoorLockTable.cpp b/src/dev_ng/game/network/General/NetworkVehicleModelDoorLockTable.cpp
--- a/src/dev_ng/game/network/General/NetworkVehicleModelDoorLockTable.cpp
+++ b/src/dev_ng/game/network/General/NetworkVehicleModelDoorLockTable.cpp
@@ -1,7 +1,7 @@
 #include "NetworkVehicleModelDoorLockTable.h"
 #include "ModelInfo/ModelInfo.h"
 #include "Vehicles/Vehicle.h"
-#include "Script\script.h"
+#include "Script/script.h"
 #include "scene/world/GameWorld.h"
 #include "peds/PlayerInfo.h"
 #include "peds/ped.h"
