diff --git a/src/dev_ng/game/network/General/NetworkSerialisers.h b/src/dev_ng/game/network/General/NetworkSerialisers.h
--- a/src/dev_ng/game/network/General/NetworkSerialisers.h
+++ b/src/dev_ng/game/network/General/NetworkSerialisers.h
@@ -1,8 +1,8 @@
 #ifndef NETWORK_SERIALISERS_H
 #define NETWORK_SERIALISERS_H
 
-#include "task/system/TaskClassInfo.h"
-#include "entity/archetypeManager.h"
+#include "task/System/TaskClassInfo.h"
+#include "entity/archetypemanager.h"
 #include "fwanimation/animdefines.h"
 #include "fwanimation/animmanager.h"
 #include "fwnet/netlog.h"
@@ -10,9 +10,9 @@
 #include "fwnet/netserialisers.h"
 #include "fwnet/nettypes.h"
 #include "data/bitbuffer.h"
-#include "modelInfo/ModelInfo.h"
-#include "peds\CombatData.h"
-#include "vehicles\Metadata\VehicleEnterExitFlags.h"
+#include "modelinfo/ModelInfo.h"
+#include "Peds/CombatData.h"
+#include "Vehicles/Metadata/VehicleEnterExitFlags.h"
 
 typedef CVehicleEnterExitFlags::FlagsBitSet VehicleEnterExitFlags;
 typedef CCombatData::BehaviourFlagsBitSet CombatBehaviorFlags;
