diff --git a/src/dev_ng/game/task/Combat/CombatMeleeDebug.cpp b/src/dev_ng/game/task/Combat/CombatMeleeDebug.cpp
--- a/src/dev_ng/game/task/Combat/CombatMeleeDebug.cpp
+++ b/src/dev_ng/game/task/Combat/CombatMeleeDebug.cpp
@@ -1,23 +1,23 @@
 // File header
-#include "Task/Combat/CombatMeleeDebug.h"
+#include "task/Combat/CombatMeleeDebug.h"
 
 // Rage headers
 #include "bank/bank.h"
 #include "bank/bkmgr.h"
 
 // Game headers
-#include "Debug/DebugScene.h"
-#include "Peds/Ped.h"
-#include "Peds/Ped.h"
+#include "debug/DebugScene.h"
+#include "Peds/ped.h"
+#include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
 #include "Peds/rendering/PedVariationDebug.h"
-#include "Scene/World/GameWorld.h"
-#include "Task/Combat/TaskCombatMelee.h"
-#include "Task/General/TaskBasic.h"
-#include "Task/Movement/TaskGoTo.h"
-#include "Task/Movement/TaskSlideToCoord.h"
-#include "Weapons/Weapon.h"
-#include "event/eventDamage.h"
+#include "scene/world/GameWorld.h"
+#include "task/Combat/TaskCombatMelee.h"
+#include "task/General/TaskBasic.h"
+#include "task/Movement/TaskGoTo.h"
+#include "task/Movement/TaskSlideToCoord.h"
+#include "weapons/Weapon.h"
+#include "event/EventDamage.h"
 
 AI_OPTIMISATIONS()
 
