diff --git a/src/dev_ng/game/task/Combat/TaskAnimatedHitByExplosion.h b/src/dev_ng/game/task/Combat/TaskAnimatedHitByExplosion.h
--- a/src/dev_ng/game/task/Combat/TaskAnimatedHitByExplosion.h
+++ b/src/dev_ng/game/task/Combat/TaskAnimatedHitByExplosion.h
@@ -5,9 +5,9 @@
 #define TASK_ANIMATED_HIT_BY_EXPLOSION_H
 
 // Game headers
-#include "Task/System/Task.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/Tuning.h"
+#include "task/System/Task.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
 
 class CTaskAnimatedHitByExplosion : public CTask
 {
