diff --git a/src/dev_ng/game/game/Clock.cpp b/src/dev_ng/game/game/Clock.cpp
--- a/src/dev_ng/game/game/Clock.cpp
+++ b/src/dev_ng/game/game/Clock.cpp
@@ -19,27 +19,27 @@
 #include "parser/manager.h"
 
 // framework
-#include "fwdebug\debugdraw.h"
+#include "fwdebug/debugdraw.h"
 #include "fwsys/timer.h"
 
 // game
-#include "Core/Game.h"
-#include "Debug\DebugGlobals.h"
-#include "Debug\MarketingTools.h"
-#include "Network\Events\NetworkEventTypes.h"
-#include "Network\NetworkInterface.h"
-#include "Scene\DataFileMgr.h"
-#include "System\ControlMgr.h"
-#include "Shaders\ShaderLib.h"
-#include "TimeCycle\TimeCycle.h"
+#include "Core/game.h"
+#include "debug/DebugGlobals.h"
+#include "debug/MarketingTools.h"
+#include "network/Events/NetworkEventTypes.h"
+#include "network/NetworkInterface.h"
+#include "scene/DataFileMgr.h"
+#include "system/controlMgr.h"
+#include "shaders/ShaderLib.h"
+#include "timecycle/TimeCycle.h"
 
 // game (for ProcessTimeSkip command - doesn't really belong in here)
-#include "Audio\PoliceScanner.h"
-#include "Audio\RadioAudioEntity.h"
-#include "peds/Ped.h"
-#include "Renderer\PostProcessFx.h"
-#include "Scene\World\GameWorld.h"
-#include "control\replay\ReplaySettings.h"
+#include "audio/policescanner.h"
+#include "audio/radioaudioentity.h"
+#include "Peds/ped.h"
+#include "renderer/PostProcessFX.h"
+#include "scene/world/GameWorld.h"
+#include "control/replay/ReplaySettings.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
