diff --git a/src/dev_ng/game/game/PopMultiplierAreas.cpp b/src/dev_ng/game/game/PopMultiplierAreas.cpp
--- a/src/dev_ng/game/game/PopMultiplierAreas.cpp
+++ b/src/dev_ng/game/game/PopMultiplierAreas.cpp
@@ -11,18 +11,18 @@
 
 // Framework headers
 #include "grcore/debugdraw.h"
-#include "fwmaths\random.h"
+#include "fwmaths/random.h"
 #include "fwsys/timer.h"
 
 // Game headers
-#include "game/popmultiplierareas.h"
+#include "game/PopMultiplierAreas.h"
 #include "camera/CamInterface.h"
-#include "debug\vectormap.h"
+#include "debug/VectorMap.h"
 
 // network includes 
 #include "network/NetworkInterface.h"
-#include "network/objects/entities/NetObjPlayer.h"
-#include "network/players/NetworkPlayerMgr.h"
+#include "network/Objects/Entities/NetObjPlayer.h"
+#include "network/Players/NetworkPlayerMgr.h"
 
 AI_OPTIMISATIONS()
 NETWORK_OPTIMISATIONS()
