diff --git a/src/dev_ng/game/game/wind.cpp b/src/dev_ng/game/game/wind.cpp
--- a/src/dev_ng/game/game/wind.cpp
+++ b/src/dev_ng/game/game/wind.cpp
@@ -12,22 +12,22 @@
 //  INCLUDES
 ///////////////////////////////////////////////////////////////////////////////
 
-#include "Wind.h"
+#include "wind.h"
 
 // rage
 #include "profile/timebars.h"
 #include "pheffects/wind.h"
 
 // game 
-#include "Camera/CamInterface.h"
+#include "camera/CamInterface.h"
 #include "control/replay/replay.h"
-#include "Core/Game.h"
-#include "Peds/Ped.h"
-#include "Renderer/River.h"
-#include "Scene/Portals/Portal.h"
-#include "Scene/World/GameWorld.h"
+#include "Core/game.h"
+#include "Peds/ped.h"
+#include "renderer/River.h"
+#include "scene/portals/Portal.h"
+#include "scene/world/GameWorld.h"
 #include "fwsys/timer.h"
-#include "Vfx/VfxHelper.h"
+#include "vfx/VfxHelper.h"
 
 
 ///////////////////////////////////////////////////////////////////////////////
