diff --git a/src/dev_ng/game/task/Response/TaskReactAndFlee.h b/src/dev_ng/game/task/Response/TaskReactAndFlee.h
--- a/src/dev_ng/game/task/Response/TaskReactAndFlee.h
+++ b/src/dev_ng/game/task/Response/TaskReactAndFlee.h
@@ -8,10 +8,10 @@
 #include "ai/AITarget.h"
 #include "event/EventWeapon.h"
 #include "Peds/QueriableInterface.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskFSMClone.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/Tuning.h"
+#include "task/System/Task.h"
+#include "task/System/TaskFSMClone.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
 
 ////////////////////////////////////////////////////////////////////////////////
 //
