diff --git a/src/dev_ng/game/task/Response/TaskReactToExplosion.h b/src/dev_ng/game/task/Response/TaskReactToExplosion.h
--- a/src/dev_ng/game/task/Response/TaskReactToExplosion.h
+++ b/src/dev_ng/game/task/Response/TaskReactToExplosion.h
@@ -10,9 +10,9 @@
 // Game headers
 #include "ai/AITarget.h"
 #include "task/Combat/Subtasks/TaskShellShocked.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/Tuning.h"
+#include "task/System/Task.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
 
 ////////////////////////////////////////////////////////////////////////////////
 // CTaskReactToExplosion
