diff --git a/src/dev_ng/game/scene/streamer/SceneLoader.cpp b/src/dev_ng/game/scene/streamer/SceneLoader.cpp
--- a/src/dev_ng/game/scene/streamer/SceneLoader.cpp
+++ b/src/dev_ng/game/scene/streamer/SceneLoader.cpp
@@ -14,12 +14,12 @@ STREAMING_OPTIMISATIONS()
 
 #include "fwscene/stores/maptypesstore.h"
 #include "fwsys/gameskeleton.h"
-#include "game/clock.h"
+#include "game/Clock.h"
 #include "modelinfo/BaseModelInfo.h"
 #include "modelinfo/TimeModelInfo.h"
-#include "objects/objectpopulation.h"
-#include "pathserver/pathserver.h"
-#include "peds/PlayerInfo.h"
+#include "Objects/objectpopulation.h"
+#include "pathserver/PathServer.h"
+#include "Peds/PlayerInfo.h"
 #include "renderer/PlantsMgr.h"
 #include "renderer/Lights/LightEntity.h"
 #include "fwscene/lod/LodTypes.h"
@@ -31,7 +31,7 @@ STREAMING_OPTIMISATIONS()
 #include "fwscene/world/EntityContainer.h"
 #include "streaming/streaming.h"
 #include "streaming/streamingvisualize.h"
-#include "scene/ipl/iplcullbox.h"
+#include "scene/ipl/IplCullBox.h"
 #include "scene/LoadScene.h"
 #include "script/script.h"
 #include "physics/physics.h"
