diff --git a/src/dev_ng/game/vehicleAi/pathzones.h b/src/dev_ng/game/vehicleAi/pathzones.h
--- a/src/dev_ng/game/vehicleAi/pathzones.h
+++ b/src/dev_ng/game/vehicleAi/pathzones.h
@@ -4,8 +4,8 @@
 // rage includes
 #include "atl/array.h"
 #include "parser/macros.h"
-#include "Vector/Vector2.h"
-#include "Vector/vector3.h"
+#include "vector/vector2.h"
+#include "vector/vector3.h"
 
 #define MAX_NUM_PATHZONES 4
 #define MAX_NUM_PATHZONE_MAPPINGS (MAX_NUM_PATHZONES * (MAX_NUM_PATHZONES - 1))
