diff --git a/src/dev_ng/game/vehicleAi/VehicleIntelligenceFactory.cpp b/src/dev_ng/game/vehicleAi/VehicleIntelligenceFactory.cpp
--- a/src/dev_ng/game/vehicleAi/VehicleIntelligenceFactory.cpp
+++ b/src/dev_ng/game/vehicleAi/VehicleIntelligenceFactory.cpp
@@ -1,4 +1,4 @@
-#include "vehicleAI\VehicleIntelligenceFactory.h"
+#include "vehicleAI/VehicleIntelligenceFactory.h"
 #include "VehicleIntelligence.h"
 #include "vehicleAi/task/TaskVehicleMissionBase.h"
 
