diff --git a/src/dev_ng/game/vehicleAi/VehicleIntelligence.h b/src/dev_ng/game/vehicleAi/VehicleIntelligence.h
--- a/src/dev_ng/game/vehicleAi/VehicleIntelligence.h
+++ b/src/dev_ng/game/vehicleAi/VehicleIntelligence.h
@@ -2,16 +2,16 @@
 #define VEHICLE_INTELLIGENCE_H
 
 #include "ai/ExpensiveProcess.h"
-#include "Debug/Debug.h"
+#include "debug/Debug.h"
 #include "Peds/PedEventScanner.h"
-#include "Event/EventGroup.h"
+#include "event/EventGroup.h"
 #include "scene/RegdRefTypes.h"
-#include "Task/System/TaskManager.h"
+#include "task/System/TaskManager.h"
 #include "vehicleAi/pathfind.h"
 #include "vehicleAi/VehMission.h"
 #include "vehicleAi/VehicleNodeList.h"
 #include "vehicleAi/task/TaskVehicleMissionBase.h"
-#include "Templates/CircularArray.h"
+#include "templates/CircularArray.h"
 
 struct tGameplayCameraSettings;
 
