diff --git a/src/dev_ng/game/renderer/Lights/AsyncLightOcclusionMgr.cpp b/src/dev_ng/game/renderer/Lights/AsyncLightOcclusionMgr.cpp
--- a/src/dev_ng/game/renderer/Lights/AsyncLightOcclusionMgr.cpp
+++ b/src/dev_ng/game/renderer/Lights/AsyncLightOcclusionMgr.cpp
@@ -2,21 +2,21 @@
 // INCLUDES
 // =============================================================================================== //
 
-#include "Renderer/Lights/AsyncLightOcclusionMgr.h"
+#include "renderer/Lights/AsyncLightOcclusionMgr.h"
 
 // rage headers
 #include "softrasterizer/scansetup.h"
 
 // framework headers
-#include "fwscene/scan/scan.h"
-#include "fwscene/scan/scanDebug.h"
+#include "fwscene/scan/Scan.h"
+#include "fwscene/scan/ScanDebug.h"
 #include "system/dependencyscheduler.h"
 
 // game headers
-#include "Debug/Rendering/DebugLights.h"
-#include "Renderer/Lights/Lights.h"
-#include "Renderer/occlusion.h"
-#include "Renderer/ProcessLightOcclusionAsync.h"
+#include "debug/Rendering/DebugLights.h"
+#include "renderer/Lights/lights.h"
+#include "renderer/occlusion.h"
+#include "renderer/ProcessLightOcclusionAsync.h"
 
 RENDER_OPTIMISATIONS()
 
