diff --git a/src/dev_ng/game/renderer/Lights/LightGroup.cpp b/src/dev_ng/game/renderer/Lights/LightGroup.cpp
--- a/src/dev_ng/game/renderer/Lights/LightGroup.cpp
+++ b/src/dev_ng/game/renderer/Lights/LightGroup.cpp
@@ -3,10 +3,10 @@
 // =============================================================================================== //
 
 // Game headers
-#include "Renderer/Lights/LightGroup.h"
-#include "Debug/Rendering/DebugLighting.h"
+#include "renderer/Lights/LightGroup.h"
+#include "debug/Rendering/DebugLighting.h"
 #include "renderer/SpuPM/SpuPmMgr.h"
-#include "renderer/Lights/Lights.h"
+#include "renderer/Lights/lights.h"
 #include "timecycle/TimeCycle.h"
 
 // Framework headers
