diff --git a/src/dev_ng/game/weapons/helpers/WeaponHelpers.cpp b/src/dev_ng/game/weapons/helpers/WeaponHelpers.cpp
--- a/src/dev_ng/game/weapons/helpers/WeaponHelpers.cpp
+++ b/src/dev_ng/game/weapons/helpers/WeaponHelpers.cpp
@@ -5,15 +5,15 @@
 //
 
 // File header
-#include "Weapons/Helpers/WeaponHelpers.h"
+#include "weapons/helpers/WeaponHelpers.h"
 
 // Rage headers
 #include "crskeleton/skeletondata.h"
 #include "parser/treenode.h"
 
 // Game headers
-#include "ModelInfo/ModelInfo.h"
-#include "ModelInfo/ModelInfo_Factories.h"
+#include "modelinfo/ModelInfo.h"
+#include "modelinfo/ModelInfo_Factories.h"
 #include "physics/gtaArchetype.h"
 
 // Parser header
