diff --git a/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.cpp b/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.cpp
--- a/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.cpp
+++ b/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.cpp
@@ -10,7 +10,7 @@
 #include "control/replay/PacketBasics.h"
 
 #include "audio/pedaudioentity.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 
 
 //////////////////////////////////////////////////////////////////////////
