diff --git a/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.h b/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.h
--- a/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.h
+++ b/src/dev_ng/game/control/replay/Audio/MeleeAudioPacket.h
@@ -6,12 +6,12 @@
 #ifndef INC_MELEEPACKET_H_
 #define INC_MELEEPACKET_H_
 
-#include "control\replay\ReplaySettings.h"
+#include "control/replay/ReplaySettings.h"
 
 // Replay only setup for win32
 #if GTA_REPLAY
 
-#include "control/replay/effects/ParticlePacket.h"
+#include "control/replay/Effects/ParticlePacket.h"
 #include "Peds/ped.h"
 
 //////////////////////////////////////////////////////////////////////////
