diff --git a/src/dev_ng/rage/base/src/bank/group.h b/src/dev_ng/rage/base/src/bank/group.h
--- a/src/dev_ng/rage/base/src/bank/group.h
+++ b/src/dev_ng/rage/base/src/bank/group.h
@@ -616,7 +616,7 @@ protected:
 	void SetClosed();
 
 protected:
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 	void WindowCreate();
 	rageLRESULT WindowMessage(rageUINT,rageWPARAM,rageLPARAM);
 	int WindowResize(int x,int y,int width,int height);
