diff --git a/src/dev_ng/rage/base/src/bank/packet.cpp b/src/dev_ng/rage/base/src/bank/packet.cpp
--- a/src/dev_ng/rage/base/src/bank/packet.cpp
+++ b/src/dev_ng/rage/base/src/bank/packet.cpp
@@ -340,7 +340,7 @@ bool bkRemotePacket::ConnectToRag(bool viewerApp)
 				}
 		} while(!result);
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 		char szWindowTitle[512];
 		/* Get the console window title. */ 
 		GetConsoleTitle(szWindowTitle, 512); 
