diff --git a/src/dev_ng/rage/base/src/bank/title.h b/src/dev_ng/rage/base/src/bank/title.h
--- a/src/dev_ng/rage/base/src/bank/title.h
+++ b/src/dev_ng/rage/base/src/bank/title.h
@@ -28,7 +28,7 @@ protected:
 	static void RemoteHandler(const bkRemotePacket& p);
 
 private:
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
 	void WindowCreate();
 	rageLRESULT WindowMessage(rageUINT,rageWPARAM,rageLPARAM);
 #endif
