diff --git a/src/dev_ng/rage/base/src/bank/treelist.h b/src/dev_ng/rage/base/src/bank/treelist.h
--- a/src/dev_ng/rage/base/src/bank/treelist.h
+++ b/src/dev_ng/rage/base/src/bank/treelist.h
@@ -456,7 +456,7 @@ protected:
     void RemoteCreate();
     void RemoteUpdate();
 
-#if __WIN32PC
+#if __WIN32PC && !__LINUX
     void WindowCreate();
 	rageLRESULT WindowMessage(rageUINT,rageWPARAM,rageLPARAM) { return 0; }
 #endif
