diff --git a/src/dev_ng/rage/framework/src/streaming/CacheLoader.cpp b/src/dev_ng/rage/framework/src/streaming/CacheLoader.cpp
--- a/src/dev_ng/rage/framework/src/streaming/CacheLoader.cpp
+++ b/src/dev_ng/rage/framework/src/streaming/CacheLoader.cpp
@@ -4,12 +4,12 @@
 // Copyright (C) 1999-2009 Rockstar Games.  All Rights Reserved.
 //
 
-#include "streaming/cacheloader.h"
+#include "streaming/CacheLoader.h"
 
 #include "file/asset.h"
 #include "file/default_paths.h"
-#include "net/netHardware.h"  // For netHardware::GetLocalIpAddress
-#include "net/netAddress.h"
+#include "net/nethardware.h"  // For netHardware::GetLocalIpAddress
+#include "net/netaddress.h"
 #include "system/param.h"
 #include "system/platform.h"
 
@@ -28,7 +28,7 @@
 #include "system/exec.h"
 #endif
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 #include <windef.h>
 #include <winbase.h>
 #endif
@@ -429,7 +429,7 @@ void strCacheLoader::Shutdown()
 		}
 	}
 
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
 	if(PARAM_exitaftercachesave.Get())
 	{
 		cacheloaderDisplayf("Completed processing cachefile - terminating per -exitaftercachesave option.");
@@ -1216,7 +1216,7 @@ void strCacheLoader::BuildCacheFilename(const char* const pFilename)
 				"%s%s_cache_packaged_%c%s%s.dat", deviceAndPath,
 				ASSET.FileName(baseName),
 				g_sysHostPlatform,
-				( __BANK ? "_bank" : "" ),
+				(!__PACKEDDEBUG ? "_bank" : "" ),
 				ipAddress
 				);
 		}
@@ -1227,7 +1227,7 @@ void strCacheLoader::BuildCacheFilename(const char* const pFilename)
 				"%s%s_cache_%c%s%s.dat", deviceAndPath,
 				ASSET.FileName(baseName),
 				g_sysHostPlatform,
-				( __BANK ? "_bank" : "" ),
+				(!__PACKEDDEBUG ? "_bank" : "" ),
 				ipAddress
 			);
 		}
