diff --git a/src/dev_ng/game/vs_project4_lib/CMakeLists.txt b/src/dev_ng/game/vs_project4_lib/CMakeLists.txt
new file mode 100644
--- /dev/null
+++ b/src/dev_ng/game/vs_project4_lib/CMakeLists.txt
@@ -0,0 +1,39 @@
+project(game4_lib)
+
+set(TARGET_TYPE "StaticLib")
+include(${CMAKE_SOURCE_DIR}/BuildOptions.cmake)
+
+if (MSVC)
+    add_compile_options(/FI${CMAKE_SOURCE_DIR}/game/basetypes.h)
+    add_compile_options(/FI${CMAKE_SOURCE_DIR}/game/game_config.h)
+else()
+    add_compile_options(-include${CMAKE_SOURCE_DIR}/game/basetypes.h)
+    add_compile_options(-include${CMAKE_SOURCE_DIR}/game/game_config.h)
+    add_definitions(-Dlinux)
+    remove_definitions(-DWIN64)
+endif()
+
+set(SRC_FILES
+    "_Unity/_unity_game4_lib_templates_text.cpp"
+    "_Unity/_unity_game4_lib_timecycle_tools.cpp"
+    "_Unity/_unity_game4_lib_vehicleai.cpp"
+    "_Unity/_unity_game4_lib_vehicles.cpp"
+    "_Unity/_unity_game4_lib_vfx.cpp"
+    "_Unity/_unity_game4_lib_weapons.cpp"
+)
+
+add_library(${PROJECT_NAME} STATIC ${SRC_FILES})
+
+target_include_directories(${PROJECT_NAME} PRIVATE
+    ${DIRECTX_INCLUDE_DIRS}
+    ${CMAKE_SOURCE_DIR}/game
+    ${CMAKE_SOURCE_DIR}/rage/3rdparty/AMD/AMD_ShadowFX/amd_lib/shared/common/inc
+    ${CMAKE_SOURCE_DIR}/rage/base/src
+    ${CMAKE_SOURCE_DIR}/rage/framework/src
+    ${CMAKE_SOURCE_DIR}/rage/naturalmotion/include
+    ${CMAKE_SOURCE_DIR}/rage/naturalmotion/src
+    ${CMAKE_SOURCE_DIR}/rage/script/src
+    ${CMAKE_SOURCE_DIR}/rage/suite/src
+)
+
+target_link_libraries(${PROJECT_NAME})
