diff --git a/src/dev_ng/game/script/Handlers/GameScriptHandlerNetwork.cpp b/src/dev_ng/game/script/Handlers/GameScriptHandlerNetwork.cpp
--- a/src/dev_ng/game/script/Handlers/GameScriptHandlerNetwork.cpp
+++ b/src/dev_ng/game/script/Handlers/GameScriptHandlerNetwork.cpp
@@ -5,30 +5,30 @@
 //
 
 
-#include "script/handlers/GameScriptHandlerNetwork.h"
+#include "script/Handlers/GameScriptHandlerNetwork.h"
 
 // game includes
 #include "event/EventGroup.h"
 #include "event/EventNetwork.h"
-#include "Network/Arrays/NetworkArrayMgr.h"
-#include "Network/Events/NetworkEventTypes.h"
-#include "Network/Network.h"
-#include "Network/NetworkInterface.h"
-#include "Network/Objects/Entities/NetObjGame.h"
-#include "Network/Players/NetGamePlayer.h"
-#include "Network/Sessions/NetworkSession.h"
-#include "Peds/Ped.h"
+#include "network/Arrays/NetworkArrayMgr.h"
+#include "network/Events/NetworkEventTypes.h"
+#include "network/Network.h"
+#include "network/NetworkInterface.h"
+#include "network/Objects/Entities/NetObjGame.h"
+#include "network/Players/NetGamePlayer.h"
+#include "network/Sessions/NetworkSession.h"
+#include "Peds/ped.h"
 #include "pickups/PickupManager.h"
 #include "pickups/PickupPlacement.h"
 #include "renderer/DrawLists/drawList.h"
-#include "Scene/Physical.h"
+#include "scene/Physical.h"
 #include "scene/world/GameWorld.h"
-#include "Script/handlers/GameScriptEntity.h"
-#include "Script/handlers/GameScriptResources.h"
-#include "Script/gta_thread.h"
+#include "script/handlers/GameScriptEntity.h"
+#include "script/handlers/GameScriptResources.h"
+#include "script/gta_thread.h"
 #include "script/script.h"
 #include "script/script_channel.h"
-#include "network/objects/NetworkObjectPopulationMgr.h"
+#include "network/Objects/NetworkObjectPopulationMgr.h"
 
 // framework includes
 #include "fwnet/netObjectMgrBase.h"
