diff --git a/src/dev_ng/rage/base/src/grcore/buffer_d3d11.h b/src/dev_ng/rage/base/src/grcore/buffer_d3d11.h
--- a/src/dev_ng/rage/base/src/grcore/buffer_d3d11.h
+++ b/src/dev_ng/rage/base/src/grcore/buffer_d3d11.h
@@ -41,7 +41,7 @@ enum grcUAVBufferFlags
 	grcBuffer_UAV_FLAG_COUNTER   = 0x4		//D3D11_BUFFER_UAV_FLAG_COUNTER
 };
 
-#if RSG_PC && __D3D11
+#if __D3D11 && (RSG_PC || RSG_LINUX)
 
 enum grcBufferCreateType
 {
@@ -199,7 +199,7 @@ inline const void *grcBufferD3D11::GetUnsafeReadPtr() const
 	return m_pCPUCopyOfData;
 }
 
-#endif // RSG_PC && __D3D11
+#endif // __D3D11 && (RSG_PC || RSG_LINUX)
 
 } // namespace rage
 
