diff --git a/src/dev_ng/rage/base/src/grcore/effect.h b/src/dev_ng/rage/base/src/grcore/effect.h
--- a/src/dev_ng/rage/base/src/grcore/effect.h
+++ b/src/dev_ng/rage/base/src/grcore/effect.h
@@ -350,7 +350,7 @@ typedef int IntVector[4];
 
 #define SUPPORT_UAV	(__D3D11 || RSG_ORBIS)
 #if SUPPORT_UAV
-#if RSG_PC &&__D3D11
+#if (RSG_PC || RSG_LINUX) &&__D3D11
 class grcBufferD3D11;
 class grcBufferD3D11Resource;
 class grcRenderTargetDX11;
@@ -501,7 +501,7 @@ class  grcInstanceData;
 class  grcFragmentProgram;
 class  grcParameter;
 class  grcVertexProgram;
-#if RSG_PC || RSG_DURANGO || RSG_ORBIS
+#if RSG_PC || RSG_DURANGO || RSG_ORBIS || RSG_LINUX
 class grcComputeProgram;
 class grcHullProgram;
 #endif
@@ -964,7 +964,7 @@ public:
 	u32 GetNameHash() const { return NameHash; }
 	u32 GetSemanticHash() const { return SemanticHash; }
 	int GetCount() const { return Count; }
-#if RSG_PC || RSG_DURANGO || RSG_ORBIS
+#if RSG_PC || RSG_DURANGO || RSG_ORBIS || RSG_LINUX
 	void SetParentCBuf(grcCBuffer *pGrcCbuf)					{ FastAssert(pGrcCbuf); CBuf = pGrcCbuf; }
 	grcCBuffer *GetParentCBuf()	const							{ return CBuf;	}
 	u32 GetCBufferNameHash() const								{ return CBufferNameHash; }
@@ -1716,7 +1716,7 @@ private:
 #endif
 };
 
-#if RSG_PC || RSG_DURANGO || RSG_ORBIS
+#if RSG_PC || RSG_DURANGO || RSG_ORBIS || RSG_LINUX
 class grcComputeProgram: public grcProgram {
 	friend class grcEffect;
 	friend class spuGeometryDma;
@@ -1966,7 +1966,7 @@ private:
 #endif	//RSG_ORBIS
 };
 
-#else	// RSG_PC || RSG_DURANGO || RSG_ORBIS
+#else	// RSG_PC || RSG_DURANGO || RSG_ORBIS || RSG_LINUX
 
 template <bool HasStreamOut>
 class grcPlaceHolderProgram: public grcProgram {
