diff --git a/src/dev_ng/rage/base/src/grcore/resourcecache.h b/src/dev_ng/rage/base/src/grcore/resourcecache.h
--- a/src/dev_ng/rage/base/src/grcore/resourcecache.h
+++ b/src/dev_ng/rage/base/src/grcore/resourcecache.h
@@ -129,8 +129,8 @@
 #endif // !__D3D11
 
 #include	"dds.h"
-#include	"system\ipc.h"
-#include	"system\criticalsection.h"
+#include	"system/ipc.h"
+#include	"system/criticalsection.h"
 
 #include	"atl/array.h"
 #include	"atl/map.h"
@@ -187,6 +187,9 @@ struct ID3D11UnorderedAccessView;
 struct ID3D11Resource;
 #endif
 
+#if __LINUX // already defined in DXVK
+#include <windows.h>
+#else
 typedef int                 INT;
 typedef unsigned int        UINT;
 typedef unsigned long       DWORD;
@@ -194,6 +197,7 @@ typedef int                 BOOL;
 typedef long LONG;
 typedef unsigned long ULONG;
 typedef LONG HRESULT;
+#endif
 
 namespace rage {
 
@@ -467,7 +471,7 @@ private:
 #if defined(_MSC_VER) && (_MSC_VER <= 1600)
 	typedef std::map<void*, ResourceData*, std::less<void*>, MapAllocator<std::pair<const void*, ResourceData*>> >    MapActive;
 #else
-	typedef std::unordered_map<void*, ResourceData*, std::hash<void*>, std::equal_to<void*>, MapAllocator<std::pair<void*, ResourceData*>> >    MapActive;
+	typedef std::unordered_map<void*, ResourceData*, std::hash<void*>, std::equal_to<void*>, MapAllocator<std::pair<void* const, ResourceData*>> >    MapActive;
 #endif
 	MapActive m_mapActiveResources;
 	
