diff --git a/src/dev_ng/rage/base/src/grcore/resourcecache.cpp b/src/dev_ng/rage/base/src/grcore/resourcecache.cpp
--- a/src/dev_ng/rage/base/src/grcore/resourcecache.cpp
+++ b/src/dev_ng/rage/base/src/grcore/resourcecache.cpp
@@ -6,15 +6,15 @@
 
 #include "resourcecache.h"
 
-#include "bank\bkmgr.h"
-#include "bank\bank.h"
+#include "bank/bkmgr.h"
+#include "bank/bank.h"
 #include "channel.h"
-#include "file\stream.h"
-#include "grcore\debugdraw.h"
+#include "file/stream.h"
+#include "grcore/debugdraw.h"
 #include "image.h"
-#include "profile\timebars.h"
-#include "system\param.h"
-#include "system\timer.h"
+#include "profile/timebars.h"
+#include "system/param.h"
+#include "system/timer.h"
 #include "texturepc.h"
 #include "wrapper_d3d.h"
 
@@ -1385,7 +1385,7 @@ s64 grcResourceCache::GetTotalFreeMemory(MemoryType eType, bool bIgnoreCache) co
 			iTotalFreeAppMemory = sysMemTotalFreeAppMemory();
 			iTotalFreeMemory = sysMemTotalFreeMemory();
 
-			uNextUpdate = min(max(iTotalFreeAppMemory, auMemoryLimits[0]), auMemoryLimits[1]);
+			uNextUpdate = Min(Max(iTotalFreeAppMemory, auMemoryLimits[0]), auMemoryLimits[1]);
 			uNextUpdate = (u64)(((uNextUpdate - auMemoryLimits[0]) / (float)(auMemoryLimits[1] - auMemoryLimits[0])) * (auUpdateRate[1] - auUpdateRate[0]) + auUpdateRate[0]);
 		}
 
