diff --git a/src/dev_ng/rage/base/src/grcore/texture_d3d11.cpp b/src/dev_ng/rage/base/src/grcore/texture_d3d11.cpp
--- a/src/dev_ng/rage/base/src/grcore/texture_d3d11.cpp
+++ b/src/dev_ng/rage/base/src/grcore/texture_d3d11.cpp
@@ -1050,7 +1050,7 @@ grcDevice::Result grcTextureDX11::CreateInternal(CREATE_INTERNAL_INFO &CreateInf
 
 					u32 noOfRows = GetRowCount(uMip);
 
-					u32 amountToCopyPerRow = min(pSrc->SysMemPitch, destMemPitch);
+					u32 amountToCopyPerRow = std::min(pSrc->SysMemPitch, destMemPitch);
 
 					// Copy each slice.
 					for(u32 uSlice=0; uSlice<noOfSlices; uSlice++)
