diff --git a/src/dev_ng/rage/base/src/grcore/vertexbuffer.h b/src/dev_ng/rage/base/src/grcore/vertexbuffer.h
--- a/src/dev_ng/rage/base/src/grcore/vertexbuffer.h
+++ b/src/dev_ng/rage/base/src/grcore/vertexbuffer.h
@@ -152,7 +152,7 @@ namespace rage
 		DECLARE_PLACE(grcVertexBuffer);
 
 		static grcVertexBuffer* Create(int vertCount, const grcFvf& pFvf, bool bReadWrite, bool bDynamic, void* preAllocatedMemory );
-#if RSG_PC && __D3D11
+#if __D3D11 && (RSG_PC || RSG_LINUX)
 		static grcVertexBuffer* Create(int vertCount, const grcFvf& pFvf, grcBufferCreateType CreateType, grcsBufferSyncType SyncType, void* preAllocatedMemory, bool AllocateCPUCopy = false);
 		static grcVertexBuffer* CreateWithData(int VertCount, const grcFvf& pFvf, grcBufferCreateType CreateType, const void *pVertexData, bool AllocateCPUCopy = false);
 #endif // RSG_PC && __D3D11
