diff --git a/src/dev_ng/game/scene/ExtraContent.cpp b/src/dev_ng/game/scene/ExtraContent.cpp
index 75e22b995..ac83d74d6 100644
--- a/src/dev_ng/game/scene/ExtraContent.cpp
+++ b/src/dev_ng/game/scene/ExtraContent.cpp
@@ -98,7 +98,7 @@ RAGE_DEFINE_CHANNEL(dlc, DIAG_SEVERITY_DISPLAY, DIAG_SEVERITY_DISPLAY, DIAG_SEVE
 #define TITLE_UPDATE_RPF_PATH "update:/update.rpf"
 #define TITLE_UPDATE2_RPF_PATH "update:/update2.rpf"
 #else // NG
-#if __FINAL
+#if __PACKEDDEBUG
 #define TITLE_UPDATE_RPF_PATH "update/update.rpf"
 #define TITLE_UPDATE2_RPF_PATH "update/update2.rpf"
 #else
