diff --git a/src/dev_ng/game/streaming/BoxStreamer.cpp b/src/dev_ng/game/streaming/BoxStreamer.cpp
--- a/src/dev_ng/game/streaming/BoxStreamer.cpp
+++ b/src/dev_ng/game/streaming/BoxStreamer.cpp
@@ -11,21 +11,21 @@ STREAMING_OPTIMISATIONS()
 
 // C headers
 // Rage headers
-#include "input\mouse.h"
+#include "input/mouse.h"
 #include "fwsys/timer.h"
 
 // Game headers
-#include "debug/vectormap.h"
+#include "debug/VectorMap.h"
 #include "network/NetworkInterface.h"
 #include "pathserver/ExportCollision.h"
 #include "pathserver/NavGenParam.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "scene/lod/LodMgr.h"
 #include "script/script.h"
 #include "streaming/streamingdebug.h"
 #include "streaming/streamingmodule.h"
 #include "streaming/streamingrequestlist.h"
-#include "vehicles/vehicle.h"
+#include "Vehicles/vehicle.h"
 #include "vehicleAi/VehicleIntelligence.h"
 #include "vehicleAi/task/TaskVehicleMissionBase.h"
 #include "debug/DebugScene.h"
