diff --git a/src/dev_ng/game/streaming/PrioritizedClipSetStreamer.cpp b/src/dev_ng/game/streaming/PrioritizedClipSetStreamer.cpp
--- a/src/dev_ng/game/streaming/PrioritizedClipSetStreamer.cpp
+++ b/src/dev_ng/game/streaming/PrioritizedClipSetStreamer.cpp
@@ -12,7 +12,7 @@
 #include "streaming/streamingvisualize.h"
 
 // Game headers
-#include "task/system/taskhelpers.h"
+#include "task/System/TaskHelpers.h"
 
 STREAMING_OPTIMISATIONS()
 AI_VEHICLE_OPTIMISATIONS()
