diff --git a/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendRunAndGun.cpp b/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendRunAndGun.cpp
--- a/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendRunAndGun.cpp
+++ b/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendRunAndGun.cpp
@@ -3,10 +3,10 @@
 #include "Peds/PedMoveBlend/PedMoveBlendRunAndGun.h"
 
 // Game headers
-#include "Camera/CamInterface.h"
-#include "Camera/Gameplay/Aim/AimCamera.h"
-#include "Camera/Gameplay/GameplayDirector.h"
-#include "Peds/Ped.h"
+#include "camera/CamInterface.h"
+#include "camera/Gameplay/Aim/AimCamera.h"
+#include "camera/Gameplay/GameplayDirector.h"
+#include "Peds/ped.h"
 #include "Peds/Ped_Channel.h"
-#include "Task/default/TaskPlayer.h"
+#include "task/default/TaskPlayer.h"
 #include "debug/DebugScene.h"
