diff --git a/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendInVehicle.cpp b/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendInVehicle.cpp
--- a/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendInVehicle.cpp
+++ b/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendInVehicle.cpp
@@ -1,8 +1,8 @@
 #include "PedMoveBlendInVehicle.h"
-#include "Peds/Ped.h"
+#include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
-#include "Task/default/TaskPlayer.h"
-#include "Task/Vehicle/TaskCar.h"
-#include "Task/Vehicle/TaskInVehicle.h"
+#include "task/default/TaskPlayer.h"
+#include "task/Vehicle/TaskCar.h"
+#include "task/Vehicle/TaskInVehicle.h"
 #include "Vehicles/Metadata/VehicleSeatInfo.h"
 #include "debug/DebugScene.h"
