diff --git a/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendOnSkis.h b/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendOnSkis.h
--- a/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendOnSkis.h
+++ b/src/dev_ng/game/Peds/PedMoveBlend/PedMoveBlendOnSkis.h
@@ -1,8 +1,8 @@
 #ifndef PED_MOVE_BLEND_ON_SKIS_H
 #define	PED_MOVE_BLEND_ON_SKIS_H
 
-#include "Peds\PedMoveBlend\PedMoveBlendBase.h"
-#include "Peds\PedMoveBlend\MoveBlendTable.h"
+#include "Peds/PedMoveBlend/PedMoveBlendBase.h"
+#include "Peds/PedMoveBlend/MoveBlendTable.h"
 #include "scene/RegdRefTypes.h"
 #include "streaming/streamingrequest.h"
 
