diff --git a/src/dev_ng/game/task/Default/TaskSlopeScramble.cpp b/src/dev_ng/game/task/Default/TaskSlopeScramble.cpp
--- a/src/dev_ng/game/task/Default/TaskSlopeScramble.cpp
+++ b/src/dev_ng/game/task/Default/TaskSlopeScramble.cpp
@@ -1,23 +1,23 @@
 
 #include "TaskSlopeScramble.h"
 
-#include "Weapons/WeaponDamage.h"
+#include "weapons/WeaponDamage.h"
 
 #include "animation/Move.h"
 #include "animation/MovePed.h"
 
 #include "event/Events.h"
 #include "math/angmath.h"
-#include "Debug/debugscene.h"
+#include "debug/debugscene.h"
 #include "Peds/PedIntelligence.h"
-#include "System\Control.h"
-#include "Task/Default/TaskPlayer.h"
-#include "Task/Physics/TaskNMHighFall.h"
-#include "Task/System/MotionTaskData.h"
+#include "system/Control.h"
+#include "task/Default/TaskPlayer.h"
+#include "task/Physics/TaskNMHighFall.h"
+#include "task/System/MotionTaskData.h"
 #include "physics/physics.h"
-#include "physics\gtaInst.h"
-#include "Task/Motion/Locomotion/TaskMotionAiming.h"
-#include "Task/Combat/TaskCombatMelee.h"
+#include "physics/gtaInst.h"
+#include "task/Motion/Locomotion/TaskMotionAiming.h"
+#include "task/Combat/TaskCombatMelee.h"
 
 AI_OPTIMISATIONS()
 
