diff --git a/src/dev_ng/game/task/Default/TaskSlopeScramble.h b/src/dev_ng/game/task/Default/TaskSlopeScramble.h
--- a/src/dev_ng/game/task/Default/TaskSlopeScramble.h
+++ b/src/dev_ng/game/task/Default/TaskSlopeScramble.h
@@ -2,9 +2,9 @@
 #define _TASK_SLOPE_SCRAMBLE_H_
 
 #include "Peds/ped.h"
-#include "task/system/Task.h"
+#include "task/System/Task.h"
 #include "Peds/QueriableInterface.h"
-#include "Task/System/TaskFSMClone.h"
+#include "task/System/TaskFSMClone.h"
 #include "task/Motion/TaskMotionBase.h"
 
 class CTaskSlopeScramble : public CTaskFSMClone
