diff --git a/src/dev_ng/game/task/Default/TaskSwimmingWander.h b/src/dev_ng/game/task/Default/TaskSwimmingWander.h
--- a/src/dev_ng/game/task/Default/TaskSwimmingWander.h
+++ b/src/dev_ng/game/task/Default/TaskSwimmingWander.h
@@ -7,8 +7,8 @@
 #ifndef TASK_SWIMMING_WANDER_H
 #define TASK_SWIMMING_WANDER_H
 
-#include "Task/System/Task.h"
-#include "Task/System/Tuning.h"
+#include "task/System/Task.h"
+#include "task/System/Tuning.h"
 
 //***************************************
 //Class:  CTaskSwimmingWander
