diff --git a/src/dev_ng/game/task/Combat/Subtasks/TaskReactToBuddyShot.h b/src/dev_ng/game/task/Combat/Subtasks/TaskReactToBuddyShot.h
--- a/src/dev_ng/game/task/Combat/Subtasks/TaskReactToBuddyShot.h
+++ b/src/dev_ng/game/task/Combat/Subtasks/TaskReactToBuddyShot.h
@@ -9,9 +9,9 @@
 // Game headers
 #include "ai/AITarget.h"
 #include "scene/RegdRefTypes.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/Tuning.h"
+#include "task/System/Task.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
 
 ////////////////////////////////////////////////////////////////////////////////
 // CTaskReactToBuddyShot
