diff --git a/src/dev_ng/game/task/Combat/Subtasks/TaskCombatSubtask.cpp b/src/dev_ng/game/task/Combat/Subtasks/TaskCombatSubtask.cpp
--- a/src/dev_ng/game/task/Combat/Subtasks/TaskCombatSubtask.cpp
+++ b/src/dev_ng/game/task/Combat/Subtasks/TaskCombatSubtask.cpp
@@ -13,7 +13,7 @@
 // Game headers
 #include "Peds/Ped.h"
 #include "Peds/PedIntelligence.h"
-#include "Task\Combat\Subtasks\TaskCombatSubTask.h"
+#include "Task/Combat/Subtasks/TaskCombatSubTask.h"
 
 
 //=========================================================================
