diff --git a/src/dev_ng/game/task/Combat/Subtasks/TaskHeliCombat.h b/src/dev_ng/game/task/Combat/Subtasks/TaskHeliCombat.h
--- a/src/dev_ng/game/task/Combat/Subtasks/TaskHeliCombat.h
+++ b/src/dev_ng/game/task/Combat/Subtasks/TaskHeliCombat.h
@@ -6,9 +6,9 @@
 
 // Game headers
 #include "ai/AITarget.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/Tuning.h"
+#include "task/System/Task.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
 
 // Forward declarations
 class CHeli;
