diff --git a/src/dev_ng/game/task/Combat/Subtasks/TaskAimFromGround.cpp b/src/dev_ng/game/task/Combat/Subtasks/TaskAimFromGround.cpp
--- a/src/dev_ng/game/task/Combat/Subtasks/TaskAimFromGround.cpp
+++ b/src/dev_ng/game/task/Combat/Subtasks/TaskAimFromGround.cpp
@@ -1,11 +1,11 @@
 
-#include "peds/Ped.h"
-#include "Task/Combat/SubTasks/TaskAimFromGround.h"
-#include "Task/Weapons/Gun/TaskGun.h"
-#include "Task/Weapons/Gun/TaskAimGunScripted.h"
-#include "Task/Weapons/Gun/Metadata/ScriptedGunTaskInfo.h"
-#include "Task/Weapons/Gun/Metadata/ScriptedGunTaskInfoMetadataMgr.h"
-#include "Weapons/WeaponTypes.h"
+#include "Peds/ped.h"
+#include "task/Combat/SubTasks/TaskAimFromGround.h"
+#include "task/Weapons/Gun/TaskGun.h"
+#include "task/Weapons/Gun/TaskAimGunScripted.h"
+#include "task/Weapons/Gun/Metadata/ScriptedGunTaskInfo.h"
+#include "task/Weapons/Gun/Metadata/ScriptedGunTaskInfoMetadataMgr.h"
+#include "weapons/WeaponTypes.h"
 
 AI_OPTIMISATIONS()
 
