diff --git a/src/dev_ng/game/task/Combat/Subtasks/TaskVehicleChase.h b/src/dev_ng/game/task/Combat/Subtasks/TaskVehicleChase.h
--- a/src/dev_ng/game/task/Combat/Subtasks/TaskVehicleChase.h
+++ b/src/dev_ng/game/task/Combat/Subtasks/TaskVehicleChase.h
@@ -8,10 +8,10 @@
 #include "atl/array.h"
 
 // Game headers
-#include "scene\RegdRefTypes.h"
-#include "Task\System\Task.h"
-#include "Task\System\TaskHelpers.h"
-#include "Task/System/Tuning.h"
+#include "scene/RegdRefTypes.h"
+#include "task/System/Task.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
 
 // Forward declarations (struct)
 struct sVehicleMissionParams;
