diff --git a/src/dev_ng/game/frontend/page_deck/items/CardItem.cpp b/src/dev_ng/game/frontend/page_deck/items/CardItem.cpp
--- a/src/dev_ng/game/frontend/page_deck/items/CardItem.cpp
+++ b/src/dev_ng/game/frontend/page_deck/items/CardItem.cpp
@@ -17,7 +17,7 @@
 
 // game
 #include "frontend/ui_channel.h"
-#include "network/live/NetworkTelemetry.h"
+#include "network/Live/NetworkTelemetry.h"
 
 FWUI_DEFINE_TYPE( CCardItem, 0x647FEF52 );
 
