diff --git a/src/dev_ng/game/Peds/AssistedMovement.cpp b/src/dev_ng/game/Peds/AssistedMovement.cpp
--- a/src/dev_ng/game/Peds/AssistedMovement.cpp
+++ b/src/dev_ng/game/Peds/AssistedMovement.cpp
@@ -7,25 +7,25 @@
 #include "AssistedMovement.h"
 
 // Rage includes
-#include "Math/angmath.h"
+#include "math/angmath.h"
 
 // Framework includes
 #include "fwmaths/vectorutil.h"
 
 // Game includes
-#include "Camera/CamInterface.h"
-#include "Camera/Debug/DebugDirector.h"
-#include "Camera/Helpers/Frame.h"
+#include "camera/CamInterface.h"
+#include "camera/debug/DebugDirector.h"
+#include "camera/helpers/Frame.h"
 #include "pathserver/ExportCollision.h"
 #include "Peds/PedCapsule.h"
 #include "Peds/PedDebugVisualiser.h"
 #include "Peds/PedMoveBlend/PedMoveBlendInWater.h"
 #include "Peds/PedMoveBlend/PedMoveBlendOnSkis.h"
-#include "Peds/Ped.h"
+#include "Peds/ped.h"
 #include "Peds/PlayerInfo.h"
-#include "Task/Motion/Locomotion/TaskInWater.h"
-#include "Task/Motion/Locomotion/TaskMotionPed.h"
-#include "Task/Movement/TaskGotoPoint.h"
+#include "task/Motion/Locomotion/TaskInWater.h"
+#include "task/Motion/Locomotion/TaskMotionPed.h"
+#include "task/Movement/TaskGotoPoint.h"
 
 AI_OPTIMISATIONS()
 
