diff --git a/src/dev_ng/game/Peds/PedFootstepHelper.cpp b/src/dev_ng/game/Peds/PedFootstepHelper.cpp
--- a/src/dev_ng/game/Peds/PedFootstepHelper.cpp
+++ b/src/dev_ng/game/Peds/PedFootstepHelper.cpp
@@ -2,10 +2,10 @@
 #include "audio/scriptaudioentity.h"
 #include "audioengine/engine.h"
 #include "PedFootstepHelper.h"
-#include "Peds/Ped.h"
-#include "peds/pedintelligence.h"
-#include "Physics/archetype.h"
-#include "Physics/gtaInst.h"
+#include "Peds/ped.h"
+#include "Peds/PedIntelligence.h"
+#include "physics/archetype.h"
+#include "physics/gtaInst.h"
 #include "physics/physics.h"
 #include "task/Weapons/Gun/TaskGun.h"
 #include "scene/playerswitch/PlayerSwitchInterface.h"
