diff --git a/src/dev_ng/game/Peds/PedMotionData.cpp b/src/dev_ng/game/Peds/PedMotionData.cpp
--- a/src/dev_ng/game/Peds/PedMotionData.cpp
+++ b/src/dev_ng/game/Peds/PedMotionData.cpp
@@ -1,9 +1,9 @@
 // File header
 #include "Peds/PedMotionData.h"
 
-#include "Debug/DebugScene.h"
+#include "debug/DebugScene.h"
 #include "fwmaths/angle.h"
-#include "Math/angmath.h"
+#include "math/angmath.h"
 
 AI_OPTIMISATIONS()
 AI_MOTION_OPTIMISATIONS()
