diff --git a/src/dev_ng/game/Peds/PedType.cpp b/src/dev_ng/game/Peds/PedType.cpp
--- a/src/dev_ng/game/Peds/PedType.cpp
+++ b/src/dev_ng/game/Peds/PedType.cpp
@@ -7,16 +7,16 @@
 #include <string.h>
 
 // Rage headers
-#include "System\Timer.h"
+#include "system/timer.h"
 #include "system/memory.h"
 
 // Framework headers
-//#include "fwmaths\Maths.h"
+//#include "fwmaths/Maths.h"
 
 // Game headers
-#include "Debug\Debug.h"
-#include "Peds\PedType.h"
-#include "System\FileMgr.h"
+#include "debug/Debug.h"
+#include "Peds/PedType.h"
+#include "system/FileMgr.h"
 
 CPedType* CPedType::ms_apPedTypes;
 dev_float CPedType::ms_fMaxHeadingChange = ( DtoR * 360.0f*2.5f);
