diff --git a/src/dev_ng/rage/framework/src/fwscript/scripthandler.cpp b/src/dev_ng/rage/framework/src/fwscript/scripthandler.cpp
--- a/src/dev_ng/rage/framework/src/fwscript/scripthandler.cpp
+++ b/src/dev_ng/rage/framework/src/fwscript/scripthandler.cpp
@@ -4,7 +4,7 @@
 // Copyright (C) 1999-2009 Rockstar Games.  All Rights Reserved.
 //
 
-#include "fwscript/scriptHandler.h"
+#include "fwscript/scripthandler.h"
 
 #include "fwnet/netarrayhandlertypes.h"
 #include "fwnet/netarraymgr.h"
