diff --git a/src/dev_ng/rage/suite/src/gpuptfx/ptxgpubase.cpp b/src/dev_ng/rage/suite/src/gpuptfx/ptxgpubase.cpp
--- a/src/dev_ng/rage/suite/src/gpuptfx/ptxgpubase.cpp
+++ b/src/dev_ng/rage/suite/src/gpuptfx/ptxgpubase.cpp
@@ -63,7 +63,7 @@ grcVertexBuffer* CreateInstancedVertexBuffer(int channel)
 	fvf0.SetPosChannel(false);
 
 	// Create the vertex buffer.
-#if RSG_PC
+#if RSG_PC || RSG_LINUX
 	grcVertexBuffer* pVB = grcVertexBuffer::CreateWithData(4, fvf0, grcsBufferCreate_NeitherReadNorWrite, (const void *)&QuadVerts[0]);
 #else // RSG_PC
 	grcVertexBuffer* pVB = grcVertexBuffer::CreateWithData(4, fvf0, (const void *)&QuadVerts[0]);
