diff --git a/src/dev_ng/game/debug/BudgetDisplay.cpp b/src/dev_ng/game/debug/BudgetDisplay.cpp
--- a/src/dev_ng/game/debug/BudgetDisplay.cpp
+++ b/src/dev_ng/game/debug/BudgetDisplay.cpp
@@ -25,7 +25,7 @@
 #include "cutfile/cutfobject.h"
 #include "cutfile/cutfdefines.h"
 #include "cutscene/CutsceneCameraEntity.h"
-#include "renderer/lights/lights.h"
+#include "renderer/Lights/lights.h"
 #include "renderer/PostProcessFX.h"
 #include "renderer/rendertargets.h"
 #include "scene/debug/PostScanDebug.h"
@@ -33,7 +33,7 @@
 #include "timecycle/timecycle.h"
 #include "camera/viewports/viewportmanager.h"
 #include "scene/world/VisibilityMasks.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "parser/manager.h"
 
 
