diff --git a/src/dev_ng/game/debug/BugstarIntegration.cpp b/src/dev_ng/game/debug/BugstarIntegration.cpp
--- a/src/dev_ng/game/debug/BugstarIntegration.cpp
+++ b/src/dev_ng/game/debug/BugstarIntegration.cpp
@@ -16,17 +16,17 @@
 
 // Game headers
 #include "debug/Debug.h"
-#include "camera/Viewports/Viewport.h"
-#include "camera/Viewports/ViewportManager.h"
+#include "camera/viewports/Viewport.h"
+#include "camera/viewports/ViewportManager.h"
 #include "debug/BlockView.h"
-#include "peds/ped.h"
-#include "peds/pedfactory.h"
+#include "Peds/ped.h"
+#include "Peds/PedFactory.h"
 #include "physics/physics.h"
-#include "scene/world/gameworld.h"
+#include "scene/world/GameWorld.h"
 #include "scene/streamer/SceneStreamerMgr.h"
-#include "vehicles/vehicle.h"
-#include "vehicles/vehiclefactory.h"
-#include "objects/DummyObject.h"
+#include "Vehicles/vehicle.h"
+#include "Vehicles/VehicleFactory.h"
+#include "Objects/DummyObject.h"
 #include "parser/manager.h"						// XML
 #include "parser/tree.h"
 #include "system/exec.h"
@@ -34,7 +34,7 @@
 #include "grcore/device.h"
 #include "grcore/light.h"
 #include "grcore/stateblock.h"
-#include "qa/bugstarinterface.h"
+#include "qa/BugstarInterface.h"
 
 #define BUGSTAR_IP_ADDRESS	"rest.bugstar.rockstargames.com:8443"
 
