diff --git a/src/dev_ng/game/debug/DebugLocation.cpp b/src/dev_ng/game/debug/DebugLocation.cpp
--- a/src/dev_ng/game/debug/DebugLocation.cpp
+++ b/src/dev_ng/game/debug/DebugLocation.cpp
@@ -6,14 +6,14 @@
 #include "parser/manager.h"
 
 #include "Peds/ped.h"
-#include "camera/caminterface.h"
-#include "camera/debug/debugdirector.h"
+#include "camera/CamInterface.h"
+#include "camera/debug/DebugDirector.h"
 #include "camera/gameplay/gameplaydirector.h"
-#include "camera/helpers/frame.h"
+#include "camera/helpers/Frame.h"
 #include "camera/viewports/ViewportManager.h"
-#include "game/clock.h"
+#include "game/Clock.h"
 #include "scene/streamer/scenestreamermgr.h"
-#include "scene/world/gameworld.h"
+#include "scene/world/GameWorld.h"
 #include "scene/WarpManager.h"
 #include "script/script_debug.h"
 
