diff --git a/src/dev_ng/game/debug/DebugVehicleInteraction.cpp b/src/dev_ng/game/debug/DebugVehicleInteraction.cpp
--- a/src/dev_ng/game/debug/DebugVehicleInteraction.cpp
+++ b/src/dev_ng/game/debug/DebugVehicleInteraction.cpp
@@ -1,6 +1,6 @@
 #include "DebugVehicleInteraction.h"
 #include "physics/debugEvents.h"
-#include "peds/ped.h"
+#include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
 #include "system/timemgr.h"
 
