diff --git a/src/dev_ng/game/debug/MarketingTools.cpp b/src/dev_ng/game/debug/MarketingTools.cpp
--- a/src/dev_ng/game/debug/MarketingTools.cpp
+++ b/src/dev_ng/game/debug/MarketingTools.cpp
@@ -33,7 +33,7 @@
 #include "scene/world/GameWorld.h"
 #include "system/controlMgr.h"
 #include "text/messages.h"
-#include "vfx\misc\markers.h"
+#include "vfx/misc/markers.h"
 enum eInvincibilityModes
 {
 	VULNERABLE = 0,
