diff --git a/src/dev_ng/game/debug/Editing/CutsceneEditing.cpp b/src/dev_ng/game/debug/Editing/CutsceneEditing.cpp
--- a/src/dev_ng/game/debug/Editing/CutsceneEditing.cpp
+++ b/src/dev_ng/game/debug/Editing/CutsceneEditing.cpp
@@ -1,14 +1,14 @@
 // =============================================================================================== //
 // INCLUDES
 // =============================================================================================== //
-#include "Bank\Console.h"
-#include "Cutscene\CutSceneLightEntity.h"
-#include "Debug\Editing\CommonEditing.h"
-#include "Debug\Editing\CutsceneEditing.h"
-
-#include "cutfile\Cutfdefines.h"
-#include "cutscene\CutSceneManagerNew.h"
-#include "cutscene\cutsmanager.h"
+#include "Bank/Console.h"
+#include "Cutscene/CutSceneLightEntity.h"
+#include "Debug/Editing/CommonEditing.h"
+#include "Debug/Editing/CutsceneEditing.h"
+
+#include "cutfile/Cutfdefines.h"
+#include "cutscene/CutSceneManagerNew.h"
+#include "cutscene/cutsmanager.h"
 
 RENDER_OPTIMISATIONS()
 
