diff --git a/src/dev_ng/game/control/replay/Misc/HidePacket.h b/src/dev_ng/game/control/replay/Misc/HidePacket.h
--- a/src/dev_ng/game/control/replay/Misc/HidePacket.h
+++ b/src/dev_ng/game/control/replay/Misc/HidePacket.h
@@ -6,7 +6,7 @@
 #if GTA_REPLAY
 
 #include "control/replay/Misc/ReplayPacket.h"
-#include "Control/Replay/PacketBasics.h"
+#include "control/replay/PacketBasics.h"
 
 //======================================================================
 
