diff --git a/src/dev_ng/game/frontend/VideoEditor/ui/InstructionalButtons.cpp b/src/dev_ng/game/frontend/VideoEditor/ui/InstructionalButtons.cpp
--- a/src/dev_ng/game/frontend/VideoEditor/ui/InstructionalButtons.cpp
+++ b/src/dev_ng/game/frontend/VideoEditor/ui/InstructionalButtons.cpp
@@ -14,12 +14,12 @@
 #include "frontend/landing_page/LandingPage.h"
 #include "frontend/PauseMenu.h"
 #include "frontend/Scaleform/ScaleFormMgr.h"
-#include "frontend/videoeditor/ui/Editor.h"
-#include "frontend/videoeditor/ui/Menu.h"
-#include "frontend/videoeditor/ui/Timeline.h"
-#include "frontend/videoeditor/ui/Playback.h"
+#include "frontend/VideoEditor/ui/Editor.h"
+#include "frontend/VideoEditor/ui/Menu.h"
+#include "frontend/VideoEditor/ui/Timeline.h"
+#include "frontend/VideoEditor/ui/Playback.h"
 
-#include "Text/TextFormat.h"
+#include "text/TextFormat.h"
 #include "system/controlMgr.h"
 
 #if RSG_ORBIS
@@ -186,7 +186,7 @@ void CVideoEditorInstructionalButtons::Update()
 
 void CVideoEditorInstructionalButtons::Render()
 {
-#if RSG_PC
+#if RSG_PC && !RSG_LINUX
     // hide instructional buttons if SCUI is showing ...hints are confusing
     if (g_rlPc.IsUiShowing())
         return;
