diff --git a/src/dev_ng/game/frontend/VideoEditor/ui/Playback.cpp b/src/dev_ng/game/frontend/VideoEditor/ui/Playback.cpp
--- a/src/dev_ng/game/frontend/VideoEditor/ui/Playback.cpp
+++ b/src/dev_ng/game/frontend/VideoEditor/ui/Playback.cpp
@@ -14,7 +14,7 @@
 #include "math/amath.h"
 #include "rline/rlgamerinfo.h"
 #include "rline/rlsystemui.h"
-#include "scaleform/Include/GFxPlayer.h"
+#include "Scaleform/Include/GFxPlayer.h"
 
 // framework
 #include "fwlocalisation/templateString.h"
@@ -36,8 +36,8 @@
 #include "frontend/PauseMenu.h"
 #include "frontend/ProfileSettings.h"
 #include "frontend/Scaleform/ScaleFormMgr.h"
-#include "Frontend/Scaleform/ScaleFormComplexObjectMgr.h"
-#include "Frontend/ui_channel.h"
+#include "frontend/Scaleform/ScaleFormComplexObjectMgr.h"
+#include "frontend/ui_channel.h"
 #include "frontend/VideoEditor/ui/Editor.h"
 #include "frontend/VideoEditor/ui/Menu.h"
 #include "frontend/VideoEditor/ui/InstructionalButtons.h"
