diff --git a/src/dev_ng/game/Cutscene/CutSceneParticleEffectObject.h b/src/dev_ng/game/Cutscene/CutSceneParticleEffectObject.h
--- a/src/dev_ng/game/Cutscene/CutSceneParticleEffectObject.h
+++ b/src/dev_ng/game/Cutscene/CutSceneParticleEffectObject.h
@@ -15,7 +15,7 @@
 
 //Game includes
 #include "CutScene/CutSceneAnimation.h"
-#include "CutScene/CutSceneDefine.h"
+#include "cutscene/CutSceneDefine.h"
 
 class CCutSceneParticleEffect
 {
