diff --git a/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp b/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp
--- a/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp
+++ b/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp
@@ -13,7 +13,7 @@
 
 //Game includes
 #include "CutScene/CutSceneAnimation.h"
-#include "Cutscene/cutscene_channel.h"
+#include "cutscene/cutscene_channel.h"
 #include "vectormath/legacyconvert.h"
 /////////////////////////////////////////////////////////////////////////////////////
 
