diff --git a/src/dev_ng/game/Cutscene/CutSceneLightObject.h b/src/dev_ng/game/Cutscene/CutSceneLightObject.h
--- a/src/dev_ng/game/Cutscene/CutSceneLightObject.h
+++ b/src/dev_ng/game/Cutscene/CutSceneLightObject.h
@@ -14,11 +14,11 @@
 //Rage includes
 
 //Game includes
-#include "CutScene/CutSceneAnimation.h"
-#include "CutScene/CutSceneDefine.h"
-#include "CutScene/cutsmanager.h"
+#include "Cutscene/CutSceneAnimation.h"
+#include "Cutscene/CutSceneDefine.h"
+#include "cutscene/cutsmanager.h"
 #include "cutfile/cutfobject.h"
-#include "debug/debugscene.h"
+#include "debug/DebugScene.h"
 /////////////////////////////////////////////////////////////////////////////////
 
 class CEntity; 
