diff --git a/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp b/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp
--- a/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp
+++ b/src/dev_ng/game/Cutscene/CutSceneAnimation.cpp
@@ -12,7 +12,7 @@
 #include "fwanimation/directorcomponentcreature.h"
 
 //Game includes
-#include "CutScene/CutSceneAnimation.h"
+#include "Cutscene/CutSceneAnimation.h"
 #include "Cutscene/cutscene_channel.h"
 #include "vectormath/legacyconvert.h"
 /////////////////////////////////////////////////////////////////////////////////////
