diff --git a/src/dev_ng/game/Cutscene/CutSceneCameraEntity.cpp b/src/dev_ng/game/Cutscene/CutSceneCameraEntity.cpp
--- a/src/dev_ng/game/Cutscene/CutSceneCameraEntity.cpp
+++ b/src/dev_ng/game/Cutscene/CutSceneCameraEntity.cpp
@@ -19,14 +19,14 @@
 #include "camera/scripted/ScriptDirector.h"
 #include "camera/system/CameraFactory.h"
 #include "camera/system/CameraMetadata.h"
-#include "cutscene/CutSceneCameraEntity.h"
-#include "cutscene/cutscenecustomevents.h"
-#include "cutscene/CutSceneAnimManager.h"
-#include "cutscene/AnimatedModelEntity.h"
-#include "camera/caminterface.h"
+#include "Cutscene/CutSceneCameraEntity.h"
+#include "Cutscene/CutsceneCustomEvents.h"
+#include "Cutscene/CutSceneAnimManager.h"
+#include "Cutscene/AnimatedModelEntity.h"
+#include "camera/CamInterface.h"
 #include "camera/cutscene/CutsceneDirector.h"
 #include "camera/gameplay/GameplayDirector.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "Cutscene/cutscene_channel.h"
 #include "renderer/PostProcessFX.h"
 #include "renderer/RenderPhases/RenderPhaseCascadeShadows.h"
