diff --git a/src/dev_ng/game/Cutscene/CutSceneLightEntity.cpp b/src/dev_ng/game/Cutscene/CutSceneLightEntity.cpp
--- a/src/dev_ng/game/Cutscene/CutSceneLightEntity.cpp
+++ b/src/dev_ng/game/Cutscene/CutSceneLightEntity.cpp
@@ -14,7 +14,7 @@
 
 //Game Headers
 #include "cutscene/CutSceneLightEntity.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "Cutscene/cutscene_channel.h"
 #include "camera/CamInterface.h"
 
