diff --git a/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h b/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h
--- a/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h
+++ b/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h
@@ -14,8 +14,8 @@
 #include "cutscene/cutsentity.h"
 
 //Game Headers
-#include "cutscene/CutSceneDefine.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneDefine.h"
+#include "Cutscene/CutSceneManagerNew.h"
 
 class CutSceneManager; 
 
