diff --git a/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h b/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h
--- a/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h
+++ b/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.h
@@ -15,7 +15,7 @@
 
 //Game Headers
 #include "cutscene/CutSceneDefine.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 
 class CutSceneManager; 
 
