diff --git a/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.cpp b/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.cpp
--- a/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.cpp
+++ b/src/dev_ng/game/Cutscene/CutSceneOverlayEntity.cpp
@@ -13,10 +13,10 @@
 
 //Game Headers
 #include "camera/viewports/ViewportManager.h"
-#include "cutscene/CutSceneOverlayEntity.h"
-#include "cutscene/cutscene_channel.h"
-#include "cutscene/CutSceneManagerNew.h"
-#include "cutscene/CutSceneCustomEvents.h"
+#include "Cutscene/CutSceneOverlayEntity.h"
+#include "Cutscene/cutscene_channel.h"
+#include "Cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutsceneCustomEvents.h"
 #include "frontend/Scaleform/ScaleFormMgr.h"
 #include "renderer/RenderTargetMgr.h"
 #include "grcore/texture.h"
