diff --git a/src/dev_ng/game/Cutscene/CutSceneLightObject.cpp b/src/dev_ng/game/Cutscene/CutSceneLightObject.cpp
--- a/src/dev_ng/game/Cutscene/CutSceneLightObject.cpp
+++ b/src/dev_ng/game/Cutscene/CutSceneLightObject.cpp
@@ -12,18 +12,18 @@
 
 //Game includes
 #include "audio/ambience/ambientaudioentity.h"
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "CutScene/CutSceneLightObject.h"
 #include "cutscene/CutSceneCameraEntity.h"
 #include "cutscene/animatedmodelentity.h"
-#include "Cutscene/cutscene_channel.h"
-#include "Game/clock.h"
-#include "fwMaths/random.h"
-#include "renderer/lights/lights.h"
+#include "cutscene/cutscene_channel.h"
+#include "game/Clock.h"
+#include "fwmaths/random.h"
+#include "renderer/Lights/lights.h"
 #include "renderer/color.h"
 #include "renderer/Util/Util.h"
 #include "timecycle/TimeCycle.h"
-#include "Vfx/Misc/Coronas.h"
+#include "vfx/misc/Coronas.h"
 #include "control/replay/ReplayLightingManager.h"
 
 /////////////////////////////////////////////////////////////////////////////////////
