diff --git a/src/dev_ng/game/Cutscene/CutSceneAnimManager.cpp b/src/dev_ng/game/Cutscene/CutSceneAnimManager.cpp
--- a/src/dev_ng/game/Cutscene/CutSceneAnimManager.cpp
+++ b/src/dev_ng/game/Cutscene/CutSceneAnimManager.cpp
@@ -11,9 +11,9 @@
 #include "fwsys/timer.h"
 
 //Game headers
-#include "cutscene/CutSceneManagerNew.h"
+#include "Cutscene/CutSceneManagerNew.h"
 #include "cutscene/CutSceneAnimManager.h"
-#include "Cutscene/cutscene_channel.h"
+#include "cutscene/cutscene_channel.h"
 #include "streaming/streaming.h "
 
 ANIM_OPTIMISATIONS()
