diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehiclePlaneChase.h b/src/dev_ng/game/vehicleAi/task/TaskVehiclePlaneChase.h
--- a/src/dev_ng/game/vehicleAi/task/TaskVehiclePlaneChase.h
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehiclePlaneChase.h
@@ -10,9 +10,9 @@
 #include "Task/System/Task.h"
 #include "Task/System/TaskComplex.h"
 #include "Task/System/TaskHelpers.h"
-#include "Task\System\TaskTypes.h"
-#include "VehicleAi\task\TaskVehicleMissionBase.h"
-#include "VehicleAi\task\TaskVehicleGoTo.h"
+#include "Task/System/TaskTypes.h"
+#include "VehicleAi/task/TaskVehicleMissionBase.h"
+#include "VehicleAi/task/TaskVehicleGoTo.h"
 
 class CVehControls;
 class CPlane;
