diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehiclePlayer.cpp b/src/dev_ng/game/vehicleAi/task/TaskVehiclePlayer.cpp
--- a/src/dev_ng/game/vehicleAi/task/TaskVehiclePlayer.cpp
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehiclePlayer.cpp
@@ -1,9 +1,9 @@
 #include "TaskVehiclePlayer.h"
 
 // Framework headers
-#include "fwmaths/vector.h"
+#include "fwmaths/Vector.h"
 #include "fwmaths/angle.h"
-#include "math/vecMath.h"
+#include "math/vecmath.h"
 
 
 // Game headers
@@ -11,14 +11,14 @@
 #include "camera/gameplay/follow/FollowVehicleCamera.h"
 #include "frontend/PauseMenu.h"
 #include "frontend/MultiplayerChat.h"
-#include "system/ControlMgr.h"
+#include "system/controlMgr.h"
 #include "network/NetworkInterface.h"
 #include "network/Events/NetworkEventTypes.h"
 #include "Peds/ped.h"
 #include "Peds/PedIntelligence.h"
-#include "peds/Ped.h"	
+#include "Peds/ped.h"	
 #include "Peds/PlayerInfo.h"
-#include "peds/PlayerSpecialAbility.h"
+#include "Peds/PlayerSpecialAbility.h"
 #include "audio/planeaudioentity.h"
 #include "control/record.h"
 #include "Vehicles/Metadata/VehicleMetadataManager.h"
@@ -31,21 +31,21 @@
 #include "Vehicles/Submarine.h"
 #include "Vehicles/VehicleGadgets.h"
 #include "Vehicles/VehicleFactory.h"
-#include "Vfx/Systems/VfxVehicle.h"
+#include "vfx/systems/VfxVehicle.h"
 #include "scene/world/GameWorld.h"
-#include "Physics/PhysicsHelpers.h"
-#include "vehicles/Metadata/VehicleSeatInfo.h"
-#include "vehicles/Metadata/VehicleLayoutInfo.h"
+#include "physics/PhysicsHelpers.h"
+#include "Vehicles/Metadata/VehicleSeatInfo.h"
+#include "Vehicles/Metadata/VehicleLayoutInfo.h"
 #include "vehicleAi/task/TaskVehicleGoToHelicopter.h"
 #include "vehicleAi/task/TaskVehicleFlying.h"
 #include "vehicleAi/task/TaskVehicleAnimation.h"
 #include "vehicleAi/task/TaskVehicleGoToAutomobile.h"
 #include "vehicleAi/VehicleIntelligence.h"
 #include "debug/DebugScene.h"
-#include "Task/Vehicle/TaskCar.h"
-#include "Task/Vehicle/TaskExitVehicle.h"
-#include "Task/Vehicle/TaskInVehicle.h"
-#include "Game/ModelIndices.h"
+#include "task/Vehicle/TaskCar.h"
+#include "task/Vehicle/TaskExitVehicle.h"
+#include "task/Vehicle/TaskInVehicle.h"
+#include "game/ModelIndices.h"
 
 AI_OPTIMISATIONS()
 AI_VEHICLE_OPTIMISATIONS()
