diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleFollowRecording.cpp b/src/dev_ng/game/vehicleAi/task/TaskVehicleFollowRecording.cpp
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleFollowRecording.cpp
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleFollowRecording.cpp
@@ -1,15 +1,15 @@
 #include "TaskVehicleFollowRecording.h"
 
 // Game headers
-#include "Control/Record.h"
-#include "VehicleAi/task/TaskVehicleGoTo.h"
-#include "VehicleAi/task/TaskVehicleGoToAutomobile.h"
-#include "VehicleAi/task/TaskVehicleGoToHelicopter.h"
-#include "VehicleAi/task/TaskVehicleGoToPlane.h"
-#include "VehicleAi/task/TaskVehiclePark.h"
-#include "VehicleAi/task/TaskVehicleCruise.h"
-#include "VehicleAi/VehicleIntelligence.h"
-#include "Vehicles/Vehicle.h"
+#include "control/Record.h"
+#include "vehicleAi/task/TaskVehicleGoTo.h"
+#include "vehicleAi/task/TaskVehicleGoToAutomobile.h"
+#include "vehicleAi/task/TaskVehicleGoToHelicopter.h"
+#include "vehicleAi/task/TaskVehicleGoToPlane.h"
+#include "vehicleAi/task/TaskVehiclePark.h"
+#include "vehicleAi/task/TaskVehicleCruise.h"
+#include "vehicleAi/VehicleIntelligence.h"
+#include "Vehicles/vehicle.h"
 #include "task/Movement/TaskFollowWaypointRecording.h"
 
 AI_OPTIMISATIONS()
