diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleBlock.h b/src/dev_ng/game/vehicleAi/task/TaskVehicleBlock.h
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleBlock.h
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleBlock.h
@@ -3,10 +3,10 @@
 
 // Game headers
 #include "physics/WorldProbe/shapetestresults.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task/System/Tuning.h"
-#include "VehicleAi/task/TaskVehicleMissionBase.h"
+#include "task/System/Task.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/Tuning.h"
+#include "vehicleAi/task/TaskVehicleMissionBase.h"
 
 //=========================================================================
 // CTaskVehicleBlock
