diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleFlying.h b/src/dev_ng/game/vehicleAi/task/TaskVehicleFlying.h
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleFlying.h
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleFlying.h
@@ -2,14 +2,14 @@
 #define TASK_VEHICLE_FLYING_H
 
 // Gta headers.
-#include "Renderer/HierarchyIds.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskComplex.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task\System\TaskTypes.h"
-#include "Task/System/Tuning.h"
-#include "VehicleAi\task\TaskVehicleMissionBase.h"
-#include "VehicleAi\task\TaskVehicleGotoHelicopter.h"
+#include "renderer/HierarchyIds.h"
+#include "task/System/Task.h"
+#include "task/System/TaskComplex.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/TaskTypes.h"
+#include "task/System/Tuning.h"
+#include "vehicleAi/task/TaskVehicleMissionBase.h"
+#include "vehicleAi/task/TaskVehicleGotoHelicopter.h"
 
 class CVehicle;
 class CVehControls;
@@ -17,7 +17,7 @@ class CHeli;
 class CPlane;
 
 //Rage headers.
-#include "Vector/Vector3.h"
+#include "vector/vector3.h"
 
 //
 //
