diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleGoToHelicopter.cpp b/src/dev_ng/game/vehicleAi/task/TaskVehicleGoToHelicopter.cpp
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleGoToHelicopter.cpp
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleGoToHelicopter.cpp
@@ -5,17 +5,17 @@
 #include "fwscene/stores/staticboundsstore.h"
 
 // Game headers
-#include "Debug/DebugScene.h"
-#include "Vehicles\Heli.h"
-#include "Vehicles\vehicle.h"
-#include "VehicleAi\task\TaskVehicleMissionBase.h"
-#include "VehicleAi\task\TaskVehicleFlying.h"
-#include "VehicleAi\VehicleIntelligence.h"
-#include "vehicleAi\VehicleNavigationHelper.h"
-#include "scene\world\GameWorldHeightMap.h"
-#include "vehicles/VehicleGadgets.h"
-
-#include "Profile/timebars.h"
+#include "debug/DebugScene.h"
+#include "Vehicles/Heli.h"
+#include "Vehicles/vehicle.h"
+#include "vehicleAi/task/TaskVehicleMissionBase.h"
+#include "vehicleAi/task/TaskVehicleFlying.h"
+#include "vehicleAi/VehicleIntelligence.h"
+#include "vehicleAi/VehicleNavigationHelper.h"
+#include "scene/world/GameWorldHeightMap.h"
+#include "Vehicles/VehicleGadgets.h"
+
+#include "profile/timebars.h"
 
 AI_OPTIMISATIONS()
 AI_VEHICLE_OPTIMISATIONS()
