diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleCircle.h b/src/dev_ng/game/vehicleAi/task/TaskVehicleCircle.h
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleCircle.h
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleCircle.h
@@ -5,8 +5,8 @@
 #include "Task/System/Task.h"
 #include "Task/System/TaskComplex.h"
 #include "Task/System/TaskHelpers.h"
-#include "Task\System\TaskTypes.h"
-#include "VehicleAi\task\TaskVehicleMissionBase.h"
+#include "Task/System/TaskTypes.h"
+#include "VehicleAi/task/TaskVehicleMissionBase.h"
 
 class CVehicle;
 class CVehControls;
