diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleGotoBoat.h b/src/dev_ng/game/vehicleAi/task/TaskVehicleGotoBoat.h
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleGotoBoat.h
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleGotoBoat.h
@@ -8,7 +8,7 @@
 
 #include "Task/System/Task.h"
 #include "Task/System/TaskHelpers.h"
-#include "VehicleAi\task\TaskVehicleMissionBase.h"
+#include "VehicleAi/task/TaskVehicleMissionBase.h"
 
 
 class CTaskVehicleGoToBoat : public CTaskVehicleMissionBase
