diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleDeadDriver.cpp b/src/dev_ng/game/vehicleAi/task/TaskVehicleDeadDriver.cpp
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleDeadDriver.cpp
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleDeadDriver.cpp
@@ -4,10 +4,10 @@
 #include "scene/world/GameWorld.h"
 #include "Vehicles/vehicle.h"
 #include "Vehicles/Planes.h"
-#include "Vehicles/heli.h"
-#include "VehicleAi\task\TaskVehicleFlying.h"
-#include "VehicleAi\VehicleIntelligence.h"
-#include "Peds\ped.h"
+#include "Vehicles/Heli.h"
+#include "vehicleAi/task/TaskVehicleFlying.h"
+#include "vehicleAi/VehicleIntelligence.h"
+#include "Peds/ped.h"
 
 AI_OPTIMISATIONS()
 AI_VEHICLE_OPTIMISATIONS()
