diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleGoTo.h b/src/dev_ng/game/vehicleAi/task/TaskVehicleGoTo.h
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleGoTo.h
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleGoTo.h
@@ -2,21 +2,21 @@
 #define TASK_VEHICLE_GO_TO_H
 
 // Gta headers.
-#include "Renderer/HierarchyIds.h"
-#include "Task/System/Task.h"
-#include "Task/System/TaskComplex.h"
-#include "Task/System/TaskHelpers.h"
-#include "Task\System\TaskTypes.h"
-#include "VehicleAi\racingline.h"
-#include "VehicleAi\VehMission.h"
-#include "VehicleAi\task\TaskVehicleMissionBase.h"
-#include "modelinfo\PedModelInfo.h"
+#include "renderer/HierarchyIds.h"
+#include "task/System/Task.h"
+#include "task/System/TaskComplex.h"
+#include "task/System/TaskHelpers.h"
+#include "task/System/TaskTypes.h"
+#include "vehicleAi/racingline.h"
+#include "vehicleAi/VehMission.h"
+#include "vehicleAi/task/TaskVehicleMissionBase.h"
+#include "modelinfo/PedModelInfo.h"
 
 class CVehicle;
 class CVehControls;
 
 //Rage headers.
-#include "Vector/Vector3.h"
+#include "vector/vector3.h"
 #include "vector/vector2.h"
 
 //
