diff --git a/src/dev_ng/game/vehicleAi/task/TaskVehicleLandPlane.cpp b/src/dev_ng/game/vehicleAi/task/TaskVehicleLandPlane.cpp
--- a/src/dev_ng/game/vehicleAi/task/TaskVehicleLandPlane.cpp
+++ b/src/dev_ng/game/vehicleAi/task/TaskVehicleLandPlane.cpp
@@ -2,11 +2,11 @@
 // TaskVehicleLandPlane
 //
 
-#include "Vehicles\Planes.h"
-#include "VehicleAi\task\TaskVehicleLandPlane.h"
-#include "VehicleAi\task\TaskVehicleGotoPlane.h"
-#include "VehicleAi\pathfind.h"
-#include "VehicleAi\VehicleIntelligence.h"
+#include "Vehicles/Planes.h"
+#include "VehicleAi/task/TaskVehicleLandPlane.h"
+#include "VehicleAi/task/TaskVehicleGotoPlane.h"
+#include "VehicleAi/pathfind.h"
+#include "VehicleAi/VehicleIntelligence.h"
 
 AI_OPTIMISATIONS()
 AI_VEHICLE_OPTIMISATIONS()
