diff --git a/src/dev_ng/game/renderer/Deferred/GBuffer.cpp b/src/dev_ng/game/renderer/Deferred/GBuffer.cpp
--- a/src/dev_ng/game/renderer/Deferred/GBuffer.cpp
+++ b/src/dev_ng/game/renderer/Deferred/GBuffer.cpp
@@ -14,12 +14,12 @@
 #include "grcore/AA_shared.h"
 
 // game
-#include "renderer/deferred/gbuffer.h"
+#include "renderer/Deferred/GBuffer.h"
 #include "renderer/rendertargets.h"
-#include "renderer/deferred/deferredlighting.h"
-#include "renderer/renderphases/renderphaseentityselect.h"
+#include "renderer/Deferred/DeferredLighting.h"
+#include "renderer/RenderPhases/RenderPhaseEntitySelect.h"
 #include "renderer/sprite2d.h"
-#include "shaders/shaderlib.h"
+#include "shaders/ShaderLib.h"
 #include "system/xtl.h"
 
 #if __XENON
