diff --git a/src/dev_ng/game/camera/animscene/AnimSceneDirector.cpp b/src/dev_ng/game/camera/animscene/AnimSceneDirector.cpp
--- a/src/dev_ng/game/camera/animscene/AnimSceneDirector.cpp
+++ b/src/dev_ng/game/camera/animscene/AnimSceneDirector.cpp
@@ -7,12 +7,12 @@
 #include "camera/animscene/AnimSceneDirector.h"
 
 
-#include "camera/base/baseCamera.h"
+#include "camera/base/BaseCamera.h"
 #include "camera/cutscene/AnimatedCamera.h"
 #include "camera/system/CameraMetadata.h"
 #include "camera/system/CameraFactory.h"
-#include "fwAnimation/animhelpers.h"
-#include "fwAnimation/animdirector.h"
+#include "fwanimation/animhelpers.h"
+#include "fwanimation/animdirector.h"
 
 CAMERA_OPTIMISATIONS()
 
