diff --git a/src/dev_ng/game/Stats/StatsDataMgr.cpp b/src/dev_ng/game/Stats/StatsDataMgr.cpp
--- a/src/dev_ng/game/Stats/StatsDataMgr.cpp
+++ b/src/dev_ng/game/Stats/StatsDataMgr.cpp
@@ -12,9 +12,9 @@
 #include "parser/tree.h"
 #include "parser/treenode.h"
 #include "parser/manager.h"
-#include "String/stringhash.h"
-#include "Diag/output.h"
-#include "System/param.h"
+#include "string/stringhash.h"
+#include "diag/output.h"
+#include "system/param.h"
 #include "data/callback.h"
 #include "rline/profilestats/rlprofilestatscommon.h"
 #include "script/thread.h"
@@ -37,9 +37,9 @@
 #include "Stats/StatsInterface.h"
 
 // Game headers
-#include "core/game.h"
-#include "Scene/DataFileMgr.h"
-#include "Text/TextFile.h"
+#include "Core/game.h"
+#include "scene/DataFileMgr.h"
+#include "text/TextFile.h"
 #include "script/script_channel.h"
 #include "script/script.h"
 #include "Peds/PlayerInfo.h"
@@ -48,15 +48,15 @@
 #include "frontend/ContextMenu.h"
 
 // network headers
-#include "Network/NetworkInterface.h"
-#include "Network/Live/livemanager.h"
-#include "Network/Network.h"
-#include "Network/players/NetGamePlayer.h"
-#include "Network/Stats/NetworkStockMarketStats.h"
-#include "Network/Live/NetworkTelemetry.h"
-#include "Network/Shop/Catalog.h"
-#include "Network/Cloud/Tunables.h"
-#include "Network/Cloud/UserContentManager.h"
+#include "network/NetworkInterface.h"
+#include "network/Live/livemanager.h"
+#include "network/Network.h"
+#include "network/Players/NetGamePlayer.h"
+#include "network/Stats/NetworkStockMarketStats.h"
+#include "network/Live/NetworkTelemetry.h"
+#include "network/Shop/Catalog.h"
+#include "network/Cloud/Tunables.h"
+#include "network/Cloud/UserContentManager.h"
 
 #include "text/TextConversion.h"
 #include "Core/gamesessionstatemachine.h"
@@ -64,7 +64,7 @@
 #include "SaveLoad/GenericGameStorage.h"
 
 #if RSG_DURANGO
-#include "Network/Live/Events_durango.h"
+#include "network/Live/Events_durango.h"
 #endif
 
 #if __ASSERT
@@ -2208,7 +2208,7 @@ CStatsDataMgr::Clear()
 	while(m_aStatsData.GetCount() > 0)
 	{
 		sStatDataPtr* ppStat = *m_aStatsData.Begin();
-		sStatData*     pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : 0;
+		sStatData*     pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : nullptr;
 
 		if (pStat)
 		{
@@ -3156,7 +3156,7 @@ CStatsDataMgr::HandleEventProfileStats(const rlProfileStatsEvent* evt)
 			if(statVerify(synchEvt->m_Value))
 			{
 				sStatDataPtr* ppStat = GetStatPtr(STAT_ID(synchEvt->m_StatId));
-				sStatData* pStat     = ppStat && ppStat->KeyIsvalid() ? *ppStat : 0;
+				sStatData* pStat     = ppStat && ppStat->KeyIsvalid() ? *ppStat : nullptr;
 
 				if(pStat)
 				{
@@ -3298,7 +3298,7 @@ CStatsDataMgr::HandleEventProfileStats(const rlProfileStatsEvent* evt)
 			if(statVerify(localValueEvt->m_Value))
 			{
 				sStatDataPtr* ppStat = GetStatPtr(STAT_ID(localValueEvt->m_StatId));
-				sStatData* pStat     = ppStat && ppStat->KeyIsvalid() ? *ppStat : 0;
+				sStatData* pStat     = ppStat && ppStat->KeyIsvalid() ? *ppStat : nullptr;
 				if(pStat)
 				{
 					if (pStat->GetDesc().GetSynched())
@@ -3352,7 +3352,7 @@ CStatsDataMgr::HandleEventProfileStats(const rlProfileStatsEvent* evt)
 #endif // __DEV
 
 			sStatDataPtr *ppStat = GetStatPtr(STAT_ID(flushEvt->m_StatId));
-			sStatData *pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : 0;
+			sStatData *pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : nullptr;
 
 			// This event is triggered in response to us publishing the stat, so it should be safe to verify that
 			// this stat is indeed a profile stat.
@@ -3386,7 +3386,7 @@ void
 CStatsDataMgr::OnSynchCommunityStats(const int statId, const float result)
 {
 	sStatDataPtr* ppStat = GetStatPtr(STAT_ID(statId));
-	sStatData*     pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : 0;
+	sStatData*     pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : nullptr;
 	if (pStat && statVerify(pStat->GetDesc().GetIsCommunityStat()) && statVerify(!pStat->GetDesc().GetIsProfileStat()))
 	{
 		if(pStat->GetDesc().GetIsOnlineData())
@@ -4693,7 +4693,7 @@ void CStatsDataMgr::Bank_CreateSpWidgets()
 					while ((iter != CStatsMgr::GetStatsDataMgr().StatIterEnd()))
 					{
 						sStatDataPtr* ppStat = *iter;
-						sStatData*     pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : 0;
+						sStatData*     pStat = ppStat && ppStat->KeyIsvalid() ? *ppStat : nullptr;
 
 						if (pStat && !pStat->GetDesc().GetIsOnlineData())
 						{
@@ -4895,7 +4895,7 @@ void CStatsDataMgr::Bank_CreateMpWidgets()
 void
 CStatsDataMgr::Bank_AddStatWidgetValue(bkBank* bank, sStatDataPtr* ppStat)
 {
-	sStatData* pStat = ppStat ? *ppStat : 0;
+	sStatData* pStat = ppStat ? *ppStat : nullptr;
 	
 	if (bank && pStat && !pStat->IsObfuscated())
 	{
